<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.39.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/BTF2T2b_R1-trimmed.error_rates.pdf">errors</a></td><td>BTF2T2b_R1-trimmed</td><td align="right">12,690,232</td><td align="right">1,905,562,065</td><td align="right">100.0%</td><td align="right">150.2&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">99.3%</td></tr><tr ><td><a href="calibration/BTF2T2b_R2-trimmed.error_rates.pdf">errors</a></td><td>BTF2T2b_R2-trimmed</td><td align="right">11,896,373</td><td align="right">1,785,689,356</td><td align="right">93.7%</td><td align="right">150.1&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">98.3%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>24,586,605</b></td><td align="right"><b>3,691,251,421</b></td><td align="right">96.9%</td><td align="right">150.1&nbsp;bases</td><td align="right"><b>151</b>&nbsp;bases</td><td align="right">98.8%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit relative_variance</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/contig_1.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>contig_1</td><td align="right">808,451</td><td align="center">748.6</td><td align="center">6.8</td><td align="center">16.6%</td><td>Escherichia coli BTF2T1Ec contig_1, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_2.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>contig_2</td><td align="right">584,778</td><td align="center">699.5</td><td align="center">5.0</td><td align="center">11.2%</td><td>Escherichia coli BTF2T1Ec contig_2, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_3.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>contig_3</td><td align="right">279,010</td><td align="center">669.7</td><td align="center">5.8</td><td align="center">5.1%</td><td>Escherichia coli BTF2T1Ec contig_3, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_4.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>contig_4</td><td align="right">258,379</td><td align="center">798.1</td><td align="center">7.3</td><td align="center">5.7%</td><td>Escherichia coli BTF2T1Ec contig_4, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_5.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>contig_5</td><td align="right">258,257</td><td align="center">826.6</td><td align="center">6.4</td><td align="center">5.8%</td><td>Escherichia coli BTF2T1Ec contig_5, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_6.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>contig_6</td><td align="right">243,983</td><td align="center">822.6</td><td align="center">7.4</td><td align="center">5.5%</td><td>Escherichia coli BTF2T1Ec contig_6, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_7.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>contig_7</td><td align="right">209,663</td><td align="center">771.3</td><td align="center">6.7</td><td align="center">4.4%</td><td>Escherichia coli BTF2T1Ec contig_7, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_8.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>contig_8</td><td align="right">200,840</td><td align="center">872.8</td><td align="center">7.1</td><td align="center">4.8%</td><td>Escherichia coli BTF2T1Ec contig_8, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_9.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>contig_9</td><td align="right">187,087</td><td align="center">803.0</td><td align="center">6.6</td><td align="center">4.1%</td><td>Escherichia coli BTF2T1Ec contig_9, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_10.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>contig_10</td><td align="right">168,181</td><td align="center">808.9</td><td align="center">5.8</td><td align="center">3.7%</td><td>Escherichia coli BTF2T1Ec contig_10, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_11.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>contig_11</td><td align="right">149,861</td><td align="center">833.0</td><td align="center">8.2</td><td align="center">3.4%</td><td>Escherichia coli BTF2T1Ec contig_11, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_12.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>contig_12</td><td align="right">144,402</td><td align="center">771.6</td><td align="center">5.1</td><td align="center">3.1%</td><td>Escherichia coli BTF2T1Ec contig_12, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_13.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>contig_13</td><td align="right">119,107</td><td align="center">718.9</td><td align="center">5.8</td><td align="center">2.4%</td><td>Escherichia coli BTF2T1Ec contig_13, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_14.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>contig_14</td><td align="right">111,442</td><td align="center">809.2</td><td align="center">5.5</td><td align="center">2.5%</td><td>Escherichia coli BTF2T1Ec contig_14, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_15.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>contig_15</td><td align="right">110,233</td><td align="center">692.1</td><td align="center">5.1</td><td align="center">2.1%</td><td>Escherichia coli BTF2T1Ec contig_15, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_16.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>contig_16</td><td align="right">88,244</td><td align="center">799.6</td><td align="center">6.3</td><td align="center">1.9%</td><td>Escherichia coli BTF2T1Ec contig_16, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_17.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>contig_17</td><td align="right">87,810</td><td align="center">817.5</td><td align="center">5.9</td><td align="center">2.0%</td><td>Escherichia coli BTF2T1Ec contig_17, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_18.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>contig_18</td><td align="right">87,791</td><td align="center">780.5</td><td align="center">5.6</td><td align="center">1.9%</td><td>Escherichia coli BTF2T1Ec contig_18, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_19.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>contig_19</td><td align="right">60,743</td><td align="center">788.7</td><td align="center">6.3</td><td align="center">1.3%</td><td>Escherichia coli BTF2T1Ec contig_19, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_20.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>contig_20</td><td align="right">58,046</td><td align="center">853.2</td><td align="center">6.0</td><td align="center">1.3%</td><td>Escherichia coli BTF2T1Ec contig_20, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_21.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>contig_21</td><td align="right">54,497</td><td align="center">667.0</td><td align="center">5.3</td><td align="center">1.0%</td><td>Escherichia coli BTF2T1Ec contig_21, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_22.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>contig_22</td><td align="right">45,322</td><td align="center">784.6</td><td align="center">5.1</td><td align="center">1.0%</td><td>Escherichia coli BTF2T1Ec contig_22, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_23.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>contig_23</td><td align="right">41,847</td><td align="center">826.5</td><td align="center">5.4</td><td align="center">0.9%</td><td>Escherichia coli BTF2T1Ec contig_23, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_24.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>contig_24</td><td align="right">40,552</td><td align="center">813.9</td><td align="center">4.8</td><td align="center">0.9%</td><td>Escherichia coli BTF2T1Ec contig_24, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_25.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>contig_25</td><td align="right">37,834</td><td align="center">679.6</td><td align="center">4.7</td><td align="center">0.7%</td><td>Escherichia coli BTF2T1Ec contig_25, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_26.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>contig_26</td><td align="right">35,455</td><td align="center">802.8</td><td align="center">5.9</td><td align="center">0.8%</td><td>Escherichia coli BTF2T1Ec contig_26, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_27.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>contig_27</td><td align="right">34,976</td><td align="center">783.0</td><td align="center">7.0</td><td align="center">0.7%</td><td>Escherichia coli BTF2T1Ec contig_27, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_28.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>contig_28</td><td align="right">32,830</td><td align="center">827.6</td><td align="center">4.5</td><td align="center">0.7%</td><td>Escherichia coli BTF2T1Ec contig_28, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_29.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>contig_29</td><td align="right">30,834</td><td align="center">816.7</td><td align="center">8.7</td><td align="center">0.7%</td><td>Escherichia coli BTF2T1Ec contig_29, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_30.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>contig_30</td><td align="right">26,140</td><td align="center">858.4</td><td align="center">4.6</td><td align="center">0.6%</td><td>Escherichia coli BTF2T1Ec contig_30, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_31.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>contig_31</td><td align="right">25,073</td><td align="center">679.2</td><td align="center">4.2</td><td align="center">0.5%</td><td>Escherichia coli BTF2T1Ec contig_31, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_32.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>contig_32</td><td align="right">11,694</td><td align="center">746.9</td><td align="center">7.3</td><td align="center">0.2%</td><td>Escherichia coli BTF2T1Ec contig_32, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_33.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>contig_33</td><td align="right">7,501</td><td align="center">823.2</td><td align="center">3.2</td><td align="center">0.2%</td><td>Escherichia coli BTF2T1Ec contig_33, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_34.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>contig_34</td><td align="right">4,392</td><td align="center">891.1</td><td align="center">7.6</td><td align="center">0.1%</td><td>Escherichia coli BTF2T1Ec contig_34, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_35.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>contig_35</td><td align="right">3,355</td><td align="center">841.9</td><td align="center">14.3</td><td align="center">0.1%</td><td>Escherichia coli BTF2T1Ec contig_35, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_36.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>contig_36</td><td align="right">3,028</td><td align="center">6158.0</td><td align="center">11.5</td><td align="center">0.5%</td><td>Escherichia coli BTF2T1Ec contig_36, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_37.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>contig_37</td><td align="right">2,805</td><td align="center">713.1</td><td align="center">3.3</td><td align="center">0.1%</td><td>Escherichia coli BTF2T1Ec contig_37, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_38.overview.png">coverage</a></td><td><a href="calibration/37.unique_coverage.pdf">distribution</a></td><td>contig_38</td><td align="right">2,227</td><td align="center">750.2</td><td align="center">32.2</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_38, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_39.overview.png">coverage</a></td><td><a href="calibration/38.unique_coverage.pdf">distribution</a></td><td>contig_39</td><td align="right">1,986</td><td align="center">954.9</td><td align="center">4.4</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_39, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_40.overview.png">coverage</a></td><td><a href="calibration/39.unique_coverage.pdf">distribution</a></td><td>contig_40</td><td align="right">1,791</td><td align="center">702.2</td><td align="center">16.1</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_40, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_41.overview.png">coverage</a></td><td><a href="calibration/40.unique_coverage.pdf">distribution</a></td><td>contig_41</td><td align="right">1,723</td><td align="center">813.2</td><td align="center">4.3</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_41, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_42.overview.png">coverage</a></td><td><a href="calibration/41.unique_coverage.pdf">distribution</a></td><td>contig_42</td><td align="right">1,702</td><td align="center">5967.2</td><td align="center">7.0</td><td align="center">0.3%</td><td>Escherichia coli BTF2T1Ec contig_42, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_43.overview.png">coverage</a></td><td><a href="calibration/42.unique_coverage.pdf">distribution</a></td><td>contig_43</td><td align="right">1,622</td><td align="center">759.6</td><td align="center">3.7</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_43, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_44.overview.png">coverage</a></td><td><a href="calibration/43.unique_coverage.pdf">distribution</a></td><td>contig_44</td><td align="right">1,395</td><td align="center">2615.3</td><td align="center">8.2</td><td align="center">0.1%</td><td>Escherichia coli BTF2T1Ec contig_44, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_45.overview.png">coverage</a></td><td><a href="calibration/44.unique_coverage.pdf">distribution</a></td><td>contig_45</td><td align="right">1,345</td><td align="center">1594.2</td><td align="center">4.0</td><td align="center">0.1%</td><td>Escherichia coli BTF2T1Ec contig_45, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_46.overview.png">coverage</a></td><td><a href="calibration/45.unique_coverage.pdf">distribution</a></td><td>contig_46</td><td align="right">1,211</td><td align="center">1626.3</td><td align="center">2.9</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_46, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_47.overview.png">coverage</a></td><td><a href="calibration/46.unique_coverage.pdf">distribution</a></td><td>contig_47</td><td align="right">1,151</td><td align="center">1423.9</td><td align="center">2.6</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_47, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_48.overview.png">coverage</a></td><td><a href="calibration/47.unique_coverage.pdf">distribution</a></td><td>contig_48</td><td align="right">1,090</td><td align="center">14545.1</td><td align="center">125.0</td><td align="center">0.4%</td><td>Escherichia coli BTF2T1Ec contig_48, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_49.overview.png">coverage</a></td><td><a href="calibration/48.unique_coverage.pdf">distribution</a></td><td>contig_49</td><td align="right">1,040</td><td align="center">753.0</td><td align="center">1.2</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_49, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_50.overview.png">coverage</a></td><td><a href="calibration/49.unique_coverage.pdf">distribution</a></td><td>contig_50</td><td align="right">1,004</td><td align="center">1640.6</td><td align="center">5.7</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_50, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_51.overview.png">coverage</a></td><td><a href="calibration/50.unique_coverage.pdf">distribution</a></td><td>contig_51</td><td align="right">956</td><td align="center">1695.9</td><td align="center">5.9</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_51, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_52.overview.png">coverage</a></td><td><a href="calibration/51.unique_coverage.pdf">distribution</a></td><td>contig_52</td><td align="right">949</td><td align="center">2112.5</td><td align="center">4.9</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_52, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_53.overview.png">coverage</a></td><td><a href="calibration/52.unique_coverage.pdf">distribution</a></td><td>contig_53</td><td align="right">909</td><td align="center">2179.8</td><td align="center">2.1</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_53, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_54.overview.png">coverage</a></td><td><a href="calibration/53.unique_coverage.pdf">distribution</a></td><td>contig_54</td><td align="right">739</td><td align="center">717.4</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_54, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_55.overview.png">coverage</a></td><td><a href="calibration/54.unique_coverage.pdf">distribution</a></td><td>contig_55</td><td align="right">590</td><td align="center">703.5</td><td align="center">2.4</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_55, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_56.overview.png">coverage</a></td><td><a href="calibration/55.unique_coverage.pdf">distribution</a></td><td>contig_56</td><td align="right">581</td><td align="center">[617.9]</td><td align="center">[40.2]</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_56, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_57.overview.png">coverage</a></td><td><a href="calibration/56.unique_coverage.pdf">distribution</a></td><td>contig_57</td><td align="right">514</td><td align="center">3296.7</td><td align="center">1.1</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_57, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_58.overview.png">coverage</a></td><td><a href="calibration/57.unique_coverage.pdf">distribution</a></td><td>contig_58</td><td align="right">430</td><td align="center">779.9</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_58, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_59.overview.png">coverage</a></td><td><a href="calibration/58.unique_coverage.pdf">distribution</a></td><td>contig_59</td><td align="right">430</td><td align="center">175.8</td><td align="center">15.8</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_59, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_60.overview.png">coverage</a></td><td><a href="calibration/59.unique_coverage.pdf">distribution</a></td><td>contig_60</td><td align="right">396</td><td align="center">741.3</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_60, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_61.overview.png">coverage</a></td><td><a href="calibration/60.unique_coverage.pdf">distribution</a></td><td>contig_61</td><td align="right">367</td><td align="center">[414.1]</td><td align="center">[131.7]</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_61, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_62.overview.png">coverage</a></td><td><a href="calibration/61.unique_coverage.pdf">distribution</a></td><td>contig_62</td><td align="right">389</td><td align="center">1519.2</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_62, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_63.overview.png">coverage</a></td><td><a href="calibration/62.unique_coverage.pdf">distribution</a></td><td>contig_63</td><td align="right">337</td><td align="center">1499.9</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_63, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_64.overview.png">coverage</a></td><td><a href="calibration/63.unique_coverage.pdf">distribution</a></td><td>contig_64</td><td align="right">310</td><td align="center">[2210.0]</td><td align="center">[15.8]</td><td align="center">0.0%</td><td>Escherichia coli BTF2T1Ec contig_64, whole genome shotgun sequence.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>4,681,627</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit relative_variance</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="reject_table_row">Fit failed</span> Negative binomial fit failed for this reference sequence. It may have an unusual coverage depth distribution. JC and MC predictions may be less accurate.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">80684</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">1013</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.065</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>contig_1</td><td align="center">0.25982</td></tr><tr >
<td>contig_2</td><td align="center">0.27957</td></tr><tr >
<td>contig_3</td><td align="center">0.29835</td></tr><tr >
<td>contig_4</td><td align="center">0.24015</td></tr><tr >
<td>contig_5</td><td align="center">0.23058</td></tr><tr >
<td>contig_6</td><td align="center">0.23543</td></tr><tr >
<td>contig_7</td><td align="center">0.25010</td></tr><tr >
<td>contig_8</td><td align="center">0.21633</td></tr><tr >
<td>contig_9</td><td align="center">0.23995</td></tr><tr >
<td>contig_10</td><td align="center">0.23864</td></tr><tr >
<td>contig_11</td><td align="center">0.23177</td></tr><tr >
<td>contig_12</td><td align="center">0.25187</td></tr><tr >
<td>contig_13</td><td align="center">0.26865</td></tr><tr >
<td>contig_14</td><td align="center">0.23975</td></tr><tr >
<td>contig_15</td><td align="center">0.28694</td></tr><tr >
<td>contig_16</td><td align="center">0.23761</td></tr><tr >
<td>contig_17</td><td align="center">0.23706</td></tr><tr >
<td>contig_18</td><td align="center">0.25011</td></tr><tr >
<td>contig_19</td><td align="center">0.24512</td></tr><tr >
<td>contig_20</td><td align="center">0.22837</td></tr><tr >
<td>contig_21</td><td align="center">0.29284</td></tr><tr >
<td>contig_22</td><td align="center">0.25162</td></tr><tr >
<td>contig_23</td><td align="center">0.23140</td></tr><tr >
<td>contig_24</td><td align="center">0.23567</td></tr><tr >
<td>contig_25</td><td align="center">0.29008</td></tr><tr >
<td>contig_26</td><td align="center">0.23925</td></tr><tr >
<td>contig_27</td><td align="center">0.24736</td></tr><tr >
<td>contig_28</td><td align="center">0.22992</td></tr><tr >
<td>contig_29</td><td align="center">0.23088</td></tr><tr >
<td>contig_30</td><td align="center">0.22559</td></tr><tr >
<td>contig_31</td><td align="center">0.29095</td></tr><tr >
<td>contig_32</td><td align="center">0.26467</td></tr><tr >
<td>contig_33</td><td align="center">0.24597</td></tr><tr >
<td>contig_34</td><td align="center">0.23691</td></tr><tr >
<td>contig_35</td><td align="center">0.25762</td></tr><tr >
<td>contig_36</td><td align="center">0.04293</td></tr><tr >
<td>contig_37</td><td align="center">0.29935</td></tr><tr >
<td>contig_38</td><td align="center">0.31702</td></tr><tr >
<td>contig_39</td><td align="center">0.24102</td></tr><tr >
<td>contig_40</td><td align="center">0.34398</td></tr><tr >
<td>contig_41</td><td align="center">0.26407</td></tr><tr >
<td>contig_42</td><td align="center">0.07697</td></tr><tr >
<td>contig_43</td><td align="center">0.33529</td></tr><tr >
<td>contig_44</td><td align="center">0.03685</td></tr><tr >
<td>contig_45</td><td align="center">0.17026</td></tr><tr >
<td>contig_46</td><td align="center">0.24057</td></tr><tr >
<td>contig_47</td><td align="center">0.20547</td></tr><tr >
<td>contig_48</td><td align="center">0.10826</td></tr><tr >
<td>contig_49</td><td align="center">0.26613</td></tr><tr >
<td>contig_50</td><td align="center">0.20169</td></tr><tr >
<td>contig_51</td><td align="center">0.19822</td></tr><tr >
<td>contig_52</td><td align="center">0.19073</td></tr><tr >
<td>contig_53</td><td align="center">0.18372</td></tr><tr >
<td>contig_54</td><td align="center">0.38566</td></tr><tr >
<td>contig_55</td><td align="center">0.42881</td></tr><tr >
<td>contig_56</td><td align="center">NA</td></tr><tr >
<td>contig_57</td><td align="center">0.17149</td></tr><tr >
<td>contig_58</td><td align="center">0.45814</td></tr><tr >
<td>contig_59</td><td align="center">1.00000</td></tr><tr >
<td>contig_60</td><td align="center">0.46717</td></tr><tr >
<td>contig_61</td><td align="center">NA</td></tr><tr >
<td>contig_62</td><td align="center">0.34906</td></tr><tr >
<td>contig_63</td><td align="center">0.45697</td></tr><tr >
<td>contig_64</td><td align="center">NA</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.5.4</td></tr>
<tr><td>R</td><td>4.4.1</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>12:55:57&nbsp;28&nbsp;Feb&nbsp;2025</td><td>13:00:34&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;minutes&nbsp;37&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>13:00:34&nbsp;28&nbsp;Feb&nbsp;2025</td><td>13:05:08&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;minutes&nbsp;34&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>13:05:08&nbsp;28&nbsp;Feb&nbsp;2025</td><td>13:08:33&nbsp;28&nbsp;Feb&nbsp;2025</td><td>3&nbsp;minutes&nbsp;25&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>13:08:33&nbsp;28&nbsp;Feb&nbsp;2025</td><td>13:16:48&nbsp;28&nbsp;Feb&nbsp;2025</td><td>8&nbsp;minutes&nbsp;15&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>13:16:48&nbsp;28&nbsp;Feb&nbsp;2025</td><td>13:20:44&nbsp;28&nbsp;Feb&nbsp;2025</td><td>3&nbsp;minutes&nbsp;56&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>13:20:44&nbsp;28&nbsp;Feb&nbsp;2025</td><td>13:23:08&nbsp;28&nbsp;Feb&nbsp;2025</td><td>2&nbsp;minutes&nbsp;24&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>13:23:08&nbsp;28&nbsp;Feb&nbsp;2025</td><td>13:29:26&nbsp;28&nbsp;Feb&nbsp;2025</td><td>6&nbsp;minutes&nbsp;18&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>13:29:26&nbsp;28&nbsp;Feb&nbsp;2025</td><td>13:34:50&nbsp;28&nbsp;Feb&nbsp;2025</td><td>5&nbsp;minutes&nbsp;24&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>13:34:50&nbsp;28&nbsp;Feb&nbsp;2025</td><td>13:41:20&nbsp;28&nbsp;Feb&nbsp;2025</td><td>6&nbsp;minutes&nbsp;30&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>13:41:20&nbsp;28&nbsp;Feb&nbsp;2025</td><td>13:42:39&nbsp;28&nbsp;Feb&nbsp;2025</td><td>1&nbsp;minute&nbsp;19&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>13:42:39&nbsp;28&nbsp;Feb&nbsp;2025</td><td>14:17:08&nbsp;28&nbsp;Feb&nbsp;2025</td><td>34&nbsp;minutes&nbsp;29&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>14:17:08&nbsp;28&nbsp;Feb&nbsp;2025</td><td>14:17:11&nbsp;28&nbsp;Feb&nbsp;2025</td><td>3&nbsp;seconds</td></tr>
<tr><td>Output</td><td>14:17:11&nbsp;28&nbsp;Feb&nbsp;2025</td><td>14:17:29&nbsp;28&nbsp;Feb&nbsp;2025</td><td>18&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>14:17:11&nbsp;28&nbsp;Feb&nbsp;2025</td><td>14:17:18&nbsp;28&nbsp;Feb&nbsp;2025</td><td>7&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>14:17:18&nbsp;28&nbsp;Feb&nbsp;2025</td><td>14:17:18&nbsp;28&nbsp;Feb&nbsp;2025</td><td>0&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>1&nbsp;hour&nbsp;21&nbsp;minutes&nbsp;39&nbsp;seconds</b></td>
</tr>
</table></body></html>