<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.38.2&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/BTF60T2_R1-trimmed.error_rates.pdf">errors</a></td><td>BTF60T2_R1-trimmed</td><td align="right">20,382,684</td><td align="right">3,047,858,547</td><td align="right">100.0%</td><td align="right">149.5&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">99.4%</td></tr><tr ><td><a href="calibration/BTF60T2_R2-trimmed.error_rates.pdf">errors</a></td><td>BTF60T2_R2-trimmed</td><td align="right">19,933,951</td><td align="right">2,980,099,865</td><td align="right">97.8%</td><td align="right">149.5&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">99.0%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>40,316,635</b></td><td align="right"><b>6,027,958,412</b></td><td align="right">98.9%</td><td align="right">149.5&nbsp;bases</td><td align="right"><b>151</b>&nbsp;bases</td><td align="right">99.2%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit relative_variance</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/contig_1.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>contig_1</td><td align="right">957,843</td><td align="center">1085.8</td><td align="center">7.6</td><td align="center">17.5%</td><td>contig_1, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_2.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>contig_2</td><td align="right">400,628</td><td align="center">1126.1</td><td align="center">8.5</td><td align="center">7.6%</td><td>contig_2, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_3.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>contig_3</td><td align="right">378,445</td><td align="center">1132.3</td><td align="center">8.8</td><td align="center">7.2%</td><td>contig_3, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_4.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>contig_4</td><td align="right">359,595</td><td align="center">1093.5</td><td align="center">7.5</td><td align="center">6.6%</td><td>contig_4, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_5.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>contig_5</td><td align="right">319,074</td><td align="center">1046.8</td><td align="center">6.7</td><td align="center">5.6%</td><td>contig_5, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_6.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>contig_6</td><td align="right">307,934</td><td align="center">1110.1</td><td align="center">7.6</td><td align="center">5.8%</td><td>contig_6, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_7.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>contig_7</td><td align="right">300,046</td><td align="center">1141.9</td><td align="center">8.2</td><td align="center">5.8%</td><td>contig_7, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_8.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>contig_8</td><td align="right">259,720</td><td align="center">1078.1</td><td align="center">7.3</td><td align="center">4.7%</td><td>contig_8, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_9.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>contig_9</td><td align="right">208,537</td><td align="center">1152.9</td><td align="center">9.6</td><td align="center">4.2%</td><td>contig_9, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_10.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>contig_10</td><td align="right">190,808</td><td align="center">1139.7</td><td align="center">6.9</td><td align="center">3.6%</td><td>contig_10, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_11.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>contig_11</td><td align="right">182,365</td><td align="center">1162.2</td><td align="center">9.5</td><td align="center">3.6%</td><td>contig_11, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_12.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>contig_12</td><td align="right">181,444</td><td align="center">1157.2</td><td align="center">8.5</td><td align="center">3.5%</td><td>contig_12, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_13.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>contig_13</td><td align="right">136,360</td><td align="center">1044.7</td><td align="center">7.8</td><td align="center">2.4%</td><td>contig_13, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_14.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>contig_14</td><td align="right">123,655</td><td align="center">1155.8</td><td align="center">6.6</td><td align="center">2.4%</td><td>contig_14, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_15.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>contig_15</td><td align="right">108,998</td><td align="center">1164.0</td><td align="center">6.5</td><td align="center">2.1%</td><td>contig_15, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_16.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>contig_16</td><td align="right">104,226</td><td align="center">1153.1</td><td align="center">7.5</td><td align="center">2.0%</td><td>contig_16, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_17.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>contig_17</td><td align="right">77,561</td><td align="center">1125.4</td><td align="center">8.1</td><td align="center">1.5%</td><td>contig_17, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_18.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>contig_18</td><td align="right">75,010</td><td align="center">1047.1</td><td align="center">6.4</td><td align="center">1.3%</td><td>contig_18, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_19.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>contig_19</td><td align="right">74,357</td><td align="center">1097.8</td><td align="center">10.1</td><td align="center">1.4%</td><td>contig_19, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_20.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>contig_20</td><td align="right">41,271</td><td align="center">1208.2</td><td align="center">9.0</td><td align="center">0.8%</td><td>contig_20, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_21.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>contig_21</td><td align="right">29,666</td><td align="center">1174.7</td><td align="center">10.4</td><td align="center">0.6%</td><td>contig_21, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_22.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>contig_22</td><td align="right">24,321</td><td align="center">2019.7</td><td align="center">17.9</td><td align="center">0.8%</td><td>contig_22, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_23.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>contig_23</td><td align="right">17,301</td><td align="center">3078.3</td><td align="center">21.9</td><td align="center">0.9%</td><td>contig_23, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_24.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>contig_24</td><td align="right">15,935</td><td align="center">1130.8</td><td align="center">9.2</td><td align="center">0.3%</td><td>contig_24, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_25.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>contig_25</td><td align="right">15,179</td><td align="center">1224.4</td><td align="center">6.9</td><td align="center">0.3%</td><td>contig_25, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_26.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>contig_26</td><td align="right">13,155</td><td align="center">1202.2</td><td align="center">7.8</td><td align="center">0.3%</td><td>contig_26, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_27.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>contig_27</td><td align="right">11,555</td><td align="center">1203.6</td><td align="center">17.8</td><td align="center">0.2%</td><td>contig_27, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_28.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>contig_28</td><td align="right">11,150</td><td align="center">1829.9</td><td align="center">7.6</td><td align="center">0.3%</td><td>contig_28, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_29.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>contig_29</td><td align="right">10,906</td><td align="center">1180.9</td><td align="center">12.0</td><td align="center">0.2%</td><td>contig_29, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_30.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>contig_30</td><td align="right">10,543</td><td align="center">1919.5</td><td align="center">14.9</td><td align="center">0.3%</td><td>contig_30, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_31.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>contig_31</td><td align="right">8,249</td><td align="center">1251.2</td><td align="center">5.9</td><td align="center">0.2%</td><td>contig_31, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_32.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>contig_32</td><td align="right">7,502</td><td align="center">1749.6</td><td align="center">43.1</td><td align="center">0.2%</td><td>contig_32, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_33.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>contig_33</td><td align="right">7,461</td><td align="center">1244.2</td><td align="center">7.0</td><td align="center">0.2%</td><td>contig_33, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_34.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>contig_34</td><td align="right">6,696</td><td align="center">1843.7</td><td align="center">16.5</td><td align="center">0.2%</td><td>contig_34, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_35.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>contig_35</td><td align="right">6,145</td><td align="center">1174.6</td><td align="center">14.1</td><td align="center">0.1%</td><td>contig_35, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_36.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>contig_36</td><td align="right">4,610</td><td align="center">3111.9</td><td align="center">36.0</td><td align="center">0.3%</td><td>contig_36, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_37.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>contig_37</td><td align="right">4,115</td><td align="center">1228.5</td><td align="center">6.9</td><td align="center">0.1%</td><td>contig_37, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_38.overview.png">coverage</a></td><td><a href="calibration/37.unique_coverage.pdf">distribution</a></td><td>contig_38</td><td align="right">3,973</td><td align="center">2676.6</td><td align="center">19.6</td><td align="center">0.2%</td><td>contig_38, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_39.overview.png">coverage</a></td><td><a href="calibration/38.unique_coverage.pdf">distribution</a></td><td>contig_39</td><td align="right">3,953</td><td align="center">2273.6</td><td align="center">3.0</td><td align="center">0.2%</td><td>contig_39, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_40.overview.png">coverage</a></td><td><a href="calibration/39.unique_coverage.pdf">distribution</a></td><td>contig_40</td><td align="right">3,341</td><td align="center">5088.7</td><td align="center">9.3</td><td align="center">0.3%</td><td>contig_40, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_41.overview.png">coverage</a></td><td><a href="calibration/40.unique_coverage.pdf">distribution</a></td><td>contig_41</td><td align="right">3,054</td><td align="center">9170.1</td><td align="center">18.7</td><td align="center">0.5%</td><td>contig_41, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_42.overview.png">coverage</a></td><td><a href="calibration/41.unique_coverage.pdf">distribution</a></td><td>contig_42</td><td align="right">3,025</td><td align="center">5629.5</td><td align="center">13.1</td><td align="center">0.3%</td><td>contig_42, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_43.overview.png">coverage</a></td><td><a href="calibration/42.unique_coverage.pdf">distribution</a></td><td>contig_43</td><td align="right">2,866</td><td align="center">1203.5</td><td align="center">7.0</td><td align="center">0.1%</td><td>contig_43, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_44.overview.png">coverage</a></td><td><a href="calibration/43.unique_coverage.pdf">distribution</a></td><td>contig_44</td><td align="right">2,823</td><td align="center">5297.1</td><td align="center">13.1</td><td align="center">0.2%</td><td>contig_44, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_45.overview.png">coverage</a></td><td><a href="calibration/44.unique_coverage.pdf">distribution</a></td><td>contig_45</td><td align="right">2,592</td><td align="center">1247.6</td><td align="center">33.5</td><td align="center">0.1%</td><td>contig_45, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_46.overview.png">coverage</a></td><td><a href="calibration/45.unique_coverage.pdf">distribution</a></td><td>contig_46</td><td align="right">2,365</td><td align="center">1241.1</td><td align="center">8.6</td><td align="center">0.1%</td><td>contig_46, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_47.overview.png">coverage</a></td><td><a href="calibration/46.unique_coverage.pdf">distribution</a></td><td>contig_47</td><td align="right">2,021</td><td align="center">5790.8</td><td align="center">57.0</td><td align="center">0.2%</td><td>contig_47, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_48.overview.png">coverage</a></td><td><a href="calibration/47.unique_coverage.pdf">distribution</a></td><td>contig_48</td><td align="right">1,927</td><td align="center">4364.7</td><td align="center">37.2</td><td align="center">0.1%</td><td>contig_48, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_49.overview.png">coverage</a></td><td><a href="calibration/48.unique_coverage.pdf">distribution</a></td><td>contig_49</td><td align="right">1,729</td><td align="center">5694.0</td><td align="center">79.4</td><td align="center">0.1%</td><td>contig_49, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_50.overview.png">coverage</a></td><td><a href="calibration/49.unique_coverage.pdf">distribution</a></td><td>contig_50</td><td align="right">1,669</td><td align="center">3545.4</td><td align="center">19.5</td><td align="center">0.1%</td><td>contig_50, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_51.overview.png">coverage</a></td><td><a href="calibration/50.unique_coverage.pdf">distribution</a></td><td>contig_51</td><td align="right">1,650</td><td align="center">5495.5</td><td align="center">15.1</td><td align="center">0.1%</td><td>contig_51, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_52.overview.png">coverage</a></td><td><a href="calibration/51.unique_coverage.pdf">distribution</a></td><td>contig_52</td><td align="right">1,615</td><td align="center">2164.3</td><td align="center">4.4</td><td align="center">0.1%</td><td>contig_52, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_53.overview.png">coverage</a></td><td><a href="calibration/52.unique_coverage.pdf">distribution</a></td><td>contig_53</td><td align="right">1,613</td><td align="center">8970.4</td><td align="center">23.8</td><td align="center">0.2%</td><td>contig_53, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_54.overview.png">coverage</a></td><td><a href="calibration/53.unique_coverage.pdf">distribution</a></td><td>contig_54</td><td align="right">1,558</td><td align="center">1872.8</td><td align="center">8.1</td><td align="center">0.0%</td><td>contig_54, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_55.overview.png">coverage</a></td><td><a href="calibration/54.unique_coverage.pdf">distribution</a></td><td>contig_55</td><td align="right">1,531</td><td align="center">3159.9</td><td align="center">39.8</td><td align="center">0.1%</td><td>contig_55, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_56.overview.png">coverage</a></td><td><a href="calibration/55.unique_coverage.pdf">distribution</a></td><td>contig_56</td><td align="right">1,451</td><td align="center">1173.4</td><td align="center">16.4</td><td align="center">0.0%</td><td>contig_56, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_57.overview.png">coverage</a></td><td><a href="calibration/56.unique_coverage.pdf">distribution</a></td><td>contig_57</td><td align="right">1,432</td><td align="center">4535.9</td><td align="center">17.6</td><td align="center">0.1%</td><td>contig_57, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_58.overview.png">coverage</a></td><td><a href="calibration/57.unique_coverage.pdf">distribution</a></td><td>contig_58</td><td align="right">1,384</td><td align="center">4084.7</td><td align="center">34.8</td><td align="center">0.1%</td><td>contig_58, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_59.overview.png">coverage</a></td><td><a href="calibration/58.unique_coverage.pdf">distribution</a></td><td>contig_59</td><td align="right">1,383</td><td align="center">1087.5</td><td align="center">2.7</td><td align="center">0.0%</td><td>contig_59, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_60.overview.png">coverage</a></td><td><a href="calibration/59.unique_coverage.pdf">distribution</a></td><td>contig_60</td><td align="right">1,331</td><td align="center">1219.4</td><td align="center">3.6</td><td align="center">0.0%</td><td>contig_60, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_61.overview.png">coverage</a></td><td><a href="calibration/60.unique_coverage.pdf">distribution</a></td><td>contig_61</td><td align="right">1,259</td><td align="center">1125.9</td><td align="center">4.1</td><td align="center">0.0%</td><td>contig_61, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_62.overview.png">coverage</a></td><td><a href="calibration/61.unique_coverage.pdf">distribution</a></td><td>contig_62</td><td align="right">1,168</td><td align="center">5128.0</td><td align="center">13.9</td><td align="center">0.1%</td><td>contig_62, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_63.overview.png">coverage</a></td><td><a href="calibration/62.unique_coverage.pdf">distribution</a></td><td>contig_63</td><td align="right">1,097</td><td align="center">6374.5</td><td align="center">55.6</td><td align="center">0.1%</td><td>contig_63, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_64.overview.png">coverage</a></td><td><a href="calibration/63.unique_coverage.pdf">distribution</a></td><td>contig_64</td><td align="right">1,074</td><td align="center">1386.7</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_64, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_65.overview.png">coverage</a></td><td><a href="calibration/64.unique_coverage.pdf">distribution</a></td><td>contig_65</td><td align="right">1,071</td><td align="center">5294.7</td><td align="center">15.6</td><td align="center">0.1%</td><td>contig_65, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_66.overview.png">coverage</a></td><td><a href="calibration/65.unique_coverage.pdf">distribution</a></td><td>contig_66</td><td align="right">1,031</td><td align="center">2424.0</td><td align="center">6.9</td><td align="center">0.0%</td><td>contig_66, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_67.overview.png">coverage</a></td><td><a href="calibration/66.unique_coverage.pdf">distribution</a></td><td>contig_67</td><td align="right">1,030</td><td align="center">2215.9</td><td align="center">2.9</td><td align="center">0.0%</td><td>contig_67, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_68.overview.png">coverage</a></td><td><a href="calibration/67.unique_coverage.pdf">distribution</a></td><td>contig_68</td><td align="right">1,003</td><td align="center">3392.4</td><td align="center">6.5</td><td align="center">0.0%</td><td>contig_68, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_69.overview.png">coverage</a></td><td><a href="calibration/68.unique_coverage.pdf">distribution</a></td><td>contig_69</td><td align="right">973</td><td align="center">2998.0</td><td align="center">4.3</td><td align="center">0.0%</td><td>contig_69, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_70.overview.png">coverage</a></td><td><a href="calibration/69.unique_coverage.pdf">distribution</a></td><td>contig_70</td><td align="right">978</td><td align="center">2378.1</td><td align="center">65.0</td><td align="center">0.0%</td><td>contig_70, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_71.overview.png">coverage</a></td><td><a href="calibration/70.unique_coverage.pdf">distribution</a></td><td>contig_71</td><td align="right">967</td><td align="center">5426.7</td><td align="center">13.3</td><td align="center">0.1%</td><td>contig_71, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_72.overview.png">coverage</a></td><td><a href="calibration/71.unique_coverage.pdf">distribution</a></td><td>contig_72</td><td align="right">952</td><td align="center">5242.2</td><td align="center">3.6</td><td align="center">0.1%</td><td>contig_72, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_73.overview.png">coverage</a></td><td><a href="calibration/72.unique_coverage.pdf">distribution</a></td><td>contig_73</td><td align="right">933</td><td align="center">2461.4</td><td align="center">2.5</td><td align="center">0.0%</td><td>contig_73, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_74.overview.png">coverage</a></td><td><a href="calibration/73.unique_coverage.pdf">distribution</a></td><td>contig_74</td><td align="right">899</td><td align="center">[2145.3]</td><td align="center">[590.2]</td><td align="center">0.0%</td><td>contig_74, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_75.overview.png">coverage</a></td><td><a href="calibration/74.unique_coverage.pdf">distribution</a></td><td>contig_75</td><td align="right">899</td><td align="center">1500.6</td><td align="center">6.2</td><td align="center">0.0%</td><td>contig_75, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_76.overview.png">coverage</a></td><td><a href="calibration/75.unique_coverage.pdf">distribution</a></td><td>contig_76</td><td align="right">902</td><td align="center">[747.2]</td><td align="center">[386.1]</td><td align="center">0.0%</td><td>contig_76, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_77.overview.png">coverage</a></td><td><a href="calibration/76.unique_coverage.pdf">distribution</a></td><td>contig_77</td><td align="right">894</td><td align="center">2635.1</td><td align="center">2.2</td><td align="center">0.0%</td><td>contig_77, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_78.overview.png">coverage</a></td><td><a href="calibration/77.unique_coverage.pdf">distribution</a></td><td>contig_78</td><td align="right">864</td><td align="center">3403.3</td><td align="center">6.4</td><td align="center">0.0%</td><td>contig_78, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_79.overview.png">coverage</a></td><td><a href="calibration/78.unique_coverage.pdf">distribution</a></td><td>contig_79</td><td align="right">864</td><td align="center">2011.4</td><td align="center">1.6</td><td align="center">0.0%</td><td>contig_79, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_80.overview.png">coverage</a></td><td><a href="calibration/79.unique_coverage.pdf">distribution</a></td><td>contig_80</td><td align="right">843</td><td align="center">774.9</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_80, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_81.overview.png">coverage</a></td><td><a href="calibration/80.unique_coverage.pdf">distribution</a></td><td>contig_81</td><td align="right">838</td><td align="center">4131.2</td><td align="center">166.5</td><td align="center">0.0%</td><td>contig_81, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_82.overview.png">coverage</a></td><td><a href="calibration/81.unique_coverage.pdf">distribution</a></td><td>contig_82</td><td align="right">836</td><td align="center">2081.9</td><td align="center">12.9</td><td align="center">0.0%</td><td>contig_82, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_83.overview.png">coverage</a></td><td><a href="calibration/82.unique_coverage.pdf">distribution</a></td><td>contig_83</td><td align="right">781</td><td align="center">2382.3</td><td align="center">9.0</td><td align="center">0.0%</td><td>contig_83, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_84.overview.png">coverage</a></td><td><a href="calibration/83.unique_coverage.pdf">distribution</a></td><td>contig_84</td><td align="right">777</td><td align="center">4348.0</td><td align="center">3.8</td><td align="center">0.0%</td><td>contig_84, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_85.overview.png">coverage</a></td><td><a href="calibration/84.unique_coverage.pdf">distribution</a></td><td>contig_85</td><td align="right">768</td><td align="center">2568.9</td><td align="center">4.6</td><td align="center">0.0%</td><td>contig_85, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_86.overview.png">coverage</a></td><td><a href="calibration/85.unique_coverage.pdf">distribution</a></td><td>contig_86</td><td align="right">764</td><td align="center">1350.7</td><td align="center">3.6</td><td align="center">0.0%</td><td>contig_86, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_87.overview.png">coverage</a></td><td><a href="calibration/86.unique_coverage.pdf">distribution</a></td><td>contig_87</td><td align="right">750</td><td align="center">4864.2</td><td align="center">9.3</td><td align="center">0.0%</td><td>contig_87, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_88.overview.png">coverage</a></td><td><a href="calibration/87.unique_coverage.pdf">distribution</a></td><td>contig_88</td><td align="right">743</td><td align="center">1258.7</td><td align="center">4.5</td><td align="center">0.0%</td><td>contig_88, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_89.overview.png">coverage</a></td><td><a href="calibration/88.unique_coverage.pdf">distribution</a></td><td>contig_89</td><td align="right">733</td><td align="center">1280.0</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_89, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_90.overview.png">coverage</a></td><td><a href="calibration/89.unique_coverage.pdf">distribution</a></td><td>contig_90</td><td align="right">689</td><td align="center">2431.3</td><td align="center">2.7</td><td align="center">0.0%</td><td>contig_90, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_91.overview.png">coverage</a></td><td><a href="calibration/90.unique_coverage.pdf">distribution</a></td><td>contig_91</td><td align="right">681</td><td align="center">1314.3</td><td align="center">2.5</td><td align="center">0.0%</td><td>contig_91, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_92.overview.png">coverage</a></td><td><a href="calibration/91.unique_coverage.pdf">distribution</a></td><td>contig_92</td><td align="right">668</td><td align="center">5891.6</td><td align="center">61.1</td><td align="center">0.0%</td><td>contig_92, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_93.overview.png">coverage</a></td><td><a href="calibration/92.unique_coverage.pdf">distribution</a></td><td>contig_93</td><td align="right">658</td><td align="center">1227.7</td><td align="center">5.3</td><td align="center">0.0%</td><td>contig_93, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_94.overview.png">coverage</a></td><td><a href="calibration/93.unique_coverage.pdf">distribution</a></td><td>contig_94</td><td align="right">638</td><td align="center">3233.7</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_94, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_95.overview.png">coverage</a></td><td><a href="calibration/94.unique_coverage.pdf">distribution</a></td><td>contig_95</td><td align="right">638</td><td align="center">1998.6</td><td align="center">2.0</td><td align="center">0.0%</td><td>contig_95, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_96.overview.png">coverage</a></td><td><a href="calibration/95.unique_coverage.pdf">distribution</a></td><td>contig_96</td><td align="right">612</td><td align="center">1766.6</td><td align="center">2.4</td><td align="center">0.0%</td><td>contig_96, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_97.overview.png">coverage</a></td><td><a href="calibration/96.unique_coverage.pdf">distribution</a></td><td>contig_97</td><td align="right">605</td><td align="center">2922.3</td><td align="center">9.1</td><td align="center">0.0%</td><td>contig_97, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_98.overview.png">coverage</a></td><td><a href="calibration/97.unique_coverage.pdf">distribution</a></td><td>contig_98</td><td align="right">588</td><td align="center">4870.3</td><td align="center">2.5</td><td align="center">0.0%</td><td>contig_98, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_99.overview.png">coverage</a></td><td><a href="calibration/98.unique_coverage.pdf">distribution</a></td><td>contig_99</td><td align="right">582</td><td align="center">1753.2</td><td align="center">1.1</td><td align="center">0.0%</td><td>contig_99, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_100.overview.png">coverage</a></td><td><a href="calibration/99.unique_coverage.pdf">distribution</a></td><td>contig_100</td><td align="right">582</td><td align="center">2939.0</td><td align="center">13.8</td><td align="center">0.0%</td><td>contig_100, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_101.overview.png">coverage</a></td><td><a href="calibration/100.unique_coverage.pdf">distribution</a></td><td>contig_101</td><td align="right">574</td><td align="center">4242.1</td><td align="center">2.6</td><td align="center">0.0%</td><td>contig_101, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_102.overview.png">coverage</a></td><td><a href="calibration/101.unique_coverage.pdf">distribution</a></td><td>contig_102</td><td align="right">572</td><td align="center">1265.9</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_102, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_103.overview.png">coverage</a></td><td><a href="calibration/102.unique_coverage.pdf">distribution</a></td><td>contig_103</td><td align="right">571</td><td align="center">[606.5]</td><td align="center">[208.8]</td><td align="center">0.0%</td><td>contig_103, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_104.overview.png">coverage</a></td><td><a href="calibration/103.unique_coverage.pdf">distribution</a></td><td>contig_104</td><td align="right">544</td><td align="center">2404.1</td><td align="center">5.6</td><td align="center">0.0%</td><td>contig_104, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_105.overview.png">coverage</a></td><td><a href="calibration/104.unique_coverage.pdf">distribution</a></td><td>contig_105</td><td align="right">520</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>contig_105, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_106.overview.png">coverage</a></td><td><a href="calibration/105.unique_coverage.pdf">distribution</a></td><td>contig_106</td><td align="right">511</td><td align="center">1149.4</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_106, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_107.overview.png">coverage</a></td><td><a href="calibration/106.unique_coverage.pdf">distribution</a></td><td>contig_107</td><td align="right">505</td><td align="center">1931.1</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_107, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_108.overview.png">coverage</a></td><td><a href="calibration/107.unique_coverage.pdf">distribution</a></td><td>contig_108</td><td align="right">464</td><td align="center">5664.2</td><td align="center">1.9</td><td align="center">0.0%</td><td>contig_108, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_109.overview.png">coverage</a></td><td><a href="calibration/108.unique_coverage.pdf">distribution</a></td><td>contig_109</td><td align="right">457</td><td align="center">927.9</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_109, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_110.overview.png">coverage</a></td><td><a href="calibration/109.unique_coverage.pdf">distribution</a></td><td>contig_110</td><td align="right">455</td><td align="center">3281.7</td><td align="center">2.6</td><td align="center">0.0%</td><td>contig_110, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_111.overview.png">coverage</a></td><td><a href="calibration/110.unique_coverage.pdf">distribution</a></td><td>contig_111</td><td align="right">451</td><td align="center">1833.8</td><td align="center">5.9</td><td align="center">0.0%</td><td>contig_111, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_112.overview.png">coverage</a></td><td><a href="calibration/111.unique_coverage.pdf">distribution</a></td><td>contig_112</td><td align="right">448</td><td align="center">2188.5</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_112, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_113.overview.png">coverage</a></td><td><a href="calibration/112.unique_coverage.pdf">distribution</a></td><td>contig_113</td><td align="right">448</td><td align="center">3283.7</td><td align="center">1.4</td><td align="center">0.0%</td><td>contig_113, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_114.overview.png">coverage</a></td><td><a href="calibration/113.unique_coverage.pdf">distribution</a></td><td>contig_114</td><td align="right">440</td><td align="center">3203.2</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_114, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_115.overview.png">coverage</a></td><td><a href="calibration/114.unique_coverage.pdf">distribution</a></td><td>contig_115</td><td align="right">426</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>contig_115, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_116.overview.png">coverage</a></td><td><a href="calibration/115.unique_coverage.pdf">distribution</a></td><td>contig_116</td><td align="right">417</td><td align="center">1082.4</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_116, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_117.overview.png">coverage</a></td><td><a href="calibration/116.unique_coverage.pdf">distribution</a></td><td>contig_117</td><td align="right">400</td><td align="center">2868.4</td><td align="center">2.2</td><td align="center">0.0%</td><td>contig_117, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_118.overview.png">coverage</a></td><td><a href="calibration/117.unique_coverage.pdf">distribution</a></td><td>contig_118</td><td align="right">392</td><td align="center">2425.7</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_118, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_119.overview.png">coverage</a></td><td><a href="calibration/118.unique_coverage.pdf">distribution</a></td><td>contig_119</td><td align="right">387</td><td align="center">2411.0</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_119, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_120.overview.png">coverage</a></td><td><a href="calibration/119.unique_coverage.pdf">distribution</a></td><td>contig_120</td><td align="right">385</td><td align="center">3555.4</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_120, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_121.overview.png">coverage</a></td><td><a href="calibration/120.unique_coverage.pdf">distribution</a></td><td>contig_121</td><td align="right">381</td><td align="center">[1236.1]</td><td align="center">[244.8]</td><td align="center">0.0%</td><td>contig_121, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_122.overview.png">coverage</a></td><td><a href="calibration/121.unique_coverage.pdf">distribution</a></td><td>contig_122</td><td align="right">381</td><td align="center">[1871.2]</td><td align="center">[367.0]</td><td align="center">0.0%</td><td>contig_122, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_123.overview.png">coverage</a></td><td><a href="calibration/122.unique_coverage.pdf">distribution</a></td><td>contig_123</td><td align="right">378</td><td align="center">5298.7</td><td align="center">1.2</td><td align="center">0.0%</td><td>contig_123, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_124.overview.png">coverage</a></td><td><a href="calibration/123.unique_coverage.pdf">distribution</a></td><td>contig_124</td><td align="right">367</td><td align="center">1613.2</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_124, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_125.overview.png">coverage</a></td><td><a href="calibration/124.unique_coverage.pdf">distribution</a></td><td>contig_125</td><td align="right">367</td><td align="center">2524.0</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_125, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_126.overview.png">coverage</a></td><td><a href="calibration/125.unique_coverage.pdf">distribution</a></td><td>contig_126</td><td align="right">364</td><td align="center">[5016.2]</td><td align="center">[234.1]</td><td align="center">0.0%</td><td>contig_126, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_127.overview.png">coverage</a></td><td><a href="calibration/126.unique_coverage.pdf">distribution</a></td><td>contig_127</td><td align="right">360</td><td align="center">1220.1</td><td align="center">2.0</td><td align="center">0.0%</td><td>contig_127, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_128.overview.png">coverage</a></td><td><a href="calibration/127.unique_coverage.pdf">distribution</a></td><td>contig_128</td><td align="right">358</td><td align="center">[1890.3]</td><td align="center">[0.1]</td><td align="center">0.0%</td><td>contig_128, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_129.overview.png">coverage</a></td><td><a href="calibration/128.unique_coverage.pdf">distribution</a></td><td>contig_129</td><td align="right">358</td><td align="center">[3050.7]</td><td align="center">[0.5]</td><td align="center">0.0%</td><td>contig_129, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_130.overview.png">coverage</a></td><td><a href="calibration/129.unique_coverage.pdf">distribution</a></td><td>contig_130</td><td align="right">332</td><td align="center">[1160.9]</td><td align="center">[0.4]</td><td align="center">0.0%</td><td>contig_130, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_131.overview.png">coverage</a></td><td><a href="calibration/130.unique_coverage.pdf">distribution</a></td><td>contig_131</td><td align="right">331</td><td align="center">1985.7</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_131, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_132.overview.png">coverage</a></td><td><a href="calibration/131.unique_coverage.pdf">distribution</a></td><td>contig_132</td><td align="right">331</td><td align="center">3220.9</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_132, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_133.overview.png">coverage</a></td><td><a href="calibration/132.unique_coverage.pdf">distribution</a></td><td>contig_133</td><td align="right">331</td><td align="center">1098.9</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_133, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_134.overview.png">coverage</a></td><td><a href="calibration/133.unique_coverage.pdf">distribution</a></td><td>contig_134</td><td align="right">325</td><td align="center">3224.3</td><td align="center">1.3</td><td align="center">0.0%</td><td>contig_134, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_135.overview.png">coverage</a></td><td><a href="calibration/134.unique_coverage.pdf">distribution</a></td><td>contig_135</td><td align="right">325</td><td align="center">2008.5</td><td align="center">2.1</td><td align="center">0.0%</td><td>contig_135, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_136.overview.png">coverage</a></td><td><a href="calibration/135.unique_coverage.pdf">distribution</a></td><td>contig_136</td><td align="right">319</td><td align="center">[2930.3]</td><td align="center">[343.9]</td><td align="center">0.0%</td><td>contig_136, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_137.overview.png">coverage</a></td><td><a href="calibration/136.unique_coverage.pdf">distribution</a></td><td>contig_137</td><td align="right">311</td><td align="center">1243.1</td><td align="center">166.7</td><td align="center">0.0%</td><td>contig_137, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_138.overview.png">coverage</a></td><td><a href="calibration/137.unique_coverage.pdf">distribution</a></td><td>contig_138</td><td align="right">311</td><td align="center">378.9</td><td align="center">11.2</td><td align="center">0.0%</td><td>contig_138, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_139.overview.png">coverage</a></td><td><a href="calibration/138.unique_coverage.pdf">distribution</a></td><td>contig_139</td><td align="right">306</td><td align="center">2095.1</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_139, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_140.overview.png">coverage</a></td><td><a href="calibration/139.unique_coverage.pdf">distribution</a></td><td>contig_140</td><td align="right">304</td><td align="center">2708.3</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_140, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_141.overview.png">coverage</a></td><td><a href="calibration/140.unique_coverage.pdf">distribution</a></td><td>contig_141</td><td align="right">304</td><td align="center">1645.7</td><td align="center">1.0</td><td align="center">0.0%</td><td>contig_141, whole genome shotgun sequence.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>5,092,795</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit relative_variance</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="reject_table_row">Fit failed</span> Negative binomial fit failed for this reference sequence. It may have an unusual coverage depth distribution. JC and MC predictions may be less accurate.
<p><span class="gray_table_row">Insufficient coverage</span> Reference sequence counted as entirely deleted due to low coverage. Try either the <code>-t,--targeted-sequencing</code> or the <code>-c,--contig-reference </code> option if you want mutations called for these reference sequences.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">100001</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">382</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.023</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>contig_1</td><td align="center">0.11370</td></tr><tr >
<td>contig_2</td><td align="center">0.10748</td></tr><tr >
<td>contig_3</td><td align="center">0.10610</td></tr><tr >
<td>contig_4</td><td align="center">0.11216</td></tr><tr >
<td>contig_5</td><td align="center">0.12173</td></tr><tr >
<td>contig_6</td><td align="center">0.10870</td></tr><tr >
<td>contig_7</td><td align="center">0.10447</td></tr><tr >
<td>contig_8</td><td align="center">0.11678</td></tr><tr >
<td>contig_9</td><td align="center">0.10154</td></tr><tr >
<td>contig_10</td><td align="center">0.10526</td></tr><tr >
<td>contig_11</td><td align="center">0.10194</td></tr><tr >
<td>contig_12</td><td align="center">0.10414</td></tr><tr >
<td>contig_13</td><td align="center">0.12228</td></tr><tr >
<td>contig_14</td><td align="center">0.10261</td></tr><tr >
<td>contig_15</td><td align="center">0.10270</td></tr><tr >
<td>contig_16</td><td align="center">0.10102</td></tr><tr >
<td>contig_17</td><td align="center">0.11078</td></tr><tr >
<td>contig_18</td><td align="center">0.12098</td></tr><tr >
<td>contig_19</td><td align="center">0.11287</td></tr><tr >
<td>contig_20</td><td align="center">0.09850</td></tr><tr >
<td>contig_21</td><td align="center">0.09958</td></tr><tr >
<td>contig_22</td><td align="center">0.03839</td></tr><tr >
<td>contig_23</td><td align="center">0.01674</td></tr><tr >
<td>contig_24</td><td align="center">0.10487</td></tr><tr >
<td>contig_25</td><td align="center">0.09570</td></tr><tr >
<td>contig_26</td><td align="center">0.09988</td></tr><tr >
<td>contig_27</td><td align="center">0.10991</td></tr><tr >
<td>contig_28</td><td align="center">0.05238</td></tr><tr >
<td>contig_29</td><td align="center">0.11484</td></tr><tr >
<td>contig_30</td><td align="center">0.04852</td></tr><tr >
<td>contig_31</td><td align="center">0.09324</td></tr><tr >
<td>contig_32</td><td align="center">0.07398</td></tr><tr >
<td>contig_33</td><td align="center">0.10919</td></tr><tr >
<td>contig_34</td><td align="center">0.04532</td></tr><tr >
<td>contig_35</td><td align="center">0.12136</td></tr><tr >
<td>contig_36</td><td align="center">0.05960</td></tr><tr >
<td>contig_37</td><td align="center">0.12199</td></tr><tr >
<td>contig_38</td><td align="center">0.06023</td></tr><tr >
<td>contig_39</td><td align="center">0.01702</td></tr><tr >
<td>contig_40</td><td align="center">0.03936</td></tr><tr >
<td>contig_41</td><td align="center">0.03635</td></tr><tr >
<td>contig_42</td><td align="center">0.04165</td></tr><tr >
<td>contig_43</td><td align="center">0.13936</td></tr><tr >
<td>contig_44</td><td align="center">0.04339</td></tr><tr >
<td>contig_45</td><td align="center">0.08631</td></tr><tr >
<td>contig_46</td><td align="center">0.11755</td></tr><tr >
<td>contig_47</td><td align="center">0.06309</td></tr><tr >
<td>contig_48</td><td align="center">0.06694</td></tr><tr >
<td>contig_49</td><td align="center">0.07432</td></tr><tr >
<td>contig_50</td><td align="center">0.07999</td></tr><tr >
<td>contig_51</td><td align="center">0.07606</td></tr><tr >
<td>contig_52</td><td align="center">0.10464</td></tr><tr >
<td>contig_53</td><td align="center">0.06882</td></tr><tr >
<td>contig_54</td><td align="center">0.14482</td></tr><tr >
<td>contig_55</td><td align="center">0.12329</td></tr><tr >
<td>contig_56</td><td align="center">0.37004</td></tr><tr >
<td>contig_57</td><td align="center">0.08624</td></tr><tr >
<td>contig_58</td><td align="center">0.09899</td></tr><tr >
<td>contig_59</td><td align="center">0.10125</td></tr><tr >
<td>contig_60</td><td align="center">0.08283</td></tr><tr >
<td>contig_61</td><td align="center">0.19102</td></tr><tr >
<td>contig_62</td><td align="center">0.10702</td></tr><tr >
<td>contig_63</td><td align="center">0.10529</td></tr><tr >
<td>contig_64</td><td align="center">0.20323</td></tr><tr >
<td>contig_65</td><td align="center">0.11671</td></tr><tr >
<td>contig_66</td><td align="center">0.14646</td></tr><tr >
<td>contig_67</td><td align="center">0.14369</td></tr><tr >
<td>contig_68</td><td align="center">0.13759</td></tr><tr >
<td>contig_69</td><td align="center">0.00995</td></tr><tr >
<td>contig_70</td><td align="center">0.03430</td></tr><tr >
<td>contig_71</td><td align="center">0.12151</td></tr><tr >
<td>contig_72</td><td align="center">0.13761</td></tr><tr >
<td>contig_73</td><td align="center">0.29429</td></tr><tr >
<td>contig_74</td><td align="center">NA</td></tr><tr >
<td>contig_75</td><td align="center">0.70968</td></tr><tr >
<td>contig_76</td><td align="center">NA</td></tr><tr >
<td>contig_77</td><td align="center">0.15492</td></tr><tr >
<td>contig_78</td><td align="center">0.23140</td></tr><tr >
<td>contig_79</td><td align="center">0.26252</td></tr><tr >
<td>contig_80</td><td align="center">1.00000</td></tr><tr >
<td>contig_81</td><td align="center">0.05520</td></tr><tr >
<td>contig_82</td><td align="center">0.07223</td></tr><tr >
<td>contig_83</td><td align="center">0.20230</td></tr><tr >
<td>contig_84</td><td align="center">0.16088</td></tr><tr >
<td>contig_85</td><td align="center">0.17969</td></tr><tr >
<td>contig_86</td><td align="center">0.07593</td></tr><tr >
<td>contig_87</td><td align="center">0.16933</td></tr><tr >
<td>contig_88</td><td align="center">0.25236</td></tr><tr >
<td>contig_89</td><td align="center">0.36394</td></tr><tr >
<td>contig_90</td><td align="center">0.04865</td></tr><tr >
<td>contig_91</td><td align="center">0.34047</td></tr><tr >
<td>contig_92</td><td align="center">0.18039</td></tr><tr >
<td>contig_93</td><td align="center">0.29711</td></tr><tr >
<td>contig_94</td><td align="center">0.36752</td></tr><tr >
<td>contig_95</td><td align="center">0.06838</td></tr><tr >
<td>contig_96</td><td align="center">0.31316</td></tr><tr >
<td>contig_97</td><td align="center">0.29526</td></tr><tr >
<td>contig_98</td><td align="center">0.33031</td></tr><tr >
<td>contig_99</td><td align="center">0.25086</td></tr><tr >
<td>contig_100</td><td align="center">0.23368</td></tr><tr >
<td>contig_101</td><td align="center">0.22125</td></tr><tr >
<td>contig_102</td><td align="center">0.25141</td></tr><tr >
<td>contig_103</td><td align="center">NA</td></tr><tr >
<td>contig_104</td><td align="center">0.24449</td></tr><tr >
<td>contig_105</td><td align="center">NA</td></tr><tr >
<td>contig_106</td><td align="center">0.20080</td></tr><tr >
<td>contig_107</td><td align="center">0.47897</td></tr><tr >
<td>contig_108</td><td align="center">0.25647</td></tr><tr >
<td>contig_109</td><td align="center">0.66667</td></tr><tr >
<td>contig_110</td><td align="center">0.28571</td></tr><tr >
<td>contig_111</td><td align="center">0.31264</td></tr><tr >
<td>contig_112</td><td align="center">0.73864</td></tr><tr >
<td>contig_113</td><td align="center">0.69318</td></tr><tr >
<td>contig_114</td><td align="center">0.18166</td></tr><tr >
<td>contig_115</td><td align="center">NA</td></tr><tr >
<td>contig_116</td><td align="center">0.21320</td></tr><tr >
<td>contig_117</td><td align="center">0.01230</td></tr><tr >
<td>contig_118</td><td align="center">0.34311</td></tr><tr >
<td>contig_119</td><td align="center">0.66667</td></tr><tr >
<td>contig_120</td><td align="center">0.31039</td></tr><tr >
<td>contig_121</td><td align="center">NA</td></tr><tr >
<td>contig_122</td><td align="center">NA</td></tr><tr >
<td>contig_123</td><td align="center">0.32672</td></tr><tr >
<td>contig_124</td><td align="center">0.65909</td></tr><tr >
<td>contig_125</td><td align="center">0.68571</td></tr><tr >
<td>contig_126</td><td align="center">NA</td></tr><tr >
<td>contig_127</td><td align="center">0.25000</td></tr><tr >
<td>contig_128</td><td align="center">NA</td></tr><tr >
<td>contig_129</td><td align="center">NA</td></tr><tr >
<td>contig_130</td><td align="center">NA</td></tr><tr >
<td>contig_131</td><td align="center">0.78947</td></tr><tr >
<td>contig_132</td><td align="center">0.13158</td></tr><tr >
<td>contig_133</td><td align="center">0.13736</td></tr><tr >
<td>contig_134</td><td align="center">0.79365</td></tr><tr >
<td>contig_135</td><td align="center">0.22619</td></tr><tr >
<td>contig_136</td><td align="center">NA</td></tr><tr >
<td>contig_137</td><td align="center">0.96552</td></tr><tr >
<td>contig_138</td><td align="center">0.96552</td></tr><tr >
<td>contig_139</td><td align="center">0.31288</td></tr><tr >
<td>contig_140</td><td align="center">0.23423</td></tr><tr >
<td>contig_141</td><td align="center">0.76577</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.4.5</td></tr>
<tr><td>R</td><td>4.4.0</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>11:20:52&nbsp;24&nbsp;Apr&nbsp;2025</td><td>11:31:01&nbsp;24&nbsp;Apr&nbsp;2025</td><td>10&nbsp;minutes&nbsp;9&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>12:22:50&nbsp;25&nbsp;Apr&nbsp;2025</td><td>13:44:03&nbsp;25&nbsp;Apr&nbsp;2025</td><td>1&nbsp;hour&nbsp;21&nbsp;minutes&nbsp;13&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>13:44:03&nbsp;25&nbsp;Apr&nbsp;2025</td><td>13:50:20&nbsp;25&nbsp;Apr&nbsp;2025</td><td>6&nbsp;minutes&nbsp;17&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>14:19:53&nbsp;25&nbsp;Apr&nbsp;2025</td><td>14:37:42&nbsp;25&nbsp;Apr&nbsp;2025</td><td>17&nbsp;minutes&nbsp;49&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>14:37:42&nbsp;25&nbsp;Apr&nbsp;2025</td><td>14:38:05&nbsp;25&nbsp;Apr&nbsp;2025</td><td>23&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>14:38:05&nbsp;25&nbsp;Apr&nbsp;2025</td><td>15:21:11&nbsp;25&nbsp;Apr&nbsp;2025</td><td>43&nbsp;minutes&nbsp;6&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>15:21:11&nbsp;25&nbsp;Apr&nbsp;2025</td><td>15:32:17&nbsp;25&nbsp;Apr&nbsp;2025</td><td>11&nbsp;minutes&nbsp;6&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>15:32:17&nbsp;25&nbsp;Apr&nbsp;2025</td><td>15:41:20&nbsp;25&nbsp;Apr&nbsp;2025</td><td>9&nbsp;minutes&nbsp;3&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>15:41:20&nbsp;25&nbsp;Apr&nbsp;2025</td><td>15:48:50&nbsp;25&nbsp;Apr&nbsp;2025</td><td>7&nbsp;minutes&nbsp;30&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>15:48:50&nbsp;25&nbsp;Apr&nbsp;2025</td><td>15:52:49&nbsp;25&nbsp;Apr&nbsp;2025</td><td>3&nbsp;minutes&nbsp;59&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>15:52:49&nbsp;25&nbsp;Apr&nbsp;2025</td><td>16:37:11&nbsp;25&nbsp;Apr&nbsp;2025</td><td>44&nbsp;minutes&nbsp;22&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>16:37:11&nbsp;25&nbsp;Apr&nbsp;2025</td><td>16:37:13&nbsp;25&nbsp;Apr&nbsp;2025</td><td>2&nbsp;seconds</td></tr>
<tr><td>Output</td><td>16:37:13&nbsp;25&nbsp;Apr&nbsp;2025</td><td>16:37:58&nbsp;25&nbsp;Apr&nbsp;2025</td><td>45&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>16:37:13&nbsp;25&nbsp;Apr&nbsp;2025</td><td>16:37:20&nbsp;25&nbsp;Apr&nbsp;2025</td><td>7&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>16:37:20&nbsp;25&nbsp;Apr&nbsp;2025</td><td>16:37:20&nbsp;25&nbsp;Apr&nbsp;2025</td><td>0&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>3&nbsp;hours&nbsp;55&nbsp;minutes&nbsp;51&nbsp;seconds</b></td>
</tr>
</table></body></html>