<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.39.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/BTF75T1_R1-trimmed.error_rates.pdf">errors</a></td><td>BTF75T1_R1-trimmed</td><td align="right">11,606,982</td><td align="right">1,746,315,538</td><td align="right">100.0%</td><td align="right">150.5&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">99.7%</td></tr><tr ><td><a href="calibration/BTF75T1_R2-trimmed.error_rates.pdf">errors</a></td><td>BTF75T1_R2-trimmed</td><td align="right">10,675,745</td><td align="right">1,605,698,751</td><td align="right">91.9%</td><td align="right">150.4&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">98.6%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>22,282,727</b></td><td align="right"><b>3,352,014,289</b></td><td align="right">96.0%</td><td align="right">150.4&nbsp;bases</td><td align="right"><b>151</b>&nbsp;bases</td><td align="right">99.2%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit relative_variance</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/contig_1.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>contig_1</td><td align="right">657,091</td><td align="center">624.0</td><td align="center">6.0</td><td align="center">12.2%</td><td>Escherichia coli BTF75T-1 contig_1, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_2.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>contig_2</td><td align="right">622,869</td><td align="center">626.0</td><td align="center">5.1</td><td align="center">11.7%</td><td>Escherichia coli BTF75T-1 contig_2, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_3.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>contig_3</td><td align="right">468,399</td><td align="center">628.1</td><td align="center">5.6</td><td align="center">8.9%</td><td>Escherichia coli BTF75T-1 contig_3, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_4.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>contig_4</td><td align="right">456,761</td><td align="center">634.9</td><td align="center">5.5</td><td align="center">8.7%</td><td>Escherichia coli BTF75T-1 contig_4, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_5.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>contig_5</td><td align="right">417,817</td><td align="center">627.1</td><td align="center">4.8</td><td align="center">7.9%</td><td>Escherichia coli BTF75T-1 contig_5, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_6.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>contig_6</td><td align="right">360,046</td><td align="center">630.6</td><td align="center">5.5</td><td align="center">6.8%</td><td>Escherichia coli BTF75T-1 contig_6, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_7.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>contig_7</td><td align="right">281,785</td><td align="center">628.9</td><td align="center">5.5</td><td align="center">5.4%</td><td>Escherichia coli BTF75T-1 contig_7, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_8.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>contig_8</td><td align="right">281,510</td><td align="center">629.3</td><td align="center">5.4</td><td align="center">5.3%</td><td>Escherichia coli BTF75T-1 contig_8, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_9.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>contig_9</td><td align="right">203,660</td><td align="center">636.7</td><td align="center">5.0</td><td align="center">3.9%</td><td>Escherichia coli BTF75T-1 contig_9, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_10.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>contig_10</td><td align="right">163,934</td><td align="center">627.2</td><td align="center">6.0</td><td align="center">3.1%</td><td>Escherichia coli BTF75T-1 contig_10, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_11.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>contig_11</td><td align="right">151,459</td><td align="center">638.3</td><td align="center">5.8</td><td align="center">2.9%</td><td>Escherichia coli BTF75T-1 contig_11, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_12.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>contig_12</td><td align="right">147,087</td><td align="center">625.6</td><td align="center">6.2</td><td align="center">2.8%</td><td>Escherichia coli BTF75T-1 contig_12, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_13.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>contig_13</td><td align="right">129,068</td><td align="center">635.0</td><td align="center">5.6</td><td align="center">2.5%</td><td>Escherichia coli BTF75T-1 contig_13, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_14.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>contig_14</td><td align="right">112,912</td><td align="center">634.9</td><td align="center">5.0</td><td align="center">2.2%</td><td>Escherichia coli BTF75T-1 contig_14, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_15.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>contig_15</td><td align="right">111,840</td><td align="center">619.7</td><td align="center">4.6</td><td align="center">2.1%</td><td>Escherichia coli BTF75T-1 contig_15, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_16.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>contig_16</td><td align="right">102,326</td><td align="center">632.7</td><td align="center">5.1</td><td align="center">2.0%</td><td>Escherichia coli BTF75T-1 contig_16, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_17.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>contig_17</td><td align="right">83,756</td><td align="center">650.1</td><td align="center">5.3</td><td align="center">1.6%</td><td>Escherichia coli BTF75T-1 contig_17, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_18.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>contig_18</td><td align="right">50,440</td><td align="center">620.2</td><td align="center">5.0</td><td align="center">0.9%</td><td>Escherichia coli BTF75T-1 contig_18, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_19.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>contig_19</td><td align="right">48,222</td><td align="center">636.3</td><td align="center">5.4</td><td align="center">0.9%</td><td>Escherichia coli BTF75T-1 contig_19, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_20.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>contig_20</td><td align="right">41,343</td><td align="center">652.9</td><td align="center">4.1</td><td align="center">0.8%</td><td>Escherichia coli BTF75T-1 contig_20, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_21.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>contig_21</td><td align="right">33,722</td><td align="center">651.0</td><td align="center">4.2</td><td align="center">0.7%</td><td>Escherichia coli BTF75T-1 contig_21, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_22.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>contig_22</td><td align="right">31,939</td><td align="center">623.8</td><td align="center">4.3</td><td align="center">0.6%</td><td>Escherichia coli BTF75T-1 contig_22, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_23.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>contig_23</td><td align="right">30,797</td><td align="center">645.1</td><td align="center">3.7</td><td align="center">0.6%</td><td>Escherichia coli BTF75T-1 contig_23, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_24.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>contig_24</td><td align="right">29,789</td><td align="center">623.7</td><td align="center">6.4</td><td align="center">0.6%</td><td>Escherichia coli BTF75T-1 contig_24, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_25.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>contig_25</td><td align="right">25,338</td><td align="center">1130.5</td><td align="center">10.6</td><td align="center">0.8%</td><td>Escherichia coli BTF75T-1 contig_25, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_26.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>contig_26</td><td align="right">16,528</td><td align="center">628.0</td><td align="center">6.3</td><td align="center">0.3%</td><td>Escherichia coli BTF75T-1 contig_26, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_27.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>contig_27</td><td align="right">12,881</td><td align="center">996.2</td><td align="center">5.8</td><td align="center">0.4%</td><td>Escherichia coli BTF75T-1 contig_27, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_28.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>contig_28</td><td align="right">12,087</td><td align="center">1074.8</td><td align="center">10.7</td><td align="center">0.4%</td><td>Escherichia coli BTF75T-1 contig_28, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_29.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>contig_29</td><td align="right">10,543</td><td align="center">1124.7</td><td align="center">6.5</td><td align="center">0.3%</td><td>Escherichia coli BTF75T-1 contig_29, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_30.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>contig_30</td><td align="right">5,934</td><td align="center">1189.7</td><td align="center">15.9</td><td align="center">0.2%</td><td>Escherichia coli BTF75T-1 contig_30, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_31.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>contig_31</td><td align="right">5,714</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_31, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_32.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>contig_32</td><td align="right">5,476</td><td align="center">988.9</td><td align="center">19.9</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_32, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_33.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>contig_33</td><td align="right">4,677</td><td align="center">1135.5</td><td align="center">11.9</td><td align="center">0.2%</td><td>Escherichia coli BTF75T-1 contig_33, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_34.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>contig_34</td><td align="right">4,610</td><td align="center">242.3</td><td align="center">22.8</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_34, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_35.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>contig_35</td><td align="right">4,173</td><td align="center">1151.2</td><td align="center">4.3</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_35, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_36.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>contig_36</td><td align="right">4,146</td><td align="center">686.5</td><td align="center">3.5</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_36, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_37.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>contig_37</td><td align="right">4,074</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_37, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_38.overview.png">coverage</a></td><td><a href="calibration/37.unique_coverage.pdf">distribution</a></td><td>contig_38</td><td align="right">4,048</td><td align="center">1173.5</td><td align="center">7.5</td><td align="center">0.2%</td><td>Escherichia coli BTF75T-1 contig_38, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_39.overview.png">coverage</a></td><td><a href="calibration/38.unique_coverage.pdf">distribution</a></td><td>contig_39</td><td align="right">4,011</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_39, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_40.overview.png">coverage</a></td><td><a href="calibration/39.unique_coverage.pdf">distribution</a></td><td>contig_40</td><td align="right">3,623</td><td align="center">1146.8</td><td align="center">4.5</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_40, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_41.overview.png">coverage</a></td><td><a href="calibration/40.unique_coverage.pdf">distribution</a></td><td>contig_41</td><td align="right">3,504</td><td align="center">631.3</td><td align="center">1.7</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_41, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_42.overview.png">coverage</a></td><td><a href="calibration/41.unique_coverage.pdf">distribution</a></td><td>contig_42</td><td align="right">3,092</td><td align="center">1104.2</td><td align="center">7.0</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_42, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_43.overview.png">coverage</a></td><td><a href="calibration/42.unique_coverage.pdf">distribution</a></td><td>contig_43</td><td align="right">3,040</td><td align="center">4777.4</td><td align="center">11.0</td><td align="center">0.4%</td><td>Escherichia coli BTF75T-1 contig_43, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_44.overview.png">coverage</a></td><td><a href="calibration/43.unique_coverage.pdf">distribution</a></td><td>contig_44</td><td align="right">2,657</td><td align="center">1111.7</td><td align="center">1.9</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_44, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_45.overview.png">coverage</a></td><td><a href="calibration/44.unique_coverage.pdf">distribution</a></td><td>contig_45</td><td align="right">1,962</td><td align="center">645.8</td><td align="center">10.8</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_45, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_46.overview.png">coverage</a></td><td><a href="calibration/45.unique_coverage.pdf">distribution</a></td><td>contig_46</td><td align="right">1,911</td><td align="center">1155.0</td><td align="center">3.8</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_46, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_47.overview.png">coverage</a></td><td><a href="calibration/46.unique_coverage.pdf">distribution</a></td><td>contig_47</td><td align="right">1,811</td><td align="center">1237.5</td><td align="center">6.4</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_47, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_48.overview.png">coverage</a></td><td><a href="calibration/47.unique_coverage.pdf">distribution</a></td><td>contig_48</td><td align="right">1,764</td><td align="center">1162.8</td><td align="center">3.1</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_48, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_49.overview.png">coverage</a></td><td><a href="calibration/48.unique_coverage.pdf">distribution</a></td><td>contig_49</td><td align="right">1,757</td><td align="center">933.1</td><td align="center">5.5</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_49, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_50.overview.png">coverage</a></td><td><a href="calibration/49.unique_coverage.pdf">distribution</a></td><td>contig_50</td><td align="right">1,705</td><td align="center">1223.1</td><td align="center">3.4</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_50, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_51.overview.png">coverage</a></td><td><a href="calibration/50.unique_coverage.pdf">distribution</a></td><td>contig_51</td><td align="right">1,669</td><td align="center">1870.4</td><td align="center">2.6</td><td align="center">0.1%</td><td>Escherichia coli BTF75T-1 contig_51, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_52.overview.png">coverage</a></td><td><a href="calibration/51.unique_coverage.pdf">distribution</a></td><td>contig_52</td><td align="right">1,599</td><td align="center">1063.1</td><td align="center">1.6</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_52, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_53.overview.png">coverage</a></td><td><a href="calibration/52.unique_coverage.pdf">distribution</a></td><td>contig_53</td><td align="right">1,599</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_53, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_54.overview.png">coverage</a></td><td><a href="calibration/53.unique_coverage.pdf">distribution</a></td><td>contig_54</td><td align="right">1,558</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_54, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_55.overview.png">coverage</a></td><td><a href="calibration/54.unique_coverage.pdf">distribution</a></td><td>contig_55</td><td align="right">1,531</td><td align="center">1166.6</td><td align="center">4.5</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_55, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_56.overview.png">coverage</a></td><td><a href="calibration/55.unique_coverage.pdf">distribution</a></td><td>contig_56</td><td align="right">1,466</td><td align="center">4756.1</td><td align="center">4.7</td><td align="center">0.2%</td><td>Escherichia coli BTF75T-1 contig_56, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_57.overview.png">coverage</a></td><td><a href="calibration/56.unique_coverage.pdf">distribution</a></td><td>contig_57</td><td align="right">1,210</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_57, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_58.overview.png">coverage</a></td><td><a href="calibration/57.unique_coverage.pdf">distribution</a></td><td>contig_58</td><td align="right">1,168</td><td align="center">1132.0</td><td align="center">5.6</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_58, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_59.overview.png">coverage</a></td><td><a href="calibration/58.unique_coverage.pdf">distribution</a></td><td>contig_59</td><td align="right">1,116</td><td align="center">1368.3</td><td align="center">8.5</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_59, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_60.overview.png">coverage</a></td><td><a href="calibration/59.unique_coverage.pdf">distribution</a></td><td>contig_60</td><td align="right">1,114</td><td align="center">1153.0</td><td align="center">4.8</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_60, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_61.overview.png">coverage</a></td><td><a href="calibration/60.unique_coverage.pdf">distribution</a></td><td>contig_61</td><td align="right">952</td><td align="center">1271.0</td><td align="center">4.4</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_61, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_62.overview.png">coverage</a></td><td><a href="calibration/61.unique_coverage.pdf">distribution</a></td><td>contig_62</td><td align="right">932</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_62, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_63.overview.png">coverage</a></td><td><a href="calibration/62.unique_coverage.pdf">distribution</a></td><td>contig_63</td><td align="right">932</td><td align="center">880.4</td><td align="center">12.9</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_63, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_64.overview.png">coverage</a></td><td><a href="calibration/63.unique_coverage.pdf">distribution</a></td><td>contig_64</td><td align="right">905</td><td align="center">1332.7</td><td align="center">6.6</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_64, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_65.overview.png">coverage</a></td><td><a href="calibration/64.unique_coverage.pdf">distribution</a></td><td>contig_65</td><td align="right">892</td><td align="center">1342.7</td><td align="center">3.5</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_65, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_66.overview.png">coverage</a></td><td><a href="calibration/65.unique_coverage.pdf">distribution</a></td><td>contig_66</td><td align="right">860</td><td align="center">709.1</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_66, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_67.overview.png">coverage</a></td><td><a href="calibration/66.unique_coverage.pdf">distribution</a></td><td>contig_67</td><td align="right">838</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_67, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_68.overview.png">coverage</a></td><td><a href="calibration/67.unique_coverage.pdf">distribution</a></td><td>contig_68</td><td align="right">853</td><td align="center">1053.3</td><td align="center">6.3</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_68, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_69.overview.png">coverage</a></td><td><a href="calibration/68.unique_coverage.pdf">distribution</a></td><td>contig_69</td><td align="right">832</td><td align="center">1224.4</td><td align="center">4.9</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_69, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_70.overview.png">coverage</a></td><td><a href="calibration/69.unique_coverage.pdf">distribution</a></td><td>contig_70</td><td align="right">809</td><td align="center">985.3</td><td align="center">1.2</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_70, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_71.overview.png">coverage</a></td><td><a href="calibration/70.unique_coverage.pdf">distribution</a></td><td>contig_71</td><td align="right">809</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_71, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_72.overview.png">coverage</a></td><td><a href="calibration/71.unique_coverage.pdf">distribution</a></td><td>contig_72</td><td align="right">798</td><td align="center">1003.3</td><td align="center">2.2</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_72, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_73.overview.png">coverage</a></td><td><a href="calibration/72.unique_coverage.pdf">distribution</a></td><td>contig_73</td><td align="right">720</td><td align="center">716.7</td><td align="center">2.3</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_73, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_74.overview.png">coverage</a></td><td><a href="calibration/73.unique_coverage.pdf">distribution</a></td><td>contig_74</td><td align="right">717</td><td align="center">711.4</td><td align="center">1.3</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_74, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_75.overview.png">coverage</a></td><td><a href="calibration/74.unique_coverage.pdf">distribution</a></td><td>contig_75</td><td align="right">703</td><td align="center">1143.6</td><td align="center">2.2</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_75, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_76.overview.png">coverage</a></td><td><a href="calibration/75.unique_coverage.pdf">distribution</a></td><td>contig_76</td><td align="right">676</td><td align="center">1318.4</td><td align="center">14.3</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_76, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_77.overview.png">coverage</a></td><td><a href="calibration/76.unique_coverage.pdf">distribution</a></td><td>contig_77</td><td align="right">656</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_77, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_78.overview.png">coverage</a></td><td><a href="calibration/77.unique_coverage.pdf">distribution</a></td><td>contig_78</td><td align="right">622</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_78, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_79.overview.png">coverage</a></td><td><a href="calibration/78.unique_coverage.pdf">distribution</a></td><td>contig_79</td><td align="right">519</td><td align="center">1330.3</td><td align="center">2.0</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_79, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_80.overview.png">coverage</a></td><td><a href="calibration/79.unique_coverage.pdf">distribution</a></td><td>contig_80</td><td align="right">508</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_80, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_81.overview.png">coverage</a></td><td><a href="calibration/80.unique_coverage.pdf">distribution</a></td><td>contig_81</td><td align="right">530</td><td align="center">1082.1</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_81, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_82.overview.png">coverage</a></td><td><a href="calibration/81.unique_coverage.pdf">distribution</a></td><td>contig_82</td><td align="right">505</td><td align="center">1080.9</td><td align="center">1.7</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_82, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_83.overview.png">coverage</a></td><td><a href="calibration/82.unique_coverage.pdf">distribution</a></td><td>contig_83</td><td align="right">499</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_83, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_84.overview.png">coverage</a></td><td><a href="calibration/83.unique_coverage.pdf">distribution</a></td><td>contig_84</td><td align="right">491</td><td align="center">1199.8</td><td align="center">4.2</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_84, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_85.overview.png">coverage</a></td><td><a href="calibration/84.unique_coverage.pdf">distribution</a></td><td>contig_85</td><td align="right">491</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_85, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_86.overview.png">coverage</a></td><td><a href="calibration/85.unique_coverage.pdf">distribution</a></td><td>contig_86</td><td align="right">490</td><td align="center">[1025.4]</td><td align="center">[108.5]</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_86, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_87.overview.png">coverage</a></td><td><a href="calibration/86.unique_coverage.pdf">distribution</a></td><td>contig_87</td><td align="right">462</td><td align="center">974.9</td><td align="center">2.2</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_87, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_88.overview.png">coverage</a></td><td><a href="calibration/87.unique_coverage.pdf">distribution</a></td><td>contig_88</td><td align="right">460</td><td align="center">1232.4</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_88, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_89.overview.png">coverage</a></td><td><a href="calibration/88.unique_coverage.pdf">distribution</a></td><td>contig_89</td><td align="right">460</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_89, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_90.overview.png">coverage</a></td><td><a href="calibration/89.unique_coverage.pdf">distribution</a></td><td>contig_90</td><td align="right">452</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_90, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_91.overview.png">coverage</a></td><td><a href="calibration/90.unique_coverage.pdf">distribution</a></td><td>contig_91</td><td align="right">449</td><td align="center">1074.8</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_91, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_92.overview.png">coverage</a></td><td><a href="calibration/91.unique_coverage.pdf">distribution</a></td><td>contig_92</td><td align="right">448</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_92, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_93.overview.png">coverage</a></td><td><a href="calibration/92.unique_coverage.pdf">distribution</a></td><td>contig_93</td><td align="right">448</td><td align="center">1255.8</td><td align="center">46.3</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_93, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_94.overview.png">coverage</a></td><td><a href="calibration/93.unique_coverage.pdf">distribution</a></td><td>contig_94</td><td align="right">440</td><td align="center">2393.2</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_94, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_95.overview.png">coverage</a></td><td><a href="calibration/94.unique_coverage.pdf">distribution</a></td><td>contig_95</td><td align="right">426</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_95, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_96.overview.png">coverage</a></td><td><a href="calibration/95.unique_coverage.pdf">distribution</a></td><td>contig_96</td><td align="right">422</td><td align="center">810.2</td><td align="center">4.7</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_96, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_97.overview.png">coverage</a></td><td><a href="calibration/96.unique_coverage.pdf">distribution</a></td><td>contig_97</td><td align="right">407</td><td align="center">6772.1</td><td align="center">4565.9</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_97, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_98.overview.png">coverage</a></td><td><a href="calibration/97.unique_coverage.pdf">distribution</a></td><td>contig_98</td><td align="right">405</td><td align="center">1635.1</td><td align="center">35.4</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_98, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_99.overview.png">coverage</a></td><td><a href="calibration/98.unique_coverage.pdf">distribution</a></td><td>contig_99</td><td align="right">400</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_99, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_100.overview.png">coverage</a></td><td><a href="calibration/99.unique_coverage.pdf">distribution</a></td><td>contig_100</td><td align="right">394</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_100, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_101.overview.png">coverage</a></td><td><a href="calibration/100.unique_coverage.pdf">distribution</a></td><td>contig_101</td><td align="right">418</td><td align="center">1182.2</td><td align="center">3.4</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_101, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_102.overview.png">coverage</a></td><td><a href="calibration/101.unique_coverage.pdf">distribution</a></td><td>contig_102</td><td align="right">381</td><td align="center">1116.3</td><td align="center">13.6</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_102, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_103.overview.png">coverage</a></td><td><a href="calibration/102.unique_coverage.pdf">distribution</a></td><td>contig_103</td><td align="right">407</td><td align="center">[110.7]</td><td align="center">[0.0]</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_103, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_104.overview.png">coverage</a></td><td><a href="calibration/103.unique_coverage.pdf">distribution</a></td><td>contig_104</td><td align="right">378</td><td align="center">1118.2</td><td align="center">1.4</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_104, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_105.overview.png">coverage</a></td><td><a href="calibration/104.unique_coverage.pdf">distribution</a></td><td>contig_105</td><td align="right">359</td><td align="center">665.6</td><td align="center">1.6</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_105, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_106.overview.png">coverage</a></td><td><a href="calibration/105.unique_coverage.pdf">distribution</a></td><td>contig_106</td><td align="right">359</td><td align="center">663.5</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_106, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_107.overview.png">coverage</a></td><td><a href="calibration/106.unique_coverage.pdf">distribution</a></td><td>contig_107</td><td align="right">340</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_107, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_108.overview.png">coverage</a></td><td><a href="calibration/107.unique_coverage.pdf">distribution</a></td><td>contig_108</td><td align="right">340</td><td align="center">1138.8</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_108, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_109.overview.png">coverage</a></td><td><a href="calibration/108.unique_coverage.pdf">distribution</a></td><td>contig_109</td><td align="right">336</td><td align="center">1348.5</td><td align="center">1.6</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_109, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_110.overview.png">coverage</a></td><td><a href="calibration/109.unique_coverage.pdf">distribution</a></td><td>contig_110</td><td align="right">324</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_110, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_111.overview.png">coverage</a></td><td><a href="calibration/110.unique_coverage.pdf">distribution</a></td><td>contig_111</td><td align="right">319</td><td align="center">874.1</td><td align="center">6.9</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_111, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_112.overview.png">coverage</a></td><td><a href="calibration/111.unique_coverage.pdf">distribution</a></td><td>contig_112</td><td align="right">313</td><td align="center">2709.9</td><td align="center">8.2</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_112, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_113.overview.png">coverage</a></td><td><a href="calibration/112.unique_coverage.pdf">distribution</a></td><td>contig_113</td><td align="right">309</td><td align="center">711.9</td><td align="center">9.0</td><td align="center">0.0%</td><td>Escherichia coli BTF75T-1 contig_113, whole genome shotgun sequence.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>5,212,863</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit relative_variance</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="reject_table_row">Fit failed</span> Negative binomial fit failed for this reference sequence. It may have an unusual coverage depth distribution. JC and MC predictions may be less accurate.
<p><span class="gray_table_row">Insufficient coverage</span> Reference sequence counted as entirely deleted due to low coverage. Try either the <code>-t,--targeted-sequencing</code> or the <code>-c,--contig-reference </code> option if you want mutations called for these reference sequences.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">100001</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">830</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.049</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>contig_1</td><td align="center">0.32286</td></tr><tr >
<td>contig_2</td><td align="center">0.31829</td></tr><tr >
<td>contig_3</td><td align="center">0.31781</td></tr><tr >
<td>contig_4</td><td align="center">0.31333</td></tr><tr >
<td>contig_5</td><td align="center">0.31817</td></tr><tr >
<td>contig_6</td><td align="center">0.31819</td></tr><tr >
<td>contig_7</td><td align="center">0.31509</td></tr><tr >
<td>contig_8</td><td align="center">0.31627</td></tr><tr >
<td>contig_9</td><td align="center">0.31528</td></tr><tr >
<td>contig_10</td><td align="center">0.31936</td></tr><tr >
<td>contig_11</td><td align="center">0.31224</td></tr><tr >
<td>contig_12</td><td align="center">0.32328</td></tr><tr >
<td>contig_13</td><td align="center">0.31476</td></tr><tr >
<td>contig_14</td><td align="center">0.31454</td></tr><tr >
<td>contig_15</td><td align="center">0.32172</td></tr><tr >
<td>contig_16</td><td align="center">0.31857</td></tr><tr >
<td>contig_17</td><td align="center">0.30641</td></tr><tr >
<td>contig_18</td><td align="center">0.32288</td></tr><tr >
<td>contig_19</td><td align="center">0.36128</td></tr><tr >
<td>contig_20</td><td align="center">0.30687</td></tr><tr >
<td>contig_21</td><td align="center">0.31058</td></tr><tr >
<td>contig_22</td><td align="center">0.32135</td></tr><tr >
<td>contig_23</td><td align="center">0.31019</td></tr><tr >
<td>contig_24</td><td align="center">0.31470</td></tr><tr >
<td>contig_25</td><td align="center">0.16751</td></tr><tr >
<td>contig_26</td><td align="center">0.32192</td></tr><tr >
<td>contig_27</td><td align="center">0.22111</td></tr><tr >
<td>contig_28</td><td align="center">0.17600</td></tr><tr >
<td>contig_29</td><td align="center">0.17357</td></tr><tr >
<td>contig_30</td><td align="center">0.16919</td></tr><tr >
<td>contig_31</td><td align="center">NA</td></tr><tr >
<td>contig_32</td><td align="center">0.91766</td></tr><tr >
<td>contig_33</td><td align="center">0.20098</td></tr><tr >
<td>contig_34</td><td align="center">0.98546</td></tr><tr >
<td>contig_35</td><td align="center">0.18368</td></tr><tr >
<td>contig_36</td><td align="center">0.31922</td></tr><tr >
<td>contig_37</td><td align="center">NA</td></tr><tr >
<td>contig_38</td><td align="center">0.14291</td></tr><tr >
<td>contig_39</td><td align="center">NA</td></tr><tr >
<td>contig_40</td><td align="center">0.16864</td></tr><tr >
<td>contig_41</td><td align="center">0.33706</td></tr><tr >
<td>contig_42</td><td align="center">0.21362</td></tr><tr >
<td>contig_43</td><td align="center">0.04918</td></tr><tr >
<td>contig_44</td><td align="center">0.19514</td></tr><tr >
<td>contig_45</td><td align="center">0.39297</td></tr><tr >
<td>contig_46</td><td align="center">0.21141</td></tr><tr >
<td>contig_47</td><td align="center">0.21425</td></tr><tr >
<td>contig_48</td><td align="center">0.23895</td></tr><tr >
<td>contig_49</td><td align="center">0.28344</td></tr><tr >
<td>contig_50</td><td align="center">0.19648</td></tr><tr >
<td>contig_51</td><td align="center">0.13571</td></tr><tr >
<td>contig_52</td><td align="center">0.24094</td></tr><tr >
<td>contig_53</td><td align="center">NA</td></tr><tr >
<td>contig_54</td><td align="center">NA</td></tr><tr >
<td>contig_55</td><td align="center">0.22700</td></tr><tr >
<td>contig_56</td><td align="center">0.09311</td></tr><tr >
<td>contig_57</td><td align="center">NA</td></tr><tr >
<td>contig_58</td><td align="center">0.27055</td></tr><tr >
<td>contig_59</td><td align="center">0.22357</td></tr><tr >
<td>contig_60</td><td align="center">0.24731</td></tr><tr >
<td>contig_61</td><td align="center">0.25998</td></tr><tr >
<td>contig_62</td><td align="center">NA</td></tr><tr >
<td>contig_63</td><td align="center">0.31384</td></tr><tr >
<td>contig_64</td><td align="center">0.25635</td></tr><tr >
<td>contig_65</td><td align="center">0.25224</td></tr><tr >
<td>contig_66</td><td align="center">0.35000</td></tr><tr >
<td>contig_67</td><td align="center">NA</td></tr><tr >
<td>contig_68</td><td align="center">0.30422</td></tr><tr >
<td>contig_69</td><td align="center">0.28305</td></tr><tr >
<td>contig_70</td><td align="center">0.31263</td></tr><tr >
<td>contig_71</td><td align="center">NA</td></tr><tr >
<td>contig_72</td><td align="center">0.30263</td></tr><tr >
<td>contig_73</td><td align="center">0.47300</td></tr><tr >
<td>contig_74</td><td align="center">0.47586</td></tr><tr >
<td>contig_75</td><td align="center">0.30370</td></tr><tr >
<td>contig_76</td><td align="center">0.30991</td></tr><tr >
<td>contig_77</td><td align="center">NA</td></tr><tr >
<td>contig_78</td><td align="center">NA</td></tr><tr >
<td>contig_79</td><td align="center">0.36031</td></tr><tr >
<td>contig_80</td><td align="center">NA</td></tr><tr >
<td>contig_81</td><td align="center">0.48021</td></tr><tr >
<td>contig_82</td><td align="center">0.38911</td></tr><tr >
<td>contig_83</td><td align="center">NA</td></tr><tr >
<td>contig_84</td><td align="center">0.38595</td></tr><tr >
<td>contig_85</td><td align="center">NA</td></tr><tr >
<td>contig_86</td><td align="center">NA</td></tr><tr >
<td>contig_87</td><td align="center">0.57468</td></tr><tr >
<td>contig_88</td><td align="center">0.83904</td></tr><tr >
<td>contig_89</td><td align="center">NA</td></tr><tr >
<td>contig_90</td><td align="center">NA</td></tr><tr >
<td>contig_91</td><td align="center">0.40757</td></tr><tr >
<td>contig_92</td><td align="center">NA</td></tr><tr >
<td>contig_93</td><td align="center">0.21717</td></tr><tr >
<td>contig_94</td><td align="center">0.34318</td></tr><tr >
<td>contig_95</td><td align="center">NA</td></tr><tr >
<td>contig_96</td><td align="center">0.44431</td></tr><tr >
<td>contig_97</td><td align="center">0.63346</td></tr><tr >
<td>contig_98</td><td align="center">0.38025</td></tr><tr >
<td>contig_99</td><td align="center">NA</td></tr><tr >
<td>contig_100</td><td align="center">NA</td></tr><tr >
<td>contig_101</td><td align="center">0.43541</td></tr><tr >
<td>contig_102</td><td align="center">0.53871</td></tr><tr >
<td>contig_103</td><td align="center">NA</td></tr><tr >
<td>contig_104</td><td align="center">0.45370</td></tr><tr >
<td>contig_105</td><td align="center">0.56546</td></tr><tr >
<td>contig_106</td><td align="center">0.56825</td></tr><tr >
<td>contig_107</td><td align="center">NA</td></tr><tr >
<td>contig_108</td><td align="center">0.48676</td></tr><tr >
<td>contig_109</td><td align="center">0.47470</td></tr><tr >
<td>contig_110</td><td align="center">NA</td></tr><tr >
<td>contig_111</td><td align="center">0.53918</td></tr><tr >
<td>contig_112</td><td align="center">0.44089</td></tr><tr >
<td>contig_113</td><td align="center">0.62945</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.5.4</td></tr>
<tr><td>R</td><td>4.4.1</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>08:04:24&nbsp;28&nbsp;Feb&nbsp;2025</td><td>08:08:32&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;minutes&nbsp;8&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>08:08:32&nbsp;28&nbsp;Feb&nbsp;2025</td><td>08:12:55&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;minutes&nbsp;23&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>08:12:55&nbsp;28&nbsp;Feb&nbsp;2025</td><td>08:15:54&nbsp;28&nbsp;Feb&nbsp;2025</td><td>2&nbsp;minutes&nbsp;59&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>08:15:54&nbsp;28&nbsp;Feb&nbsp;2025</td><td>08:25:17&nbsp;28&nbsp;Feb&nbsp;2025</td><td>9&nbsp;minutes&nbsp;23&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>08:25:17&nbsp;28&nbsp;Feb&nbsp;2025</td><td>08:27:27&nbsp;28&nbsp;Feb&nbsp;2025</td><td>2&nbsp;minutes&nbsp;10&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>08:27:27&nbsp;28&nbsp;Feb&nbsp;2025</td><td>08:29:29&nbsp;28&nbsp;Feb&nbsp;2025</td><td>2&nbsp;minutes&nbsp;2&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>08:29:29&nbsp;28&nbsp;Feb&nbsp;2025</td><td>08:35:00&nbsp;28&nbsp;Feb&nbsp;2025</td><td>5&nbsp;minutes&nbsp;31&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>08:35:00&nbsp;28&nbsp;Feb&nbsp;2025</td><td>08:40:27&nbsp;28&nbsp;Feb&nbsp;2025</td><td>5&nbsp;minutes&nbsp;27&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>08:40:27&nbsp;28&nbsp;Feb&nbsp;2025</td><td>08:46:08&nbsp;28&nbsp;Feb&nbsp;2025</td><td>5&nbsp;minutes&nbsp;41&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>08:46:08&nbsp;28&nbsp;Feb&nbsp;2025</td><td>08:47:04&nbsp;28&nbsp;Feb&nbsp;2025</td><td>56&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>08:47:04&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:17:46&nbsp;28&nbsp;Feb&nbsp;2025</td><td>30&nbsp;minutes&nbsp;42&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>09:17:46&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:17:48&nbsp;28&nbsp;Feb&nbsp;2025</td><td>2&nbsp;seconds</td></tr>
<tr><td>Output</td><td>09:17:48&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:18:10&nbsp;28&nbsp;Feb&nbsp;2025</td><td>22&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>09:17:48&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:17:55&nbsp;28&nbsp;Feb&nbsp;2025</td><td>7&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>09:17:55&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:17:56&nbsp;28&nbsp;Feb&nbsp;2025</td><td>1&nbsp;second</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>1&nbsp;hour&nbsp;13&nbsp;minutes&nbsp;54&nbsp;seconds</b></td>
</tr>
</table></body></html>