<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.35.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/BTF1T2_R1-trimmed.error_rates.pdf">errors</a></td><td>BTF1T2_R1-trimmed</td><td align="right">928,848</td><td align="right">136,184,037</td><td align="right">100.0%</td><td align="right">146.6&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">99.8%</td></tr><tr ><td><a href="calibration/BTF1T2_R2-trimmed.error_rates.pdf">errors</a></td><td>BTF1T2_R2-trimmed</td><td align="right">927,961</td><td align="right">135,839,007</td><td align="right">99.9%</td><td align="right">146.4&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">99.7%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>1,856,809</b></td><td align="right"><b>272,023,044</b></td><td align="right">100.0%</td><td align="right">146.5&nbsp;bases</td><td align="right"><b>151</b>&nbsp;bases</td><td align="right">99.8%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit dispersion</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/contig_1.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>contig_1</td><td align="right">380,550</td><td align="center">38.2</td><td align="center">1.8</td><td align="center">5.5%</td><td>BTF1T1 contig_1, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_2.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>contig_2</td><td align="right">375,186</td><td align="center">39.9</td><td align="center">1.9</td><td align="center">5.5%</td><td>BTF1T1 contig_2, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_3.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>contig_3</td><td align="right">307,534</td><td align="center">38.0</td><td align="center">1.8</td><td align="center">4.4%</td><td>BTF1T1 contig_3, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_4.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>contig_4</td><td align="right">283,621</td><td align="center">40.0</td><td align="center">2.0</td><td align="center">4.2%</td><td>BTF1T1 contig_4, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_5.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>contig_5</td><td align="right">280,154</td><td align="center">39.1</td><td align="center">1.8</td><td align="center">4.1%</td><td>BTF1T1 contig_5, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_6.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>contig_6</td><td align="right">248,407</td><td align="center">40.1</td><td align="center">1.7</td><td align="center">3.8%</td><td>BTF1T1 contig_6, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_7.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>contig_7</td><td align="right">237,094</td><td align="center">38.0</td><td align="center">1.7</td><td align="center">3.4%</td><td>BTF1T1 contig_7, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_8.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>contig_8</td><td align="right">205,669</td><td align="center">40.2</td><td align="center">1.7</td><td align="center">3.1%</td><td>BTF1T1 contig_8, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_9.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>contig_9</td><td align="right">196,099</td><td align="center">40.2</td><td align="center">2.0</td><td align="center">2.9%</td><td>BTF1T1 contig_9, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_10.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>contig_10</td><td align="right">183,430</td><td align="center">38.2</td><td align="center">1.7</td><td align="center">2.6%</td><td>BTF1T1 contig_10, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_11.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>contig_11</td><td align="right">174,958</td><td align="center">39.0</td><td align="center">1.8</td><td align="center">2.5%</td><td>BTF1T1 contig_11, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_12.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>contig_12</td><td align="right">172,611</td><td align="center">40.3</td><td align="center">1.9</td><td align="center">2.6%</td><td>BTF1T1 contig_12, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_13.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>contig_13</td><td align="right">168,639</td><td align="center">37.7</td><td align="center">1.8</td><td align="center">2.4%</td><td>BTF1T1 contig_13, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_14.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>contig_14</td><td align="right">153,483</td><td align="center">40.4</td><td align="center">1.9</td><td align="center">2.3%</td><td>BTF1T1 contig_14, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_15.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>contig_15</td><td align="right">139,938</td><td align="center">40.7</td><td align="center">2.0</td><td align="center">2.1%</td><td>BTF1T1 contig_15, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_16.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>contig_16</td><td align="right">136,855</td><td align="center">41.0</td><td align="center">1.8</td><td align="center">2.1%</td><td>BTF1T1 contig_16, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_17.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>contig_17</td><td align="right">135,397</td><td align="center">38.7</td><td align="center">2.0</td><td align="center">2.0%</td><td>BTF1T1 contig_17, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_18.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>contig_18</td><td align="right">113,392</td><td align="center">39.4</td><td align="center">1.8</td><td align="center">1.7%</td><td>BTF1T1 contig_18, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_19.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>contig_19</td><td align="right">112,842</td><td align="center">39.5</td><td align="center">1.9</td><td align="center">1.7%</td><td>BTF1T1 contig_19, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_20.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>contig_20</td><td align="right">104,514</td><td align="center">38.1</td><td align="center">1.8</td><td align="center">1.5%</td><td>BTF1T1 contig_20, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_21.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>contig_21</td><td align="right">94,404</td><td align="center">39.5</td><td align="center">1.9</td><td align="center">1.4%</td><td>BTF1T1 contig_21, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_22.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>contig_22</td><td align="right">76,235</td><td align="center">39.0</td><td align="center">1.8</td><td align="center">1.1%</td><td>BTF1T1 contig_22, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_23.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>contig_23</td><td align="right">72,143</td><td align="center">40.6</td><td align="center">1.9</td><td align="center">1.1%</td><td>BTF1T1 contig_23, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_24.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>contig_24</td><td align="right">70,047</td><td align="center">38.1</td><td align="center">2.0</td><td align="center">1.0%</td><td>BTF1T1 contig_24, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_25.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>contig_25</td><td align="right">63,858</td><td align="center">39.0</td><td align="center">2.1</td><td align="center">0.9%</td><td>BTF1T1 contig_25, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_26.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>contig_26</td><td align="right">52,964</td><td align="center">40.0</td><td align="center">1.8</td><td align="center">0.8%</td><td>BTF1T1 contig_26, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_27.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>contig_27</td><td align="right">41,325</td><td align="center">41.2</td><td align="center">2.0</td><td align="center">0.6%</td><td>BTF1T1 contig_27, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_28.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>contig_28</td><td align="right">37,243</td><td align="center">38.1</td><td align="center">1.9</td><td align="center">0.5%</td><td>BTF1T1 contig_28, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_29.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>contig_29</td><td align="right">36,718</td><td align="center">38.5</td><td align="center">1.6</td><td align="center">0.5%</td><td>BTF1T1 contig_29, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_30.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>contig_30</td><td align="right">34,681</td><td align="center">39.5</td><td align="center">1.7</td><td align="center">0.5%</td><td>BTF1T1 contig_30, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_31.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>contig_31</td><td align="right">33,108</td><td align="center">42.9</td><td align="center">1.6</td><td align="center">0.5%</td><td>BTF1T1 contig_31, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_32.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>contig_32</td><td align="right">31,847</td><td align="center">50.4</td><td align="center">1.6</td><td align="center">0.6%</td><td>BTF1T1 contig_32, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_33.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>contig_33</td><td align="right">29,837</td><td align="center">40.4</td><td align="center">1.9</td><td align="center">0.5%</td><td>BTF1T1 contig_33, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_34.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>contig_34</td><td align="right">29,093</td><td align="center">39.0</td><td align="center">1.7</td><td align="center">0.4%</td><td>BTF1T1 contig_34, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_35.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>contig_35</td><td align="right">27,700</td><td align="center">38.6</td><td align="center">1.7</td><td align="center">0.4%</td><td>BTF1T1 contig_35, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_36.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>contig_36</td><td align="right">23,800</td><td align="center">41.5</td><td align="center">1.8</td><td align="center">0.4%</td><td>BTF1T1 contig_36, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_37.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>contig_37</td><td align="right">23,386</td><td align="center">39.6</td><td align="center">2.0</td><td align="center">0.4%</td><td>BTF1T1 contig_37, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_38.overview.png">coverage</a></td><td><a href="calibration/37.unique_coverage.pdf">distribution</a></td><td>contig_38</td><td align="right">19,988</td><td align="center">38.2</td><td align="center">1.5</td><td align="center">0.3%</td><td>BTF1T1 contig_38, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_39.overview.png">coverage</a></td><td><a href="calibration/38.unique_coverage.pdf">distribution</a></td><td>contig_39</td><td align="right">17,647</td><td align="center">38.4</td><td align="center">1.4</td><td align="center">0.2%</td><td>BTF1T1 contig_39, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_40.overview.png">coverage</a></td><td><a href="calibration/39.unique_coverage.pdf">distribution</a></td><td>contig_40</td><td align="right">17,587</td><td align="center">37.2</td><td align="center">1.8</td><td align="center">0.2%</td><td>BTF1T1 contig_40, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_41.overview.png">coverage</a></td><td><a href="calibration/40.unique_coverage.pdf">distribution</a></td><td>contig_41</td><td align="right">16,649</td><td align="center">111.6</td><td align="center">2.3</td><td align="center">0.7%</td><td>BTF1T1 contig_41, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_42.overview.png">coverage</a></td><td><a href="calibration/41.unique_coverage.pdf">distribution</a></td><td>contig_42</td><td align="right">11,663</td><td align="center">50.6</td><td align="center">1.8</td><td align="center">0.2%</td><td>BTF1T1 contig_42, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_43.overview.png">coverage</a></td><td><a href="calibration/42.unique_coverage.pdf">distribution</a></td><td>contig_43</td><td align="right">11,201</td><td align="center">722.4</td><td align="center">5.1</td><td align="center">3.0%</td><td>BTF1T1 contig_43, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_44.overview.png">coverage</a></td><td><a href="calibration/43.unique_coverage.pdf">distribution</a></td><td>contig_44</td><td align="right">11,134</td><td align="center">39.9</td><td align="center">1.8</td><td align="center">0.2%</td><td>BTF1T1 contig_44, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_45.overview.png">coverage</a></td><td><a href="calibration/44.unique_coverage.pdf">distribution</a></td><td>contig_45</td><td align="right">10,026</td><td align="center">36.8</td><td align="center">1.4</td><td align="center">0.1%</td><td>BTF1T1 contig_45, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_46.overview.png">coverage</a></td><td><a href="calibration/45.unique_coverage.pdf">distribution</a></td><td>contig_46</td><td align="right">8,763</td><td align="center">114.4</td><td align="center">1.9</td><td align="center">0.4%</td><td>BTF1T1 contig_46, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_47.overview.png">coverage</a></td><td><a href="calibration/46.unique_coverage.pdf">distribution</a></td><td>contig_47</td><td align="right">6,329</td><td align="center">146.2</td><td align="center">1.7</td><td align="center">0.3%</td><td>BTF1T1 contig_47, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_48.overview.png">coverage</a></td><td><a href="calibration/47.unique_coverage.pdf">distribution</a></td><td>contig_48</td><td align="right">5,570</td><td align="center">3078.2</td><td align="center">14.5</td><td align="center">6.2%</td><td>BTF1T1 contig_48, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_49.overview.png">coverage</a></td><td><a href="calibration/48.unique_coverage.pdf">distribution</a></td><td>contig_49</td><td align="right">5,563</td><td align="center">113.8</td><td align="center">2.1</td><td align="center">0.2%</td><td>BTF1T1 contig_49, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_50.overview.png">coverage</a></td><td><a href="calibration/49.unique_coverage.pdf">distribution</a></td><td>contig_50</td><td align="right">5,167</td><td align="center">2636.2</td><td align="center">10.4</td><td align="center">4.9%</td><td>BTF1T1 contig_50, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_51.overview.png">coverage</a></td><td><a href="calibration/50.unique_coverage.pdf">distribution</a></td><td>contig_51</td><td align="right">4,676</td><td align="center">36.8</td><td align="center">1.2</td><td align="center">0.1%</td><td>BTF1T1 contig_51, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_52.overview.png">coverage</a></td><td><a href="calibration/51.unique_coverage.pdf">distribution</a></td><td>contig_52</td><td align="right">4,143</td><td align="center">33.9</td><td align="center">2.2</td><td align="center">0.1%</td><td>BTF1T1 contig_52, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_53.overview.png">coverage</a></td><td><a href="calibration/52.unique_coverage.pdf">distribution</a></td><td>contig_53</td><td align="right">3,935</td><td align="center">773.6</td><td align="center">4.0</td><td align="center">1.1%</td><td>BTF1T1 contig_53, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_54.overview.png">coverage</a></td><td><a href="calibration/53.unique_coverage.pdf">distribution</a></td><td>contig_54</td><td align="right">3,849</td><td align="center">112.8</td><td align="center">2.4</td><td align="center">0.2%</td><td>BTF1T1 contig_54, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_55.overview.png">coverage</a></td><td><a href="calibration/54.unique_coverage.pdf">distribution</a></td><td>contig_55</td><td align="right">3,835</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>BTF1T1 contig_55, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_56.overview.png">coverage</a></td><td><a href="calibration/55.unique_coverage.pdf">distribution</a></td><td>contig_56</td><td align="right">3,722</td><td align="center">48.5</td><td align="center">1.0</td><td align="center">0.0%</td><td>BTF1T1 contig_56, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_57.overview.png">coverage</a></td><td><a href="calibration/56.unique_coverage.pdf">distribution</a></td><td>contig_57</td><td align="right">3,719</td><td align="center">38.5</td><td align="center">1.3</td><td align="center">0.1%</td><td>BTF1T1 contig_57, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_58.overview.png">coverage</a></td><td><a href="calibration/57.unique_coverage.pdf">distribution</a></td><td>contig_58</td><td align="right">3,623</td><td align="center">53.2</td><td align="center">2.1</td><td align="center">0.1%</td><td>BTF1T1 contig_58, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_59.overview.png">coverage</a></td><td><a href="calibration/58.unique_coverage.pdf">distribution</a></td><td>contig_59</td><td align="right">3,151</td><td align="center">276.1</td><td align="center">2.4</td><td align="center">0.3%</td><td>BTF1T1 contig_59, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_60.overview.png">coverage</a></td><td><a href="calibration/59.unique_coverage.pdf">distribution</a></td><td>contig_60</td><td align="right">2,793</td><td align="center">147.0</td><td align="center">2.2</td><td align="center">0.1%</td><td>BTF1T1 contig_60, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_61.overview.png">coverage</a></td><td><a href="calibration/60.unique_coverage.pdf">distribution</a></td><td>contig_61</td><td align="right">2,656</td><td align="center">38.7</td><td align="center">3.2</td><td align="center">0.0%</td><td>BTF1T1 contig_61, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_62.overview.png">coverage</a></td><td><a href="calibration/61.unique_coverage.pdf">distribution</a></td><td>contig_62</td><td align="right">2,276</td><td align="center">84.4</td><td align="center">1.2</td><td align="center">0.1%</td><td>BTF1T1 contig_62, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_63.overview.png">coverage</a></td><td><a href="calibration/62.unique_coverage.pdf">distribution</a></td><td>contig_63</td><td align="right">2,184</td><td align="center">76.3</td><td align="center">2.6</td><td align="center">0.1%</td><td>BTF1T1 contig_63, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_64.overview.png">coverage</a></td><td><a href="calibration/63.unique_coverage.pdf">distribution</a></td><td>contig_64</td><td align="right">2,113</td><td align="center">9380.3</td><td align="center">19.3</td><td align="center">6.8%</td><td>BTF1T1 contig_64, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_65.overview.png">coverage</a></td><td><a href="calibration/64.unique_coverage.pdf">distribution</a></td><td>contig_65</td><td align="right">2,098</td><td align="center">36.5</td><td align="center">1.6</td><td align="center">0.0%</td><td>BTF1T1 contig_65, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_66.overview.png">coverage</a></td><td><a href="calibration/65.unique_coverage.pdf">distribution</a></td><td>contig_66</td><td align="right">2,008</td><td align="center">149.4</td><td align="center">2.3</td><td align="center">0.1%</td><td>BTF1T1 contig_66, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_67.overview.png">coverage</a></td><td><a href="calibration/66.unique_coverage.pdf">distribution</a></td><td>contig_67</td><td align="right">1,669</td><td align="center">86.1</td><td align="center">1.0</td><td align="center">0.1%</td><td>BTF1T1 contig_67, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_68.overview.png">coverage</a></td><td><a href="calibration/67.unique_coverage.pdf">distribution</a></td><td>contig_68</td><td align="right">1,564</td><td align="center">282.8</td><td align="center">2.1</td><td align="center">0.2%</td><td>BTF1T1 contig_68, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_69.overview.png">coverage</a></td><td><a href="calibration/68.unique_coverage.pdf">distribution</a></td><td>contig_69</td><td align="right">1,403</td><td align="center">776.5</td><td align="center">3.4</td><td align="center">0.4%</td><td>BTF1T1 contig_69, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_70.overview.png">coverage</a></td><td><a href="calibration/69.unique_coverage.pdf">distribution</a></td><td>contig_70</td><td align="right">1,323</td><td align="center">83.9</td><td align="center">2.8</td><td align="center">0.0%</td><td>BTF1T1 contig_70, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_71.overview.png">coverage</a></td><td><a href="calibration/70.unique_coverage.pdf">distribution</a></td><td>contig_71</td><td align="right">1,259</td><td align="center">36.8</td><td align="center">1.4</td><td align="center">0.0%</td><td>BTF1T1 contig_71, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_72.overview.png">coverage</a></td><td><a href="calibration/71.unique_coverage.pdf">distribution</a></td><td>contig_72</td><td align="right">1,242</td><td align="center">57.6</td><td align="center">1.4</td><td align="center">0.0%</td><td>BTF1T1 contig_72, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_73.overview.png">coverage</a></td><td><a href="calibration/72.unique_coverage.pdf">distribution</a></td><td>contig_73</td><td align="right">1,235</td><td align="center">46.2</td><td align="center">4.6</td><td align="center">0.0%</td><td>BTF1T1 contig_73, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_74.overview.png">coverage</a></td><td><a href="calibration/73.unique_coverage.pdf">distribution</a></td><td>contig_74</td><td align="right">1,225</td><td align="center">37.1</td><td align="center">1.2</td><td align="center">0.0%</td><td>BTF1T1 contig_74, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_75.overview.png">coverage</a></td><td><a href="calibration/74.unique_coverage.pdf">distribution</a></td><td>contig_75</td><td align="right">1,215</td><td align="center">35.9</td><td align="center">1.8</td><td align="center">0.0%</td><td>BTF1T1 contig_75, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_76.overview.png">coverage</a></td><td><a href="calibration/75.unique_coverage.pdf">distribution</a></td><td>contig_76</td><td align="right">1,150</td><td align="center">32.3</td><td align="center">1.6</td><td align="center">0.0%</td><td>BTF1T1 contig_76, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_77.overview.png">coverage</a></td><td><a href="calibration/76.unique_coverage.pdf">distribution</a></td><td>contig_77</td><td align="right">1,150</td><td align="center">41.3</td><td align="center">2.5</td><td align="center">0.0%</td><td>BTF1T1 contig_77, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_78.overview.png">coverage</a></td><td><a href="calibration/77.unique_coverage.pdf">distribution</a></td><td>contig_78</td><td align="right">1,114</td><td align="center">72.5</td><td align="center">2.8</td><td align="center">0.0%</td><td>BTF1T1 contig_78, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_79.overview.png">coverage</a></td><td><a href="calibration/78.unique_coverage.pdf">distribution</a></td><td>contig_79</td><td align="right">1,018</td><td align="center">30.3</td><td align="center">1.0</td><td align="center">0.0%</td><td>BTF1T1 contig_79, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_80.overview.png">coverage</a></td><td><a href="calibration/79.unique_coverage.pdf">distribution</a></td><td>contig_80</td><td align="right">950</td><td align="center">65.1</td><td align="center">1.7</td><td align="center">0.0%</td><td>BTF1T1 contig_80, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_81.overview.png">coverage</a></td><td><a href="calibration/80.unique_coverage.pdf">distribution</a></td><td>contig_81</td><td align="right">905</td><td align="center">78.1</td><td align="center">1.0</td><td align="center">0.0%</td><td>BTF1T1 contig_81, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_82.overview.png">coverage</a></td><td><a href="calibration/81.unique_coverage.pdf">distribution</a></td><td>contig_82</td><td align="right">851</td><td align="center">154.2</td><td align="center">1.8</td><td align="center">0.0%</td><td>BTF1T1 contig_82, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_83.overview.png">coverage</a></td><td><a href="calibration/82.unique_coverage.pdf">distribution</a></td><td>contig_83</td><td align="right">849</td><td align="center">126.3</td><td align="center">2.3</td><td align="center">0.0%</td><td>BTF1T1 contig_83, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_84.overview.png">coverage</a></td><td><a href="calibration/83.unique_coverage.pdf">distribution</a></td><td>contig_84</td><td align="right">840</td><td align="center">151.2</td><td align="center">1.6</td><td align="center">0.0%</td><td>BTF1T1 contig_84, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_85.overview.png">coverage</a></td><td><a href="calibration/84.unique_coverage.pdf">distribution</a></td><td>contig_85</td><td align="right">838</td><td align="center">147.5</td><td align="center">2.2</td><td align="center">0.0%</td><td>BTF1T1 contig_85, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_86.overview.png">coverage</a></td><td><a href="calibration/85.unique_coverage.pdf">distribution</a></td><td>contig_86</td><td align="right">821</td><td align="center">1620.6</td><td align="center">7.5</td><td align="center">0.4%</td><td>BTF1T1 contig_86, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_87.overview.png">coverage</a></td><td><a href="calibration/86.unique_coverage.pdf">distribution</a></td><td>contig_87</td><td align="right">802</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>BTF1T1 contig_87, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_88.overview.png">coverage</a></td><td><a href="calibration/87.unique_coverage.pdf">distribution</a></td><td>contig_88</td><td align="right">725</td><td align="center">110.2</td><td align="center">3.8</td><td align="center">0.0%</td><td>BTF1T1 contig_88, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_89.overview.png">coverage</a></td><td><a href="calibration/88.unique_coverage.pdf">distribution</a></td><td>contig_89</td><td align="right">716</td><td align="center">33.6</td><td align="center">1.2</td><td align="center">0.0%</td><td>BTF1T1 contig_89, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_90.overview.png">coverage</a></td><td><a href="calibration/89.unique_coverage.pdf">distribution</a></td><td>contig_90</td><td align="right">716</td><td align="center">152.3</td><td align="center">1.4</td><td align="center">0.0%</td><td>BTF1T1 contig_90, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_91.overview.png">coverage</a></td><td><a href="calibration/90.unique_coverage.pdf">distribution</a></td><td>contig_91</td><td align="right">689</td><td align="center">82.2</td><td align="center">2.0</td><td align="center">0.0%</td><td>BTF1T1 contig_91, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_92.overview.png">coverage</a></td><td><a href="calibration/91.unique_coverage.pdf">distribution</a></td><td>contig_92</td><td align="right">648</td><td align="center">91.7</td><td align="center">2.1</td><td align="center">0.0%</td><td>BTF1T1 contig_92, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_93.overview.png">coverage</a></td><td><a href="calibration/92.unique_coverage.pdf">distribution</a></td><td>contig_93</td><td align="right">641</td><td align="center">31.9</td><td align="center">1.0</td><td align="center">0.0%</td><td>BTF1T1 contig_93, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_94.overview.png">coverage</a></td><td><a href="calibration/93.unique_coverage.pdf">distribution</a></td><td>contig_94</td><td align="right">624</td><td align="center">79.9</td><td align="center">1.2</td><td align="center">0.0%</td><td>BTF1T1 contig_94, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_95.overview.png">coverage</a></td><td><a href="calibration/94.unique_coverage.pdf">distribution</a></td><td>contig_95</td><td align="right">612</td><td align="center">220.3</td><td align="center">1.0</td><td align="center">0.0%</td><td>BTF1T1 contig_95, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_96.overview.png">coverage</a></td><td><a href="calibration/95.unique_coverage.pdf">distribution</a></td><td>contig_96</td><td align="right">601</td><td align="center">41.4</td><td align="center">3.1</td><td align="center">0.0%</td><td>BTF1T1 contig_96, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_97.overview.png">coverage</a></td><td><a href="calibration/96.unique_coverage.pdf">distribution</a></td><td>contig_97</td><td align="right">553</td><td align="center">77.1</td><td align="center">1.0</td><td align="center">0.0%</td><td>BTF1T1 contig_97, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_98.overview.png">coverage</a></td><td><a href="calibration/97.unique_coverage.pdf">distribution</a></td><td>contig_98</td><td align="right">557</td><td align="center">662.8</td><td align="center">5.0</td><td align="center">0.1%</td><td>BTF1T1 contig_98, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_99.overview.png">coverage</a></td><td><a href="calibration/98.unique_coverage.pdf">distribution</a></td><td>contig_99</td><td align="right">518</td><td align="center">269.4</td><td align="center">1.6</td><td align="center">0.0%</td><td>BTF1T1 contig_99, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_100.overview.png">coverage</a></td><td><a href="calibration/99.unique_coverage.pdf">distribution</a></td><td>contig_100</td><td align="right">461</td><td align="center">36.0</td><td align="center">1.0</td><td align="center">0.0%</td><td>BTF1T1 contig_100, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_101.overview.png">coverage</a></td><td><a href="calibration/100.unique_coverage.pdf">distribution</a></td><td>contig_101</td><td align="right">449</td><td align="center">[61.2]</td><td align="center">[19.2]</td><td align="center">0.0%</td><td>BTF1T1 contig_101, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_102.overview.png">coverage</a></td><td><a href="calibration/101.unique_coverage.pdf">distribution</a></td><td>contig_102</td><td align="right">412</td><td align="center">[95.4]</td><td align="center">[24.6]</td><td align="center">0.0%</td><td>BTF1T1 contig_102, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_103.overview.png">coverage</a></td><td><a href="calibration/102.unique_coverage.pdf">distribution</a></td><td>contig_103</td><td align="right">361</td><td align="center">222.7</td><td align="center">1.0</td><td align="center">0.0%</td><td>BTF1T1 contig_103, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_104.overview.png">coverage</a></td><td><a href="calibration/103.unique_coverage.pdf">distribution</a></td><td>contig_104</td><td align="right">347</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>BTF1T1 contig_104, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_105.overview.png">coverage</a></td><td><a href="calibration/104.unique_coverage.pdf">distribution</a></td><td>contig_105</td><td align="right">344</td><td align="center">245.8</td><td align="center">1.8</td><td align="center">0.0%</td><td>BTF1T1 contig_105, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_106.overview.png">coverage</a></td><td><a href="calibration/105.unique_coverage.pdf">distribution</a></td><td>contig_106</td><td align="right">330</td><td align="center">40.8</td><td align="center">4.1</td><td align="center">0.0%</td><td>BTF1T1 contig_106, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_107.overview.png">coverage</a></td><td><a href="calibration/106.unique_coverage.pdf">distribution</a></td><td>contig_107</td><td align="right">330</td><td align="center">166.2</td><td align="center">1.0</td><td align="center">0.0%</td><td>BTF1T1 contig_107, whole genome shotgun sequence.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>5,120,887</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit dispersion</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="reject_table_row">Fit failed</span> Negative binomial fit failed for this reference sequence. It may have an unusual coverage depth distribution. JC and MC predictions may be less accurate.
<p><span class="gray_table_row">Insufficient coverage</span> Reference sequence counted as entirely deleted due to low coverage. Try either the <code>-t,--targeted-sequencing</code> or the <code>-c,--contig-reference </code> option if you want mutations called for these reference sequences.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">15223</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">276</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.016</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>contig_1</td><td align="center">0.91834</td></tr><tr >
<td>contig_2</td><td align="center">0.91728</td></tr><tr >
<td>contig_3</td><td align="center">0.91927</td></tr><tr >
<td>contig_4</td><td align="center">0.91587</td></tr><tr >
<td>contig_5</td><td align="center">0.91648</td></tr><tr >
<td>contig_6</td><td align="center">0.91475</td></tr><tr >
<td>contig_7</td><td align="center">0.91900</td></tr><tr >
<td>contig_8</td><td align="center">0.91515</td></tr><tr >
<td>contig_9</td><td align="center">0.91588</td></tr><tr >
<td>contig_10</td><td align="center">0.91863</td></tr><tr >
<td>contig_11</td><td align="center">0.91755</td></tr><tr >
<td>contig_12</td><td align="center">0.91531</td></tr><tr >
<td>contig_13</td><td align="center">0.91834</td></tr><tr >
<td>contig_14</td><td align="center">0.91509</td></tr><tr >
<td>contig_15</td><td align="center">0.91544</td></tr><tr >
<td>contig_16</td><td align="center">0.91455</td></tr><tr >
<td>contig_17</td><td align="center">0.91774</td></tr><tr >
<td>contig_18</td><td align="center">0.91627</td></tr><tr >
<td>contig_19</td><td align="center">0.91637</td></tr><tr >
<td>contig_20</td><td align="center">0.91866</td></tr><tr >
<td>contig_21</td><td align="center">0.91616</td></tr><tr >
<td>contig_22</td><td align="center">0.91773</td></tr><tr >
<td>contig_23</td><td align="center">0.91651</td></tr><tr >
<td>contig_24</td><td align="center">0.91856</td></tr><tr >
<td>contig_25</td><td align="center">0.91757</td></tr><tr >
<td>contig_26</td><td align="center">0.91650</td></tr><tr >
<td>contig_27</td><td align="center">0.91441</td></tr><tr >
<td>contig_28</td><td align="center">0.91926</td></tr><tr >
<td>contig_29</td><td align="center">0.91807</td></tr><tr >
<td>contig_30</td><td align="center">0.91691</td></tr><tr >
<td>contig_31</td><td align="center">0.90827</td></tr><tr >
<td>contig_32</td><td align="center">0.90086</td></tr><tr >
<td>contig_33</td><td align="center">0.91509</td></tr><tr >
<td>contig_34</td><td align="center">0.91793</td></tr><tr >
<td>contig_35</td><td align="center">0.91821</td></tr><tr >
<td>contig_36</td><td align="center">0.91511</td></tr><tr >
<td>contig_37</td><td align="center">0.91081</td></tr><tr >
<td>contig_38</td><td align="center">0.91197</td></tr><tr >
<td>contig_39</td><td align="center">0.91874</td></tr><tr >
<td>contig_40</td><td align="center">0.91954</td></tr><tr >
<td>contig_41</td><td align="center">0.82874</td></tr><tr >
<td>contig_42</td><td align="center">0.89971</td></tr><tr >
<td>contig_43</td><td align="center">0.59164</td></tr><tr >
<td>contig_44</td><td align="center">0.91553</td></tr><tr >
<td>contig_45</td><td align="center">0.91732</td></tr><tr >
<td>contig_46</td><td align="center">0.83010</td></tr><tr >
<td>contig_47</td><td align="center">0.79997</td></tr><tr >
<td>contig_48</td><td align="center">0.38878</td></tr><tr >
<td>contig_49</td><td align="center">0.83049</td></tr><tr >
<td>contig_50</td><td align="center">0.41639</td></tr><tr >
<td>contig_51</td><td align="center">0.92266</td></tr><tr >
<td>contig_52</td><td align="center">0.92517</td></tr><tr >
<td>contig_53</td><td align="center">0.59174</td></tr><tr >
<td>contig_54</td><td align="center">0.83174</td></tr><tr >
<td>contig_55</td><td align="center">NA</td></tr><tr >
<td>contig_56</td><td align="center">0.99516</td></tr><tr >
<td>contig_57</td><td align="center">0.91422</td></tr><tr >
<td>contig_58</td><td align="center">0.89690</td></tr><tr >
<td>contig_59</td><td align="center">0.72749</td></tr><tr >
<td>contig_60</td><td align="center">0.81078</td></tr><tr >
<td>contig_61</td><td align="center">0.92639</td></tr><tr >
<td>contig_62</td><td align="center">0.85874</td></tr><tr >
<td>contig_63</td><td align="center">0.87363</td></tr><tr >
<td>contig_64</td><td align="center">0.28357</td></tr><tr >
<td>contig_65</td><td align="center">0.91962</td></tr><tr >
<td>contig_66</td><td align="center">0.81125</td></tr><tr >
<td>contig_67</td><td align="center">0.85440</td></tr><tr >
<td>contig_68</td><td align="center">0.73849</td></tr><tr >
<td>contig_69</td><td align="center">0.59954</td></tr><tr >
<td>contig_70</td><td align="center">0.87377</td></tr><tr >
<td>contig_71</td><td align="center">0.92732</td></tr><tr >
<td>contig_72</td><td align="center">0.89694</td></tr><tr >
<td>contig_73</td><td align="center">0.91862</td></tr><tr >
<td>contig_74</td><td align="center">0.93378</td></tr><tr >
<td>contig_75</td><td align="center">0.93139</td></tr><tr >
<td>contig_76</td><td align="center">0.93555</td></tr><tr >
<td>contig_77</td><td align="center">0.91944</td></tr><tr >
<td>contig_78</td><td align="center">0.88869</td></tr><tr >
<td>contig_79</td><td align="center">0.93434</td></tr><tr >
<td>contig_80</td><td align="center">0.90316</td></tr><tr >
<td>contig_81</td><td align="center">0.87459</td></tr><tr >
<td>contig_82</td><td align="center">0.82785</td></tr><tr >
<td>contig_83</td><td align="center">0.82686</td></tr><tr >
<td>contig_84</td><td align="center">0.81845</td></tr><tr >
<td>contig_85</td><td align="center">0.83294</td></tr><tr >
<td>contig_86</td><td align="center">0.53715</td></tr><tr >
<td>contig_87</td><td align="center">NA</td></tr><tr >
<td>contig_88</td><td align="center">0.86061</td></tr><tr >
<td>contig_89</td><td align="center">0.93925</td></tr><tr >
<td>contig_90</td><td align="center">0.82891</td></tr><tr >
<td>contig_91</td><td align="center">0.87881</td></tr><tr >
<td>contig_92</td><td align="center">0.86420</td></tr><tr >
<td>contig_93</td><td align="center">0.92652</td></tr><tr >
<td>contig_94</td><td align="center">0.90946</td></tr><tr >
<td>contig_95</td><td align="center">0.78758</td></tr><tr >
<td>contig_96</td><td align="center">0.92596</td></tr><tr >
<td>contig_97</td><td align="center">0.90145</td></tr><tr >
<td>contig_98</td><td align="center">0.63910</td></tr><tr >
<td>contig_99</td><td align="center">0.80309</td></tr><tr >
<td>contig_100</td><td align="center">0.95011</td></tr><tr >
<td>contig_101</td><td align="center">NA</td></tr><tr >
<td>contig_102</td><td align="center">NA</td></tr><tr >
<td>contig_103</td><td align="center">0.83241</td></tr><tr >
<td>contig_104</td><td align="center">NA</td></tr><tr >
<td>contig_105</td><td align="center">0.81977</td></tr><tr >
<td>contig_106</td><td align="center">0.95623</td></tr><tr >
<td>contig_107</td><td align="center">0.87273</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.3.5.1</td></tr>
<tr><td>R</td><td>3.6.0</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>10:47:28&nbsp;08&nbsp;Dec&nbsp;2024</td><td>10:48:39&nbsp;08&nbsp;Dec&nbsp;2024</td><td>1&nbsp;minute&nbsp;11&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>10:48:40&nbsp;08&nbsp;Dec&nbsp;2024</td><td>10:55:10&nbsp;08&nbsp;Dec&nbsp;2024</td><td>6&nbsp;minutes&nbsp;30&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>10:55:10&nbsp;08&nbsp;Dec&nbsp;2024</td><td>10:55:51&nbsp;08&nbsp;Dec&nbsp;2024</td><td>41&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>10:55:51&nbsp;08&nbsp;Dec&nbsp;2024</td><td>10:58:35&nbsp;08&nbsp;Dec&nbsp;2024</td><td>2&nbsp;minutes&nbsp;44&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>10:58:35&nbsp;08&nbsp;Dec&nbsp;2024</td><td>10:58:48&nbsp;08&nbsp;Dec&nbsp;2024</td><td>13&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>10:58:48&nbsp;08&nbsp;Dec&nbsp;2024</td><td>11:00:44&nbsp;08&nbsp;Dec&nbsp;2024</td><td>1&nbsp;minute&nbsp;56&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>11:00:44&nbsp;08&nbsp;Dec&nbsp;2024</td><td>11:01:53&nbsp;08&nbsp;Dec&nbsp;2024</td><td>1&nbsp;minute&nbsp;9&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>11:01:53&nbsp;08&nbsp;Dec&nbsp;2024</td><td>11:03:08&nbsp;08&nbsp;Dec&nbsp;2024</td><td>1&nbsp;minute&nbsp;15&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>11:03:08&nbsp;08&nbsp;Dec&nbsp;2024</td><td>11:03:40&nbsp;08&nbsp;Dec&nbsp;2024</td><td>32&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>11:03:40&nbsp;08&nbsp;Dec&nbsp;2024</td><td>11:04:55&nbsp;08&nbsp;Dec&nbsp;2024</td><td>1&nbsp;minute&nbsp;15&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>11:04:55&nbsp;08&nbsp;Dec&nbsp;2024</td><td>11:12:08&nbsp;08&nbsp;Dec&nbsp;2024</td><td>7&nbsp;minutes&nbsp;13&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>11:12:08&nbsp;08&nbsp;Dec&nbsp;2024</td><td>11:12:09&nbsp;08&nbsp;Dec&nbsp;2024</td><td>1&nbsp;second</td></tr>
<tr><td>Output</td><td>11:12:09&nbsp;08&nbsp;Dec&nbsp;2024</td><td>11:14:28&nbsp;08&nbsp;Dec&nbsp;2024</td><td>2&nbsp;minutes&nbsp;19&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>26&nbsp;minutes&nbsp;59&nbsp;seconds</b></td>
</tr>
</table></body></html>