<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.38.2&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/BTF37T2b_R1-trimmed.error_rates.pdf">errors</a></td><td>BTF37T2b_R1-trimmed</td><td align="right">18,098,915</td><td align="right">2,666,968,096</td><td align="right">100.0%</td><td align="right">147.4&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">98.3%</td></tr><tr ><td><a href="calibration/BTF46T2b_R2-trimmed.error_rates.pdf">errors</a></td><td>BTF46T2b_R2-trimmed</td><td align="right">19,893,049</td><td align="right">2,964,674,853</td><td align="right">98.8%</td><td align="right">149.0&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">0.1%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>37,991,964</b></td><td align="right"><b>5,631,642,949</b></td><td align="right">99.4%</td><td align="right">148.2&nbsp;bases</td><td align="right"><b>151</b>&nbsp;bases</td><td align="right">46.9%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit relative_variance</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/contig_1.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>contig_1</td><td align="right">625,321</td><td align="center">845.8</td><td align="center">9.7</td><td align="center">20.1%</td><td>contig_1, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_2.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>contig_2</td><td align="right">446,328</td><td align="center">795.6</td><td align="center">7.7</td><td align="center">13.5%</td><td>contig_2, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_3.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>contig_3</td><td align="right">326,963</td><td align="center">847.9</td><td align="center">8.1</td><td align="center">10.6%</td><td>contig_3, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_4.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>contig_4</td><td align="right">326,164</td><td align="center">922.2</td><td align="center">9.8</td><td align="center">11.5%</td><td>contig_4, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_5.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>contig_5</td><td align="right">195,696</td><td align="center">787.8</td><td align="center">8.0</td><td align="center">5.8%</td><td>contig_5, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_6.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>contig_6</td><td align="right">179,540</td><td align="center">935.0</td><td align="center">10.5</td><td align="center">6.4%</td><td>contig_6, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_7.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>contig_7</td><td align="right">177,065</td><td align="center">906.7</td><td align="center">8.6</td><td align="center">6.1%</td><td>contig_7, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_8.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>contig_8</td><td align="right">103,915</td><td align="center">954.4</td><td align="center">10.2</td><td align="center">3.8%</td><td>contig_8, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_9.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>contig_9</td><td align="right">99,798</td><td align="center">799.7</td><td align="center">7.6</td><td align="center">3.0%</td><td>contig_9, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_10.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>contig_10</td><td align="right">71,019</td><td align="center">845.2</td><td align="center">15.2</td><td align="center">2.3%</td><td>contig_10, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_11.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>contig_11</td><td align="right">68,611</td><td align="center">1282.7</td><td align="center">13.9</td><td align="center">3.3%</td><td>contig_11, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_12.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>contig_12</td><td align="right">66,638</td><td align="center">883.7</td><td align="center">9.2</td><td align="center">2.2%</td><td>contig_12, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_13.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>contig_13</td><td align="right">65,135</td><td align="center">778.6</td><td align="center">7.5</td><td align="center">1.9%</td><td>contig_13, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_14.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>contig_14</td><td align="right">38,814</td><td align="center">864.9</td><td align="center">7.8</td><td align="center">1.3%</td><td>contig_14, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_15.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>contig_15</td><td align="right">34,288</td><td align="center">883.6</td><td align="center">4.9</td><td align="center">1.2%</td><td>contig_15, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_16.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>contig_16</td><td align="right">32,573</td><td align="center">822.4</td><td align="center">6.8</td><td align="center">1.0%</td><td>contig_16, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_17.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>contig_17</td><td align="right">28,114</td><td align="center">706.9</td><td align="center">22.4</td><td align="center">0.7%</td><td>contig_17, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_18.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>contig_18</td><td align="right">23,922</td><td align="center">981.4</td><td align="center">6.7</td><td align="center">0.9%</td><td>contig_18, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_19.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>contig_19</td><td align="right">16,136</td><td align="center">669.0</td><td align="center">11.1</td><td align="center">0.4%</td><td>contig_19, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_20.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>contig_20</td><td align="right">15,651</td><td align="center">900.3</td><td align="center">6.4</td><td align="center">0.5%</td><td>contig_20, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_21.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>contig_21</td><td align="right">15,356</td><td align="center">784.5</td><td align="center">8.3</td><td align="center">0.5%</td><td>contig_21, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_22.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>contig_22</td><td align="right">13,576</td><td align="center">839.3</td><td align="center">5.8</td><td align="center">0.4%</td><td>contig_22, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_23.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>contig_23</td><td align="right">7,926</td><td align="center">776.7</td><td align="center">4.2</td><td align="center">0.2%</td><td>contig_23, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_24.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>contig_24</td><td align="right">6,279</td><td align="center">848.0</td><td align="center">9.8</td><td align="center">0.2%</td><td>contig_24, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_25.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>contig_25</td><td align="right">6,003</td><td align="center">794.3</td><td align="center">6.6</td><td align="center">0.2%</td><td>contig_25, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_26.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>contig_26</td><td align="right">5,238</td><td align="center">707.9</td><td align="center">5.3</td><td align="center">0.1%</td><td>contig_26, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_27.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>contig_27</td><td align="right">5,183</td><td align="center">4290.7</td><td align="center">17.3</td><td align="center">0.9%</td><td>contig_27, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_28.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>contig_28</td><td align="right">3,987</td><td align="center">1794.9</td><td align="center">3.0</td><td align="center">0.3%</td><td>contig_28, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_29.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>contig_29</td><td align="right">3,281</td><td align="center">985.9</td><td align="center">1.7</td><td align="center">0.1%</td><td>contig_29, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_30.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>contig_30</td><td align="right">1,775</td><td align="center">1666.3</td><td align="center">7.2</td><td align="center">0.1%</td><td>contig_30, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_31.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>contig_31</td><td align="right">1,681</td><td align="center">3926.5</td><td align="center">100.7</td><td align="center">0.2%</td><td>contig_31, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_32.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>contig_32</td><td align="right">1,065</td><td align="center">3057.1</td><td align="center">2.7</td><td align="center">0.1%</td><td>contig_32, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_33.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>contig_33</td><td align="right">814</td><td align="center">1940.4</td><td align="center">2.8</td><td align="center">0.0%</td><td>contig_33, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_34.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>contig_34</td><td align="right">610</td><td align="center">2109.3</td><td align="center">1.5</td><td align="center">0.0%</td><td>contig_34, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_35.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>contig_35</td><td align="right">373</td><td align="center">[2618.6]</td><td align="center">[536.1]</td><td align="center">0.0%</td><td>contig_35, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_36.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>contig_36</td><td align="right">331</td><td align="center">[1237.3]</td><td align="center">[196.9]</td><td align="center">0.0%</td><td>contig_36, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_37.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>contig_37</td><td align="right">331</td><td align="center">[1711.4]</td><td align="center">[253.8]</td><td align="center">0.0%</td><td>contig_37, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_38.overview.png">coverage</a></td><td><a href="calibration/37.unique_coverage.pdf">distribution</a></td><td>contig_38</td><td align="right">326</td><td align="center">[547.3]</td><td align="center">[109.3]</td><td align="center">0.0%</td><td>contig_38, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_39.overview.png">coverage</a></td><td><a href="calibration/38.unique_coverage.pdf">distribution</a></td><td>contig_39</td><td align="right">306</td><td align="center">734.8</td><td align="center">5.7</td><td align="center">0.0%</td><td>contig_39, whole genome shotgun sequence.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>3,016,132</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit relative_variance</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="reject_table_row">Fit failed</span> Negative binomial fit failed for this reference sequence. It may have an unusual coverage depth distribution. JC and MC predictions may be less accurate.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">37464</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">143</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.015</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>contig_1</td><td align="center">0.17371</td></tr><tr >
<td>contig_2</td><td align="center">0.18886</td></tr><tr >
<td>contig_3</td><td align="center">0.17151</td></tr><tr >
<td>contig_4</td><td align="center">0.15055</td></tr><tr >
<td>contig_5</td><td align="center">0.19422</td></tr><tr >
<td>contig_6</td><td align="center">0.15121</td></tr><tr >
<td>contig_7</td><td align="center">0.15721</td></tr><tr >
<td>contig_8</td><td align="center">0.14295</td></tr><tr >
<td>contig_9</td><td align="center">0.18929</td></tr><tr >
<td>contig_10</td><td align="center">0.17732</td></tr><tr >
<td>contig_11</td><td align="center">0.09261</td></tr><tr >
<td>contig_12</td><td align="center">0.16530</td></tr><tr >
<td>contig_13</td><td align="center">0.19378</td></tr><tr >
<td>contig_14</td><td align="center">0.17092</td></tr><tr >
<td>contig_15</td><td align="center">0.16515</td></tr><tr >
<td>contig_16</td><td align="center">0.18334</td></tr><tr >
<td>contig_17</td><td align="center">0.23150</td></tr><tr >
<td>contig_18</td><td align="center">0.14136</td></tr><tr >
<td>contig_19</td><td align="center">0.24611</td></tr><tr >
<td>contig_20</td><td align="center">0.16260</td></tr><tr >
<td>contig_21</td><td align="center">0.20096</td></tr><tr >
<td>contig_22</td><td align="center">0.17435</td></tr><tr >
<td>contig_23</td><td align="center">0.20963</td></tr><tr >
<td>contig_24</td><td align="center">0.18888</td></tr><tr >
<td>contig_25</td><td align="center">0.20998</td></tr><tr >
<td>contig_26</td><td align="center">0.24447</td></tr><tr >
<td>contig_27</td><td align="center">0.02447</td></tr><tr >
<td>contig_28</td><td align="center">0.07750</td></tr><tr >
<td>contig_29</td><td align="center">0.14227</td></tr><tr >
<td>contig_30</td><td align="center">0.11775</td></tr><tr >
<td>contig_31</td><td align="center">0.07049</td></tr><tr >
<td>contig_32</td><td align="center">0.12113</td></tr><tr >
<td>contig_33</td><td align="center">0.17752</td></tr><tr >
<td>contig_34</td><td align="center">0.11266</td></tr><tr >
<td>contig_35</td><td align="center">NA</td></tr><tr >
<td>contig_36</td><td align="center">NA</td></tr><tr >
<td>contig_37</td><td align="center">NA</td></tr><tr >
<td>contig_38</td><td align="center">NA</td></tr><tr >
<td>contig_39</td><td align="center">0.54739</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.4.5</td></tr>
<tr><td>R</td><td>4.4.0</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>16:24:41&nbsp;26&nbsp;Apr&nbsp;2025</td><td>16:34:21&nbsp;26&nbsp;Apr&nbsp;2025</td><td>9&nbsp;minutes&nbsp;40&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>16:34:21&nbsp;26&nbsp;Apr&nbsp;2025</td><td>17:51:58&nbsp;26&nbsp;Apr&nbsp;2025</td><td>1&nbsp;hour&nbsp;17&nbsp;minutes&nbsp;37&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>17:51:58&nbsp;26&nbsp;Apr&nbsp;2025</td><td>17:54:40&nbsp;26&nbsp;Apr&nbsp;2025</td><td>2&nbsp;minutes&nbsp;42&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>17:54:40&nbsp;26&nbsp;Apr&nbsp;2025</td><td>17:59:39&nbsp;26&nbsp;Apr&nbsp;2025</td><td>4&nbsp;minutes&nbsp;59&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>17:59:39&nbsp;26&nbsp;Apr&nbsp;2025</td><td>17:59:45&nbsp;26&nbsp;Apr&nbsp;2025</td><td>6&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>17:59:45&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18:14:17&nbsp;26&nbsp;Apr&nbsp;2025</td><td>14&nbsp;minutes&nbsp;32&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>18:14:17&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18:22:41&nbsp;26&nbsp;Apr&nbsp;2025</td><td>8&nbsp;minutes&nbsp;24&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>18:22:41&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18:26:17&nbsp;26&nbsp;Apr&nbsp;2025</td><td>3&nbsp;minutes&nbsp;36&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>18:26:17&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18:29:31&nbsp;26&nbsp;Apr&nbsp;2025</td><td>3&nbsp;minutes&nbsp;14&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>18:29:31&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18:30:43&nbsp;26&nbsp;Apr&nbsp;2025</td><td>1&nbsp;minute&nbsp;12&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>18:30:43&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18:49:20&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18&nbsp;minutes&nbsp;37&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>18:49:20&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18:49:22&nbsp;26&nbsp;Apr&nbsp;2025</td><td>2&nbsp;seconds</td></tr>
<tr><td>Output</td><td>18:49:22&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18:49:38&nbsp;26&nbsp;Apr&nbsp;2025</td><td>16&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>18:49:22&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18:49:24&nbsp;26&nbsp;Apr&nbsp;2025</td><td>2&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>18:49:24&nbsp;26&nbsp;Apr&nbsp;2025</td><td>18:49:24&nbsp;26&nbsp;Apr&nbsp;2025</td><td>0&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>2&nbsp;hours&nbsp;24&nbsp;minutes&nbsp;59&nbsp;seconds</b></td>
</tr>
</table></body></html>