<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.39.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/BTF24T2_R1-trimmed.error_rates.pdf">errors</a></td><td>BTF24T2_R1-trimmed</td><td align="right">11,091,151</td><td align="right">1,659,523,433</td><td align="right">100.0%</td><td align="right">149.6&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">99.7%</td></tr><tr ><td><a href="calibration/BTF24T2_R2-trimmed.error_rates.pdf">errors</a></td><td>BTF24T2_R2-trimmed</td><td align="right">10,444,223</td><td align="right">1,561,837,305</td><td align="right">94.1%</td><td align="right">149.5&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">98.8%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>21,535,374</b></td><td align="right"><b>3,221,360,738</b></td><td align="right">97.1%</td><td align="right">149.6&nbsp;bases</td><td align="right"><b>151</b>&nbsp;bases</td><td align="right">99.2%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit relative_variance</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/contig_1.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>contig_1</td><td align="right">800,602</td><td align="center">599.5</td><td align="center">4.8</td><td align="center">15.0%</td><td>Escherichia coli BTF24T1 contig_1, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_2.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>contig_2</td><td align="right">677,962</td><td align="center">594.2</td><td align="center">5.8</td><td align="center">12.8%</td><td>Escherichia coli BTF24T1 contig_2, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_3.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>contig_3</td><td align="right">627,969</td><td align="center">600.5</td><td align="center">4.5</td><td align="center">11.6%</td><td>Escherichia coli BTF24T1 contig_3, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_4.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>contig_4</td><td align="right">369,842</td><td align="center">642.1</td><td align="center">4.5</td><td align="center">7.4%</td><td>Escherichia coli BTF24T1 contig_4, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_5.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>contig_5</td><td align="right">314,924</td><td align="center">634.5</td><td align="center">4.5</td><td align="center">6.2%</td><td>Escherichia coli BTF24T1 contig_5, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_6.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>contig_6</td><td align="right">287,306</td><td align="center">645.8</td><td align="center">4.4</td><td align="center">5.8%</td><td>Escherichia coli BTF24T1 contig_6, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_7.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>contig_7</td><td align="right">205,155</td><td align="center">628.2</td><td align="center">4.4</td><td align="center">4.0%</td><td>Escherichia coli BTF24T1 contig_7, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_8.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>contig_8</td><td align="right">204,487</td><td align="center">596.8</td><td align="center">3.7</td><td align="center">3.8%</td><td>Escherichia coli BTF24T1 contig_8, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_9.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>contig_9</td><td align="right">202,437</td><td align="center">628.0</td><td align="center">4.2</td><td align="center">4.0%</td><td>Escherichia coli BTF24T1 contig_9, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_10.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>contig_10</td><td align="right">193,774</td><td align="center">604.7</td><td align="center">4.6</td><td align="center">3.7%</td><td>Escherichia coli BTF24T1 contig_10, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_11.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>contig_11</td><td align="right">170,739</td><td align="center">655.8</td><td align="center">4.2</td><td align="center">3.5%</td><td>Escherichia coli BTF24T1 contig_11, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_12.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>contig_12</td><td align="right">152,390</td><td align="center">641.8</td><td align="center">4.3</td><td align="center">3.0%</td><td>Escherichia coli BTF24T1 contig_12, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_13.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>contig_13</td><td align="right">151,825</td><td align="center">618.0</td><td align="center">3.9</td><td align="center">2.9%</td><td>Escherichia coli BTF24T1 contig_13, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_14.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>contig_14</td><td align="right">128,097</td><td align="center">671.6</td><td align="center">4.4</td><td align="center">2.7%</td><td>Escherichia coli BTF24T1 contig_14, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_15.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>contig_15</td><td align="right">112,958</td><td align="center">633.2</td><td align="center">4.0</td><td align="center">2.2%</td><td>Escherichia coli BTF24T1 contig_15, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_16.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>contig_16</td><td align="right">107,365</td><td align="center">630.4</td><td align="center">4.8</td><td align="center">2.1%</td><td>Escherichia coli BTF24T1 contig_16, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_17.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>contig_17</td><td align="right">71,008</td><td align="center">658.2</td><td align="center">7.3</td><td align="center">1.4%</td><td>Escherichia coli BTF24T1 contig_17, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_18.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>contig_18</td><td align="right">41,345</td><td align="center">638.8</td><td align="center">4.0</td><td align="center">0.8%</td><td>Escherichia coli BTF24T1 contig_18, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_19.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>contig_19</td><td align="right">38,537</td><td align="center">730.1</td><td align="center">3.8</td><td align="center">0.4%</td><td>Escherichia coli BTF24T1 contig_19, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_20.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>contig_20</td><td align="right">33,369</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_20, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_21.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>contig_21</td><td align="right">29,791</td><td align="center">636.6</td><td align="center">5.6</td><td align="center">0.6%</td><td>Escherichia coli BTF24T1 contig_21, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_22.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>contig_22</td><td align="right">19,147</td><td align="center">691.8</td><td align="center">6.0</td><td align="center">0.4%</td><td>Escherichia coli BTF24T1 contig_22, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_23.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>contig_23</td><td align="right">17,015</td><td align="center">657.1</td><td align="center">3.3</td><td align="center">0.3%</td><td>Escherichia coli BTF24T1 contig_23, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_24.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>contig_24</td><td align="right">13,510</td><td align="center">634.9</td><td align="center">3.7</td><td align="center">0.3%</td><td>Escherichia coli BTF24T1 contig_24, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_25.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>contig_25</td><td align="right">10,543</td><td align="center">668.7</td><td align="center">2.8</td><td align="center">0.2%</td><td>Escherichia coli BTF24T1 contig_25, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_26.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>contig_26</td><td align="right">8,128</td><td align="center">5403.2</td><td align="center">9.6</td><td align="center">1.4%</td><td>Escherichia coli BTF24T1 contig_26, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_27.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>contig_27</td><td align="right">7,722</td><td align="center">649.4</td><td align="center">3.0</td><td align="center">0.2%</td><td>Escherichia coli BTF24T1 contig_27, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_28.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>contig_28</td><td align="right">7,674</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_28, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_29.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>contig_29</td><td align="right">7,326</td><td align="center">649.6</td><td align="center">4.0</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_29, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_30.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>contig_30</td><td align="right">6,220</td><td align="center">649.8</td><td align="center">4.7</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_30, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_31.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>contig_31</td><td align="right">5,635</td><td align="center">1227.8</td><td align="center">3.8</td><td align="center">0.2%</td><td>Escherichia coli BTF24T1 contig_31, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_32.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>contig_32</td><td align="right">4,819</td><td align="center">663.8</td><td align="center">2.9</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_32, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_33.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>contig_33</td><td align="right">4,242</td><td align="center">655.5</td><td align="center">2.9</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_33, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_34.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>contig_34</td><td align="right">3,350</td><td align="center">692.1</td><td align="center">5.5</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_34, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_35.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>contig_35</td><td align="right">3,247</td><td align="center">647.8</td><td align="center">2.9</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_35, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_36.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>contig_36</td><td align="right">3,245</td><td align="center">4753.1</td><td align="center">9.0</td><td align="center">0.5%</td><td>Escherichia coli BTF24T1 contig_36, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_37.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>contig_37</td><td align="right">3,027</td><td align="center">9743.1</td><td align="center">37.3</td><td align="center">0.9%</td><td>Escherichia coli BTF24T1 contig_37, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_38.overview.png">coverage</a></td><td><a href="calibration/37.unique_coverage.pdf">distribution</a></td><td>contig_38</td><td align="right">2,416</td><td align="center">712.8</td><td align="center">5.1</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_38, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_39.overview.png">coverage</a></td><td><a href="calibration/38.unique_coverage.pdf">distribution</a></td><td>contig_39</td><td align="right">1,887</td><td align="center">1969.8</td><td align="center">6.0</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_39, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_40.overview.png">coverage</a></td><td><a href="calibration/39.unique_coverage.pdf">distribution</a></td><td>contig_40</td><td align="right">1,801</td><td align="center">1353.1</td><td align="center">3.0</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_40, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_41.overview.png">coverage</a></td><td><a href="calibration/40.unique_coverage.pdf">distribution</a></td><td>contig_41</td><td align="right">1,673</td><td align="center">4551.5</td><td align="center">6.5</td><td align="center">0.2%</td><td>Escherichia coli BTF24T1 contig_41, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_42.overview.png">coverage</a></td><td><a href="calibration/41.unique_coverage.pdf">distribution</a></td><td>contig_42</td><td align="right">1,661</td><td align="center">687.5</td><td align="center">2.5</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_42, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_43.overview.png">coverage</a></td><td><a href="calibration/42.unique_coverage.pdf">distribution</a></td><td>contig_43</td><td align="right">1,651</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_43, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_44.overview.png">coverage</a></td><td><a href="calibration/43.unique_coverage.pdf">distribution</a></td><td>contig_44</td><td align="right">1,446</td><td align="center">1274.7</td><td align="center">3.0</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_44, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_45.overview.png">coverage</a></td><td><a href="calibration/44.unique_coverage.pdf">distribution</a></td><td>contig_45</td><td align="right">1,428</td><td align="center">1301.4</td><td align="center">4.6</td><td align="center">0.1%</td><td>Escherichia coli BTF24T1 contig_45, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_46.overview.png">coverage</a></td><td><a href="calibration/45.unique_coverage.pdf">distribution</a></td><td>contig_46</td><td align="right">1,405</td><td align="center">1389.8</td><td align="center">3.3</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_46, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_47.overview.png">coverage</a></td><td><a href="calibration/46.unique_coverage.pdf">distribution</a></td><td>contig_47</td><td align="right">1,329</td><td align="center">1347.8</td><td align="center">4.6</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_47, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_48.overview.png">coverage</a></td><td><a href="calibration/47.unique_coverage.pdf">distribution</a></td><td>contig_48</td><td align="right">1,218</td><td align="center">1162.8</td><td align="center">2.1</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_48, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_49.overview.png">coverage</a></td><td><a href="calibration/48.unique_coverage.pdf">distribution</a></td><td>contig_49</td><td align="right">1,114</td><td align="center">1227.3</td><td align="center">4.2</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_49, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_50.overview.png">coverage</a></td><td><a href="calibration/49.unique_coverage.pdf">distribution</a></td><td>contig_50</td><td align="right">999</td><td align="center">698.9</td><td align="center">2.5</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_50, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_51.overview.png">coverage</a></td><td><a href="calibration/50.unique_coverage.pdf">distribution</a></td><td>contig_51</td><td align="right">930</td><td align="center">1964.2</td><td align="center">1.2</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_51, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_52.overview.png">coverage</a></td><td><a href="calibration/51.unique_coverage.pdf">distribution</a></td><td>contig_52</td><td align="right">855</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_52, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_53.overview.png">coverage</a></td><td><a href="calibration/52.unique_coverage.pdf">distribution</a></td><td>contig_53</td><td align="right">849</td><td align="center">700.9</td><td align="center">1.9</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_53, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_54.overview.png">coverage</a></td><td><a href="calibration/53.unique_coverage.pdf">distribution</a></td><td>contig_54</td><td align="right">806</td><td align="center">1233.1</td><td align="center">1.8</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_54, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_55.overview.png">coverage</a></td><td><a href="calibration/54.unique_coverage.pdf">distribution</a></td><td>contig_55</td><td align="right">779</td><td align="center">691.3</td><td align="center">11.6</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_55, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_56.overview.png">coverage</a></td><td><a href="calibration/55.unique_coverage.pdf">distribution</a></td><td>contig_56</td><td align="right">779</td><td align="center">616.1</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_56, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_57.overview.png">coverage</a></td><td><a href="calibration/56.unique_coverage.pdf">distribution</a></td><td>contig_57</td><td align="right">728</td><td align="center">2718.4</td><td align="center">2.2</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_57, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_58.overview.png">coverage</a></td><td><a href="calibration/57.unique_coverage.pdf">distribution</a></td><td>contig_58</td><td align="right">671</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_58, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_59.overview.png">coverage</a></td><td><a href="calibration/58.unique_coverage.pdf">distribution</a></td><td>contig_59</td><td align="right">630</td><td align="center">725.1</td><td align="center">1.5</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_59, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_60.overview.png">coverage</a></td><td><a href="calibration/59.unique_coverage.pdf">distribution</a></td><td>contig_60</td><td align="right">582</td><td align="center">660.8</td><td align="center">1.3</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_60, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_61.overview.png">coverage</a></td><td><a href="calibration/60.unique_coverage.pdf">distribution</a></td><td>contig_61</td><td align="right">552</td><td align="center">735.4</td><td align="center">18.1</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_61, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_62.overview.png">coverage</a></td><td><a href="calibration/61.unique_coverage.pdf">distribution</a></td><td>contig_62</td><td align="right">550</td><td align="center">629.6</td><td align="center">2.0</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_62, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_63.overview.png">coverage</a></td><td><a href="calibration/62.unique_coverage.pdf">distribution</a></td><td>contig_63</td><td align="right">546</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_63, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_64.overview.png">coverage</a></td><td><a href="calibration/63.unique_coverage.pdf">distribution</a></td><td>contig_64</td><td align="right">540</td><td align="center">[440.1]</td><td align="center">[79.5]</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_64, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_65.overview.png">coverage</a></td><td><a href="calibration/64.unique_coverage.pdf">distribution</a></td><td>contig_65</td><td align="right">487</td><td align="center">642.7</td><td align="center">2.8</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_65, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_66.overview.png">coverage</a></td><td><a href="calibration/65.unique_coverage.pdf">distribution</a></td><td>contig_66</td><td align="right">504</td><td align="center">[437.6]</td><td align="center">[72.0]</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_66, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_67.overview.png">coverage</a></td><td><a href="calibration/66.unique_coverage.pdf">distribution</a></td><td>contig_67</td><td align="right">472</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_67, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_68.overview.png">coverage</a></td><td><a href="calibration/67.unique_coverage.pdf">distribution</a></td><td>contig_68</td><td align="right">439</td><td align="center">623.0</td><td align="center">5.8</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_68, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_69.overview.png">coverage</a></td><td><a href="calibration/68.unique_coverage.pdf">distribution</a></td><td>contig_69</td><td align="right">439</td><td align="center">571.7</td><td align="center">4.2</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_69, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_70.overview.png">coverage</a></td><td><a href="calibration/69.unique_coverage.pdf">distribution</a></td><td>contig_70</td><td align="right">435</td><td align="center">696.3</td><td align="center">2.4</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_70, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_71.overview.png">coverage</a></td><td><a href="calibration/70.unique_coverage.pdf">distribution</a></td><td>contig_71</td><td align="right">423</td><td align="center">[406.6]</td><td align="center">[94.9]</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_71, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_72.overview.png">coverage</a></td><td><a href="calibration/71.unique_coverage.pdf">distribution</a></td><td>contig_72</td><td align="right">419</td><td align="center">1329.2</td><td align="center">5.3</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_72, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_73.overview.png">coverage</a></td><td><a href="calibration/72.unique_coverage.pdf">distribution</a></td><td>contig_73</td><td align="right">325</td><td align="center">[2657.2]</td><td align="center">[0.2]</td><td align="center">0.0%</td><td>Escherichia coli BTF24T1 contig_73, whole genome shotgun sequence.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>5,080,500</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit relative_variance</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="reject_table_row">Fit failed</span> Negative binomial fit failed for this reference sequence. It may have an unusual coverage depth distribution. JC and MC predictions may be less accurate.
<p><span class="gray_table_row">Insufficient coverage</span> Reference sequence counted as entirely deleted due to low coverage. Try either the <code>-t,--targeted-sequencing</code> or the <code>-c,--contig-reference </code> option if you want mutations called for these reference sequences.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">49074</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">697</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.042</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>contig_1</td><td align="center">0.30677</td></tr><tr >
<td>contig_2</td><td align="center">0.30940</td></tr><tr >
<td>contig_3</td><td align="center">0.32079</td></tr><tr >
<td>contig_4</td><td align="center">0.28781</td></tr><tr >
<td>contig_5</td><td align="center">0.28910</td></tr><tr >
<td>contig_6</td><td align="center">0.28483</td></tr><tr >
<td>contig_7</td><td align="center">0.29407</td></tr><tr >
<td>contig_8</td><td align="center">0.31118</td></tr><tr >
<td>contig_9</td><td align="center">0.29303</td></tr><tr >
<td>contig_10</td><td align="center">0.30526</td></tr><tr >
<td>contig_11</td><td align="center">0.28042</td></tr><tr >
<td>contig_12</td><td align="center">0.28896</td></tr><tr >
<td>contig_13</td><td align="center">0.30140</td></tr><tr >
<td>contig_14</td><td align="center">0.27941</td></tr><tr >
<td>contig_15</td><td align="center">0.29079</td></tr><tr >
<td>contig_16</td><td align="center">0.29158</td></tr><tr >
<td>contig_17</td><td align="center">0.28058</td></tr><tr >
<td>contig_18</td><td align="center">0.28909</td></tr><tr >
<td>contig_19</td><td align="center">0.70467</td></tr><tr >
<td>contig_20</td><td align="center">NA</td></tr><tr >
<td>contig_21</td><td align="center">0.28616</td></tr><tr >
<td>contig_22</td><td align="center">0.27739</td></tr><tr >
<td>contig_23</td><td align="center">0.28071</td></tr><tr >
<td>contig_24</td><td align="center">0.29678</td></tr><tr >
<td>contig_25</td><td align="center">0.29375</td></tr><tr >
<td>contig_26</td><td align="center">0.01432</td></tr><tr >
<td>contig_27</td><td align="center">0.30331</td></tr><tr >
<td>contig_28</td><td align="center">NA</td></tr><tr >
<td>contig_29</td><td align="center">0.29655</td></tr><tr >
<td>contig_30</td><td align="center">0.30707</td></tr><tr >
<td>contig_31</td><td align="center">0.13390</td></tr><tr >
<td>contig_32</td><td align="center">0.29313</td></tr><tr >
<td>contig_33</td><td align="center">0.30882</td></tr><tr >
<td>contig_34</td><td align="center">0.30828</td></tr><tr >
<td>contig_35</td><td align="center">0.28902</td></tr><tr >
<td>contig_36</td><td align="center">0.00451</td></tr><tr >
<td>contig_37</td><td align="center">0.01306</td></tr><tr >
<td>contig_38</td><td align="center">0.29905</td></tr><tr >
<td>contig_39</td><td align="center">0.11447</td></tr><tr >
<td>contig_40</td><td align="center">0.17962</td></tr><tr >
<td>contig_41</td><td align="center">0.07980</td></tr><tr >
<td>contig_42</td><td align="center">0.30734</td></tr><tr >
<td>contig_43</td><td align="center">NA</td></tr><tr >
<td>contig_44</td><td align="center">0.18707</td></tr><tr >
<td>contig_45</td><td align="center">0.19433</td></tr><tr >
<td>contig_46</td><td align="center">0.24285</td></tr><tr >
<td>contig_47</td><td align="center">0.18360</td></tr><tr >
<td>contig_48</td><td align="center">0.21634</td></tr><tr >
<td>contig_49</td><td align="center">0.23429</td></tr><tr >
<td>contig_50</td><td align="center">0.35128</td></tr><tr >
<td>contig_51</td><td align="center">0.05018</td></tr><tr >
<td>contig_52</td><td align="center">NA</td></tr><tr >
<td>contig_53</td><td align="center">0.38751</td></tr><tr >
<td>contig_54</td><td align="center">0.24876</td></tr><tr >
<td>contig_55</td><td align="center">0.39923</td></tr><tr >
<td>contig_56</td><td align="center">0.41977</td></tr><tr >
<td>contig_57</td><td align="center">0.21473</td></tr><tr >
<td>contig_58</td><td align="center">NA</td></tr><tr >
<td>contig_59</td><td align="center">0.39841</td></tr><tr >
<td>contig_60</td><td align="center">0.54100</td></tr><tr >
<td>contig_61</td><td align="center">0.47796</td></tr><tr >
<td>contig_62</td><td align="center">0.48881</td></tr><tr >
<td>contig_63</td><td align="center">NA</td></tr><tr >
<td>contig_64</td><td align="center">NA</td></tr><tr >
<td>contig_65</td><td align="center">0.48255</td></tr><tr >
<td>contig_66</td><td align="center">NA</td></tr><tr >
<td>contig_67</td><td align="center">NA</td></tr><tr >
<td>contig_68</td><td align="center">0.37640</td></tr><tr >
<td>contig_69</td><td align="center">0.50562</td></tr><tr >
<td>contig_70</td><td align="center">0.57792</td></tr><tr >
<td>contig_71</td><td align="center">NA</td></tr><tr >
<td>contig_72</td><td align="center">0.37351</td></tr><tr >
<td>contig_73</td><td align="center">NA</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.5.4</td></tr>
<tr><td>R</td><td>4.4.1</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>09:19:09&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:23:07&nbsp;28&nbsp;Feb&nbsp;2025</td><td>3&nbsp;minutes&nbsp;58&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>09:23:07&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:27:18&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;minutes&nbsp;11&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>09:27:18&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:30:08&nbsp;28&nbsp;Feb&nbsp;2025</td><td>2&nbsp;minutes&nbsp;50&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>09:30:08&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:37:51&nbsp;28&nbsp;Feb&nbsp;2025</td><td>7&nbsp;minutes&nbsp;43&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>09:37:51&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:38:40&nbsp;28&nbsp;Feb&nbsp;2025</td><td>49&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>09:38:40&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:40:19&nbsp;28&nbsp;Feb&nbsp;2025</td><td>1&nbsp;minute&nbsp;39&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>09:40:19&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:45:27&nbsp;28&nbsp;Feb&nbsp;2025</td><td>5&nbsp;minutes&nbsp;8&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>09:45:27&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:50:34&nbsp;28&nbsp;Feb&nbsp;2025</td><td>5&nbsp;minutes&nbsp;7&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>09:50:34&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:56:03&nbsp;28&nbsp;Feb&nbsp;2025</td><td>5&nbsp;minutes&nbsp;29&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>09:56:03&nbsp;28&nbsp;Feb&nbsp;2025</td><td>09:57:33&nbsp;28&nbsp;Feb&nbsp;2025</td><td>1&nbsp;minute&nbsp;30&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>09:57:33&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:27:09&nbsp;28&nbsp;Feb&nbsp;2025</td><td>29&nbsp;minutes&nbsp;36&nbsp;seconds</td></tr>
<tr><td>Polymorphism statistics</td><td>10:27:09&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:27:10&nbsp;28&nbsp;Feb&nbsp;2025</td><td>1&nbsp;second</td></tr>
<tr><td>Output</td><td>10:27:10&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:27:24&nbsp;28&nbsp;Feb&nbsp;2025</td><td>14&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>10:27:10&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:27:14&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>10:27:14&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:27:14&nbsp;28&nbsp;Feb&nbsp;2025</td><td>0&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>1&nbsp;hour&nbsp;8&nbsp;minutes&nbsp;19&nbsp;seconds</b></td>
</tr>
</table></body></html>