<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Summary Statistics</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.39.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<h2>Read File Information</h2>
<table border="0" cellspace="1" cellpadding="5">
<tr ><th></th><th>read file</th><th>reads</th><th>bases</th><th>passed&nbsp;filters</th><th>average</th><th>longest</th><th>mapped</th></tr>
<tr ><td><a href="calibration/BTF69T-1_R1-trimmed.error_rates.pdf">errors</a></td><td>BTF69T-1_R1-trimmed</td><td align="right">9,726,145</td><td align="right">1,459,730,049</td><td align="right">100.0%</td><td align="right">150.1&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">99.8%</td></tr><tr ><td><a href="calibration/BTF69T-1_R2-trimmed.error_rates.pdf">errors</a></td><td>BTF69T-1_R2-trimmed</td><td align="right">8,957,755</td><td align="right">1,343,703,159</td><td align="right">92.1%</td><td align="right">150.0&nbsp;bases</td><td align="right">151&nbsp;bases</td><td align="right">98.6%</td></tr><tr class="highlight_table_row"><td></td><td><b>total</b></td><td align="right"><b>18,683,900</b></td><td align="right"><b>2,803,433,208</b></td><td align="right">96.0%</td><td align="right">150.0&nbsp;bases</td><td align="right"><b>151</b>&nbsp;bases</td><td align="right">99.2%</td></tr></table><h2>Reference Sequence Information</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5" >
<tr><th></th><th></th><th>seq id</th><th>length</th><th align="center">fit mean</th><th align="center">fit relative_variance</th><th align="center">% mapped reads</th><th align="left">description</th></tr>
<tr><td><a href="evidence/contig_1.overview.png">coverage</a></td><td><a href="calibration/0.unique_coverage.pdf">distribution</a></td><td>contig_1</td><td align="right">622,859</td><td align="center">500.7</td><td align="center">4.7</td><td align="center">11.2%</td><td>Escherichia coli BTF69T-2 contig_1, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_2.overview.png">coverage</a></td><td><a href="calibration/1.unique_coverage.pdf">distribution</a></td><td>contig_2</td><td align="right">622,077</td><td align="center">500.3</td><td align="center">4.8</td><td align="center">11.2%</td><td>Escherichia coli BTF69T-2 contig_2, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_3.overview.png">coverage</a></td><td><a href="calibration/2.unique_coverage.pdf">distribution</a></td><td>contig_3</td><td align="right">469,135</td><td align="center">501.3</td><td align="center">4.8</td><td align="center">8.5%</td><td>Escherichia coli BTF69T-2 contig_3, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_4.overview.png">coverage</a></td><td><a href="calibration/3.unique_coverage.pdf">distribution</a></td><td>contig_4</td><td align="right">387,242</td><td align="center">506.8</td><td align="center">4.3</td><td align="center">7.1%</td><td>Escherichia coli BTF69T-2 contig_4, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_5.overview.png">coverage</a></td><td><a href="calibration/4.unique_coverage.pdf">distribution</a></td><td>contig_5</td><td align="right">377,071</td><td align="center">513.1</td><td align="center">4.2</td><td align="center">7.0%</td><td>Escherichia coli BTF69T-2 contig_5, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_6.overview.png">coverage</a></td><td><a href="calibration/5.unique_coverage.pdf">distribution</a></td><td>contig_6</td><td align="right">336,053</td><td align="center">513.2</td><td align="center">5.5</td><td align="center">6.2%</td><td>Escherichia coli BTF69T-2 contig_6, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_7.overview.png">coverage</a></td><td><a href="calibration/6.unique_coverage.pdf">distribution</a></td><td>contig_7</td><td align="right">333,620</td><td align="center">510.4</td><td align="center">4.8</td><td align="center">6.1%</td><td>Escherichia coli BTF69T-2 contig_7, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_8.overview.png">coverage</a></td><td><a href="calibration/7.unique_coverage.pdf">distribution</a></td><td>contig_8</td><td align="right">307,923</td><td align="center">502.8</td><td align="center">5.2</td><td align="center">5.6%</td><td>Escherichia coli BTF69T-2 contig_8, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_9.overview.png">coverage</a></td><td><a href="calibration/8.unique_coverage.pdf">distribution</a></td><td>contig_9</td><td align="right">275,941</td><td align="center">515.8</td><td align="center">4.7</td><td align="center">5.1%</td><td>Escherichia coli BTF69T-2 contig_9, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_10.overview.png">coverage</a></td><td><a href="calibration/9.unique_coverage.pdf">distribution</a></td><td>contig_10</td><td align="right">191,115</td><td align="center">504.4</td><td align="center">4.7</td><td align="center">3.5%</td><td>Escherichia coli BTF69T-2 contig_10, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_11.overview.png">coverage</a></td><td><a href="calibration/10.unique_coverage.pdf">distribution</a></td><td>contig_11</td><td align="right">190,808</td><td align="center">509.3</td><td align="center">4.5</td><td align="center">3.5%</td><td>Escherichia coli BTF69T-2 contig_11, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_12.overview.png">coverage</a></td><td><a href="calibration/11.unique_coverage.pdf">distribution</a></td><td>contig_12</td><td align="right">173,576</td><td align="center">513.9</td><td align="center">4.7</td><td align="center">3.2%</td><td>Escherichia coli BTF69T-2 contig_12, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_13.overview.png">coverage</a></td><td><a href="calibration/12.unique_coverage.pdf">distribution</a></td><td>contig_13</td><td align="right">156,260</td><td align="center">514.0</td><td align="center">4.5</td><td align="center">2.9%</td><td>Escherichia coli BTF69T-2 contig_13, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_14.overview.png">coverage</a></td><td><a href="calibration/13.unique_coverage.pdf">distribution</a></td><td>contig_14</td><td align="right">109,397</td><td align="center">806.5</td><td align="center">6.2</td><td align="center">3.2%</td><td>Escherichia coli BTF69T-2 contig_14, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_15.overview.png">coverage</a></td><td><a href="calibration/14.unique_coverage.pdf">distribution</a></td><td>contig_15</td><td align="right">108,987</td><td align="center">510.2</td><td align="center">4.2</td><td align="center">2.0%</td><td>Escherichia coli BTF69T-2 contig_15, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_16.overview.png">coverage</a></td><td><a href="calibration/15.unique_coverage.pdf">distribution</a></td><td>contig_16</td><td align="right">103,792</td><td align="center">518.2</td><td align="center">7.4</td><td align="center">1.9%</td><td>Escherichia coli BTF69T-2 contig_16, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_17.overview.png">coverage</a></td><td><a href="calibration/16.unique_coverage.pdf">distribution</a></td><td>contig_17</td><td align="right">53,255</td><td align="center">494.2</td><td align="center">3.8</td><td align="center">0.9%</td><td>Escherichia coli BTF69T-2 contig_17, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_18.overview.png">coverage</a></td><td><a href="calibration/17.unique_coverage.pdf">distribution</a></td><td>contig_18</td><td align="right">41,392</td><td align="center">531.5</td><td align="center">4.6</td><td align="center">0.8%</td><td>Escherichia coli BTF69T-2 contig_18, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_19.overview.png">coverage</a></td><td><a href="calibration/18.unique_coverage.pdf">distribution</a></td><td>contig_19</td><td align="right">40,293</td><td align="center">536.3</td><td align="center">5.4</td><td align="center">0.8%</td><td>Escherichia coli BTF69T-2 contig_19, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_20.overview.png">coverage</a></td><td><a href="calibration/19.unique_coverage.pdf">distribution</a></td><td>contig_20</td><td align="right">29,791</td><td align="center">507.3</td><td align="center">6.7</td><td align="center">0.5%</td><td>Escherichia coli BTF69T-2 contig_20, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_21.overview.png">coverage</a></td><td><a href="calibration/20.unique_coverage.pdf">distribution</a></td><td>contig_21</td><td align="right">24,499</td><td align="center">980.2</td><td align="center">4.9</td><td align="center">0.9%</td><td>Escherichia coli BTF69T-2 contig_21, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_22.overview.png">coverage</a></td><td><a href="calibration/21.unique_coverage.pdf">distribution</a></td><td>contig_22</td><td align="right">21,996</td><td align="center">551.1</td><td align="center">4.9</td><td align="center">0.4%</td><td>Escherichia coli BTF69T-2 contig_22, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_23.overview.png">coverage</a></td><td><a href="calibration/22.unique_coverage.pdf">distribution</a></td><td>contig_23</td><td align="right">20,179</td><td align="center">538.5</td><td align="center">4.0</td><td align="center">0.4%</td><td>Escherichia coli BTF69T-2 contig_23, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_24.overview.png">coverage</a></td><td><a href="calibration/23.unique_coverage.pdf">distribution</a></td><td>contig_24</td><td align="right">20,177</td><td align="center">571.2</td><td align="center">4.6</td><td align="center">0.4%</td><td>Escherichia coli BTF69T-2 contig_24, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_25.overview.png">coverage</a></td><td><a href="calibration/24.unique_coverage.pdf">distribution</a></td><td>contig_25</td><td align="right">16,053</td><td align="center">929.2</td><td align="center">20.2</td><td align="center">0.5%</td><td>Escherichia coli BTF69T-2 contig_25, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_26.overview.png">coverage</a></td><td><a href="calibration/25.unique_coverage.pdf">distribution</a></td><td>contig_26</td><td align="right">13,795</td><td align="center">960.8</td><td align="center">12.9</td><td align="center">0.5%</td><td>Escherichia coli BTF69T-2 contig_26, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_27.overview.png">coverage</a></td><td><a href="calibration/26.unique_coverage.pdf">distribution</a></td><td>contig_27</td><td align="right">12,460</td><td align="center">515.7</td><td align="center">5.6</td><td align="center">0.2%</td><td>Escherichia coli BTF69T-2 contig_27, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_28.overview.png">coverage</a></td><td><a href="calibration/27.unique_coverage.pdf">distribution</a></td><td>contig_28</td><td align="right">10,937</td><td align="center">495.3</td><td align="center">4.4</td><td align="center">0.2%</td><td>Escherichia coli BTF69T-2 contig_28, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_29.overview.png">coverage</a></td><td><a href="calibration/28.unique_coverage.pdf">distribution</a></td><td>contig_29</td><td align="right">10,522</td><td align="center">562.8</td><td align="center">3.4</td><td align="center">0.2%</td><td>Escherichia coli BTF69T-2 contig_29, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_30.overview.png">coverage</a></td><td><a href="calibration/29.unique_coverage.pdf">distribution</a></td><td>contig_30</td><td align="right">6,245</td><td align="center">526.4</td><td align="center">3.6</td><td align="center">0.1%</td><td>Escherichia coli BTF69T-2 contig_30, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_31.overview.png">coverage</a></td><td><a href="calibration/30.unique_coverage.pdf">distribution</a></td><td>contig_31</td><td align="right">6,097</td><td align="center">926.2</td><td align="center">7.0</td><td align="center">0.2%</td><td>Escherichia coli BTF69T-2 contig_31, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_32.overview.png">coverage</a></td><td><a href="calibration/31.unique_coverage.pdf">distribution</a></td><td>contig_32</td><td align="right">5,167</td><td align="center">12298.4</td><td align="center">132.4</td><td align="center">2.2%</td><td>Escherichia coli BTF69T-2 contig_32, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_33.overview.png">coverage</a></td><td><a href="calibration/32.unique_coverage.pdf">distribution</a></td><td>contig_33</td><td align="right">4,873</td><td align="center">558.1</td><td align="center">2.6</td><td align="center">0.1%</td><td>Escherichia coli BTF69T-2 contig_33, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_34.overview.png">coverage</a></td><td><a href="calibration/33.unique_coverage.pdf">distribution</a></td><td>contig_34</td><td align="right">4,123</td><td align="center">557.3</td><td align="center">4.0</td><td align="center">0.1%</td><td>Escherichia coli BTF69T-2 contig_34, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_35.overview.png">coverage</a></td><td><a href="calibration/34.unique_coverage.pdf">distribution</a></td><td>contig_35</td><td align="right">3,300</td><td align="center">987.0</td><td align="center">4.4</td><td align="center">0.1%</td><td>Escherichia coli BTF69T-2 contig_35, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_36.overview.png">coverage</a></td><td><a href="calibration/35.unique_coverage.pdf">distribution</a></td><td>contig_36</td><td align="right">3,169</td><td align="center">3879.4</td><td align="center">3.5</td><td align="center">0.4%</td><td>Escherichia coli BTF69T-2 contig_36, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_37.overview.png">coverage</a></td><td><a href="calibration/36.unique_coverage.pdf">distribution</a></td><td>contig_37</td><td align="right">2,583</td><td align="center">1178.0</td><td align="center">4.8</td><td align="center">0.1%</td><td>Escherichia coli BTF69T-2 contig_37, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_38.overview.png">coverage</a></td><td><a href="calibration/37.unique_coverage.pdf">distribution</a></td><td>contig_38</td><td align="right">2,532</td><td align="center">3727.3</td><td align="center">3.2</td><td align="center">0.3%</td><td>Escherichia coli BTF69T-2 contig_38, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_39.overview.png">coverage</a></td><td><a href="calibration/38.unique_coverage.pdf">distribution</a></td><td>contig_39</td><td align="right">2,354</td><td align="center">565.3</td><td align="center">12.0</td><td align="center">0.1%</td><td>Escherichia coli BTF69T-2 contig_39, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_40.overview.png">coverage</a></td><td><a href="calibration/39.unique_coverage.pdf">distribution</a></td><td>contig_40</td><td align="right">2,101</td><td align="center">8888.4</td><td align="center">38.4</td><td align="center">0.6%</td><td>Escherichia coli BTF69T-2 contig_40, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_41.overview.png">coverage</a></td><td><a href="calibration/40.unique_coverage.pdf">distribution</a></td><td>contig_41</td><td align="right">2,069</td><td align="center">571.9</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_41, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_42.overview.png">coverage</a></td><td><a href="calibration/41.unique_coverage.pdf">distribution</a></td><td>contig_42</td><td align="right">1,885</td><td align="center">1050.3</td><td align="center">5.4</td><td align="center">0.1%</td><td>Escherichia coli BTF69T-2 contig_42, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_43.overview.png">coverage</a></td><td><a href="calibration/42.unique_coverage.pdf">distribution</a></td><td>contig_43</td><td align="right">1,577</td><td align="center">3697.2</td><td align="center">9.8</td><td align="center">0.2%</td><td>Escherichia coli BTF69T-2 contig_43, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_44.overview.png">coverage</a></td><td><a href="calibration/43.unique_coverage.pdf">distribution</a></td><td>contig_44</td><td align="right">1,457</td><td align="center">1562.7</td><td align="center">1.3</td><td align="center">0.1%</td><td>Escherichia coli BTF69T-2 contig_44, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_45.overview.png">coverage</a></td><td><a href="calibration/44.unique_coverage.pdf">distribution</a></td><td>contig_45</td><td align="right">1,271</td><td align="center">944.2</td><td align="center">4.8</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_45, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_46.overview.png">coverage</a></td><td><a href="calibration/45.unique_coverage.pdf">distribution</a></td><td>contig_46</td><td align="right">1,259</td><td align="center">576.3</td><td align="center">1.3</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_46, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_47.overview.png">coverage</a></td><td><a href="calibration/46.unique_coverage.pdf">distribution</a></td><td>contig_47</td><td align="right">1,209</td><td align="center">530.9</td><td align="center">2.7</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_47, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_48.overview.png">coverage</a></td><td><a href="calibration/47.unique_coverage.pdf">distribution</a></td><td>contig_48</td><td align="right">1,150</td><td align="center">935.6</td><td align="center">1.9</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_48, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_49.overview.png">coverage</a></td><td><a href="calibration/48.unique_coverage.pdf">distribution</a></td><td>contig_49</td><td align="right">1,030</td><td align="center">1021.2</td><td align="center">1.5</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_49, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_50.overview.png">coverage</a></td><td><a href="calibration/49.unique_coverage.pdf">distribution</a></td><td>contig_50</td><td align="right">1,018</td><td align="center">1548.6</td><td align="center">12.7</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_50, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_51.overview.png">coverage</a></td><td><a href="calibration/50.unique_coverage.pdf">distribution</a></td><td>contig_51</td><td align="right">905</td><td align="center">1089.4</td><td align="center">1.9</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_51, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_52.overview.png">coverage</a></td><td><a href="calibration/51.unique_coverage.pdf">distribution</a></td><td>contig_52</td><td align="right">821</td><td align="center">1917.3</td><td align="center">5.3</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_52, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_53.overview.png">coverage</a></td><td><a href="calibration/52.unique_coverage.pdf">distribution</a></td><td>contig_53</td><td align="right">674</td><td align="center">1381.2</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_53, whole genome shotgun sequence.</td></tr><tr class="gray_table_row"><td><a href="evidence/contig_54.overview.png">coverage</a></td><td><a href="calibration/53.unique_coverage.pdf">distribution</a></td><td>contig_54</td><td align="right">576</td><td align="center">NA</td><td align="center">NA</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_54, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_55.overview.png">coverage</a></td><td><a href="calibration/54.unique_coverage.pdf">distribution</a></td><td>contig_55</td><td align="right">539</td><td align="center">1602.7</td><td align="center">1.6</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_55, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_56.overview.png">coverage</a></td><td><a href="calibration/55.unique_coverage.pdf">distribution</a></td><td>contig_56</td><td align="right">511</td><td align="center">525.2</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_56, whole genome shotgun sequence.</td></tr><tr class="reject_table_row"><td><a href="evidence/contig_57.overview.png">coverage</a></td><td><a href="calibration/56.unique_coverage.pdf">distribution</a></td><td>contig_57</td><td align="right">417</td><td align="center">[345.5]</td><td align="center">[129.3]</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_57, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_58.overview.png">coverage</a></td><td><a href="calibration/57.unique_coverage.pdf">distribution</a></td><td>contig_58</td><td align="right">386</td><td align="center">1514.2</td><td align="center">1.3</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_58, whole genome shotgun sequence.</td></tr><tr><td><a href="evidence/contig_59.overview.png">coverage</a></td><td><a href="calibration/58.unique_coverage.pdf">distribution</a></td><td>contig_59</td><td align="right">306</td><td align="center">986.0</td><td align="center">1.0</td><td align="center">0.0%</td><td>Escherichia coli BTF69T-2 contig_59, whole genome shotgun sequence.</td></tr><tr class="highlight_table_row"><td></td><td></td><td><b>total</b></td><td align="right"><b>5,142,809</b></td><td></td><td></td><td align="center">100.0%</td><td></td></tr>
</table>
<p><b>fit relative_variance</b> is the ratio of the variance to the mean for the negative binomial fit. It is =1 for Poisson and >1 for over-dispersed data.
<p><span class="reject_table_row">Fit failed</span> Negative binomial fit failed for this reference sequence. It may have an unusual coverage depth distribution. JC and MC predictions may be less accurate.
<p><span class="gray_table_row">Insufficient coverage</span> Reference sequence counted as entirely deleted due to low coverage. Try either the <code>-t,--targeted-sequencing</code> or the <code>-c,--contig-reference </code> option if you want mutations called for these reference sequences.
<h2>New Junction Evidence</h2>
<p>
<h3>Junction Candidates Tested</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">limit</th><th align="center">actual</th></tr><tr><td>Number of alignment pairs examined for constructing junction candidates</td><td align="center">&le;&nbsp;100000</td><td align="center">55386</td></tr><tr><td>Coverage evenness (position-hash) score of junction candidates</td><td align="center">&ge;&nbsp;2</td><td align="center">&ge;&nbsp;2</td></tr><tr><td>Test this many junction candidates (n). May be smaller if not enough passed the coverage evenness threshold</td><td align="center">100&nbsp;&le;&nbsp;n&nbsp;&le;&nbsp;5000</td><td align="center">616</td></tr><tr><td>Total length of all junction candidates (factor times the reference genome length)</td><td align="center">&le;&nbsp;0.1</td><td align="center">0.037</td></tr></table>
<h3>Junction Skew Score Calculation</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr ><th>reference sequence</th><th>pr(no read start)</th></tr>
<tr >
<td>contig_1</td><td align="center">0.37640</td></tr><tr >
<td>contig_2</td><td align="center">0.37683</td></tr><tr >
<td>contig_3</td><td align="center">0.37880</td></tr><tr >
<td>contig_4</td><td align="center">0.37542</td></tr><tr >
<td>contig_5</td><td align="center">0.37036</td></tr><tr >
<td>contig_6</td><td align="center">0.36971</td></tr><tr >
<td>contig_7</td><td align="center">0.37092</td></tr><tr >
<td>contig_8</td><td align="center">0.37582</td></tr><tr >
<td>contig_9</td><td align="center">0.36873</td></tr><tr >
<td>contig_10</td><td align="center">0.37600</td></tr><tr >
<td>contig_11</td><td align="center">0.37161</td></tr><tr >
<td>contig_12</td><td align="center">0.37035</td></tr><tr >
<td>contig_13</td><td align="center">0.37008</td></tr><tr >
<td>contig_14</td><td align="center">0.23775</td></tr><tr >
<td>contig_15</td><td align="center">0.37066</td></tr><tr >
<td>contig_16</td><td align="center">0.37109</td></tr><tr >
<td>contig_17</td><td align="center">0.38290</td></tr><tr >
<td>contig_18</td><td align="center">0.35804</td></tr><tr >
<td>contig_19</td><td align="center">0.35489</td></tr><tr >
<td>contig_20</td><td align="center">0.37046</td></tr><tr >
<td>contig_21</td><td align="center">0.18451</td></tr><tr >
<td>contig_22</td><td align="center">0.35388</td></tr><tr >
<td>contig_23</td><td align="center">0.35782</td></tr><tr >
<td>contig_24</td><td align="center">0.34197</td></tr><tr >
<td>contig_25</td><td align="center">0.21400</td></tr><tr >
<td>contig_26</td><td align="center">0.19351</td></tr><tr >
<td>contig_27</td><td align="center">0.36613</td></tr><tr >
<td>contig_28</td><td align="center">0.37806</td></tr><tr >
<td>contig_29</td><td align="center">0.35288</td></tr><tr >
<td>contig_30</td><td align="center">0.37790</td></tr><tr >
<td>contig_31</td><td align="center">0.22191</td></tr><tr >
<td>contig_32</td><td align="center">0.02468</td></tr><tr >
<td>contig_33</td><td align="center">0.37527</td></tr><tr >
<td>contig_34</td><td align="center">0.37711</td></tr><tr >
<td>contig_35</td><td align="center">0.20864</td></tr><tr >
<td>contig_36</td><td align="center">0.01472</td></tr><tr >
<td>contig_37</td><td align="center">0.18293</td></tr><tr >
<td>contig_38</td><td align="center">0.06812</td></tr><tr >
<td>contig_39</td><td align="center">0.31287</td></tr><tr >
<td>contig_40</td><td align="center">0.05759</td></tr><tr >
<td>contig_41</td><td align="center">0.34457</td></tr><tr >
<td>contig_42</td><td align="center">0.22202</td></tr><tr >
<td>contig_43</td><td align="center">0.08973</td></tr><tr >
<td>contig_44</td><td align="center">0.16438</td></tr><tr >
<td>contig_45</td><td align="center">0.28009</td></tr><tr >
<td>contig_46</td><td align="center">0.41342</td></tr><tr >
<td>contig_47</td><td align="center">0.42266</td></tr><tr >
<td>contig_48</td><td align="center">0.29634</td></tr><tr >
<td>contig_49</td><td align="center">0.27379</td></tr><tr >
<td>contig_50</td><td align="center">0.20339</td></tr><tr >
<td>contig_51</td><td align="center">0.28343</td></tr><tr >
<td>contig_52</td><td align="center">0.21011</td></tr><tr >
<td>contig_53</td><td align="center">0.28932</td></tr><tr >
<td>contig_54</td><td align="center">NA</td></tr><tr >
<td>contig_55</td><td align="center">0.30705</td></tr><tr >
<td>contig_56</td><td align="center">0.73349</td></tr><tr >
<td>contig_57</td><td align="center">NA</td></tr><tr >
<td>contig_58</td><td align="center">0.38860</td></tr><tr >
<td>contig_59</td><td align="center">0.60909</td></tr></table><p><b>pr(no read start)</b> is the probability that there will not be an aligned read whose first base matches a given position on a given strand.
<h3>Final Junction Predictions</h3>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th align="center">option</th><th align="center">value</th></tr><tr><td>Coverage evenness (position-hash) score of predicted junctions must be</td><td align="center">&ge;&nbsp;3</td></tr><tr><td>Minimum probablilty assigned that no mapped read will start at a given position and strand for junction prediction</td><td align="center">0.1</td></tr><tr><td>Junction allow suboptimal matches</td><td align="center">FALSE</td></tr><tr><td>Skew score of predicted junction (&minus;log10 probability of unusual coverage evenness) must be</td><td align="center">&le;&nbsp;3</td></tr><tr><td>Number of bases that at least one read must overlap each uniquely aligned side of a predicted junction</td><td align="center">&ge;&nbsp;1</td></tr></table>
<h2>Read Alignment Evidence</h2>
<p>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>option</th><th>value</th></tr><tr><td>Mode</td><td>Consensus/Mixed Base</td></tr><tr><td>Ploidy</td><td>1 (haploid)</td></tr><tr><td>Consensus mutation E-value cutoff</td><td>10</td></tr><tr><td>Consensus frequency cutoff</td><td>0.8</td></tr><tr><td>Consensus minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Consensus minimum variant coverage</td><td>OFF</td></tr><tr><td>Consensus minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism E-value cutoff</td><td>10</td></tr><tr><td>Polymorphism frequency cutoff</td><td>0.2</td></tr><tr><td>Polymorphism minimum variant coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage each strand</td><td>OFF</td></tr><tr><td>Polymorphism minimum variant coverage</td><td>OFF</td></tr><tr><td>Polymorphism minimum total coverage</td><td>OFF</td></tr><tr><td>Polymorphism bias cutoff</td><td>OFF</td></tr><tr><td>Predict indel polymorphisms</td><td>YES</td></tr><tr><td>Skip indel polymorphisms in homopolymers runs of</td><td>OFF</td></tr><tr><td>Skip base substitutions when they create a homopolymer flanked on each side by</td><td>OFF</td></tr></table><p>
<h2>Software Versions</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>program</th><th>version</th></tr>
<tr><td>bowtie2</td><td>2.5.4</td></tr>
<tr><td>R</td><td>4.4.1</td></tr>
</table><p>
<h2>Execution Times</h2>
<table border="0" cellspacing="1" cellpadding="5">
<tr><th>step</th><th>start</th><th>end</th><th>elapsed</th></tr>
<tr><td>Read and reference sequence file input</td><td>10:28:18&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:31:44&nbsp;28&nbsp;Feb&nbsp;2025</td><td>3&nbsp;minutes&nbsp;26&nbsp;seconds</td></tr>
<tr><td>Read alignment to reference genome</td><td>10:31:44&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:36:13&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;minutes&nbsp;29&nbsp;seconds</td></tr>
<tr><td>Preprocessing alignments for candidate junction identification</td><td>10:36:13&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:38:44&nbsp;28&nbsp;Feb&nbsp;2025</td><td>2&nbsp;minutes&nbsp;31&nbsp;seconds</td></tr>
<tr><td>Preliminary analysis of coverage distribution</td><td>10:38:44&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:45:09&nbsp;28&nbsp;Feb&nbsp;2025</td><td>6&nbsp;minutes&nbsp;25&nbsp;seconds</td></tr>
<tr><td>Identifying junction candidates</td><td>10:45:09&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:45:51&nbsp;28&nbsp;Feb&nbsp;2025</td><td>42&nbsp;seconds</td></tr>
<tr><td>Re-alignment to junction candidates</td><td>10:45:51&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:47:07&nbsp;28&nbsp;Feb&nbsp;2025</td><td>1&nbsp;minute&nbsp;16&nbsp;seconds</td></tr>
<tr><td>Resolving best read alignments</td><td>10:47:07&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:51:41&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;minutes&nbsp;34&nbsp;seconds</td></tr>
<tr><td>Creating BAM files</td><td>10:51:41&nbsp;28&nbsp;Feb&nbsp;2025</td><td>10:55:55&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;minutes&nbsp;14&nbsp;seconds</td></tr>
<tr><td>Tabulating error counts</td><td>10:55:55&nbsp;28&nbsp;Feb&nbsp;2025</td><td>11:00:41&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;minutes&nbsp;46&nbsp;seconds</td></tr>
<tr><td>Re-calibrating base error rates</td><td>11:00:41&nbsp;28&nbsp;Feb&nbsp;2025</td><td>11:01:25&nbsp;28&nbsp;Feb&nbsp;2025</td><td>44&nbsp;seconds</td></tr>
<tr><td>Examining read alignment evidence</td><td>11:01:25&nbsp;28&nbsp;Feb&nbsp;2025</td><td>11:27:26&nbsp;28&nbsp;Feb&nbsp;2025</td><td>26&nbsp;minutes&nbsp;1&nbsp;second</td></tr>
<tr><td>Polymorphism statistics</td><td>11:27:26&nbsp;28&nbsp;Feb&nbsp;2025</td><td>11:27:28&nbsp;28&nbsp;Feb&nbsp;2025</td><td>2&nbsp;seconds</td></tr>
<tr><td>Output</td><td>11:27:28&nbsp;28&nbsp;Feb&nbsp;2025</td><td>11:27:40&nbsp;28&nbsp;Feb&nbsp;2025</td><td>12&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Prediction</td><td>11:27:28&nbsp;28&nbsp;Feb&nbsp;2025</td><td>11:27:32&nbsp;28&nbsp;Feb&nbsp;2025</td><td>4&nbsp;seconds</td></tr>
<tr><td>Output :: Mutation Annotation</td><td>11:27:32&nbsp;28&nbsp;Feb&nbsp;2025</td><td>11:27:32&nbsp;28&nbsp;Feb&nbsp;2025</td><td>0&nbsp;seconds</td></tr>
<tr class="highlight_table_row">
<td colspan="3" ><b>Total</b></td>
<td><b>59&nbsp;minutes&nbsp;26&nbsp;seconds</b></td>
</tr>
</table></body></html>